<!DOCTYPE html>
<!-- saved from url=(0022)http://localhost:8080/ -->
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>PubMedylicious </title>

    <!-- Bootstrap core CSS -->
    <link href="http://localhost:8080/css/bootstrap.css" rel="stylesheet">
    <link href="http://netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap-glyphicons.css" rel="stylesheet">
    <link href="http://netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css" rel="stylesheet">

    <style>
        .dropdown-backdrop {
            position: static;
        }
    </style>
    <style id="style-1-cropbar-clipper">/* Copyright 2014 Evernote Corporation. All rights reserved. */
.en-markup-crop-options {
    top: 18px !important;
    left: 50% !important;
    margin-left: -100px !important;
    width: 200px !important;
    border: 2px rgba(255,255,255,.38) solid !important;
    border-radius: 4px !important;
}

.en-markup-crop-options div div:first-of-type {
    margin-left: 0px !important;
}
</style></head>

    <body>

<div class="well well-lg">
    <h1>
        Your Query : <strong>Dna Methylation</strong>
    </h1>                    
</div>

<!-- Area for abstracts -->
<h1>197 Abstracts Found </h1>

<div class="container">
    <div class="row">
            <div class="col-md-6">
            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25890369" target="_blank">
                    miR-130a can predict response to temozolomide in patients with glioblastoma multiforme, independently of O6-methylguanine-DNA methyltransferase.
                </a>
            </h2>
            <p>Currently, O6-methylguanine-DNA methyltransferase(MGMT) promoter methylation is the most convincing predictive biomarker for temozolomide (TMZ) response in patients with glioblastoma multiforme (GBM). However, technical obstacles prevent this biomarker from being applied widely. On the other hand, microRNAs (miRNAs) are easily investigated in the clinical setting using quantitative real-time polymerase chain reactions. <b>This study aimed to identify miRNAs that could serve as predictive biomarkers for TMZ response.The Cancer Genome Atlas (TCGA) and the Chinese Glioma Genome Atlas (CGGA) databases were used to investigate the significance of associations between miRNA expression and overall survival (OS) in TMZ-treated patients with GBM.</b> <b>Cytotoxicity assays were used to validate the miRNAs' roles in the response of glioma cells to TMZ.</b> Biological insights concerning the miRNAs were explored using gene set enrichment analysis (GSEA) and gene ontology (GO) analysis.miR-130a was found to be significantly associated with OS in TMZ-treated patients from TCGA and the CGGA. In contrast, miR-130a appeared to be unassociated with OS in patients who only received radiotherapy. <b>The TMZ cytotoxicity assay showed that miR-130a over-expression could sensitize response to TMZ in glioma cells.</b> <b>GSEA and GO analysis indicated that lower miR-130a could generate a more extensive response to oxidative stress, which in turn could elevate Ape1 and mediate resistance to TMZ.</b> <b>In vitro experiment verified that cells with lower miR-130a express higher Ape1 under oxidative stress.Our data suggested that miR-130a could be a predictive marker for TMZ response in patients with GBM, independently of the mechanism by which MGMT acts as a biomarker.</b> miR-130a could serve as a guide for treatment strategy selection in cases of GBM.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25890285" target="_blank">
                    In-depth genomic data analyses revealed complex transcriptional and epigenetic dysregulations of BRAF (V600E) in melanoma.
                </a>
            </h2>
            <p>The recurrent BRAF driver mutation V600E (BRAF (V600E)) is currently one of the most clinically relevant mutations in melanoma. However, the genome-wide transcriptional and epigenetic dysregulations induced by BRAF (V600E) are still unclear. The investigation of this driver mutation's functional consequences is critical to the understanding of tumorigenesis and the development of therapeutic strategies.We performed an integrative analysis of transcriptomic and epigenomic changes disturbed by BRAF (V600E) by comparing the gene expression and methylation profiles of 34 primary cutaneous melanoma tumors harboring BRAF (V600E) with those of 27 BRAF (WT) samples available from The Cancer Genome Atlas (TCGA). A total of 711 significantly differentially expressed genes were identified as putative BRAF (V600E) target genes. Functional enrichment analyses revealed the transcription factor MITF (p &lt; 3.6 × 10(-16)) and growth factor TGFB1 (p &lt; 3.1 × 10(-9)) were the most significantly enriched up-regulators, with MITF being significantly up-regulated, whereas TGFB1 was significantly down-regulated in BRAF (V600E), suggesting that they may mediate tumorigenesis driven by BRAF (V600E). <b>Further investigation using the MITF ChIP-Seq data confirmed that BRAF (V600E) led to an overall increased level of gene expression for the MITF targets.</b> <b>Furthermore, DNA methylation analysis revealed a global DNA methylation loss in BRAF (V600E) relative to BRAF (WT).</b> <b>This might be due to BRAF dysregulation of DNMT3A, which was identified as a potential target with significant down-regulation in BRAF (V600E).</b> <b>Finally, we demonstrated that BRAF (V600E) targets may play essential functional roles in cell growth and proliferation, measured by their effects on melanoma tumor growth using a short hairpin RNA silencing experimental dataset.Our integrative analysis identified a set of BRAF (V600E) target genes.</b> <b>Further analyses suggested a complex mechanism driven by mutation BRAF (V600E) on melanoma tumorigenesis that disturbs specific cancer-related genes, pathways, and methylation modifications.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25889455" target="_blank">
                    TP53 R72P polymorphism modulates DNA methylation in hepatocellular carcinoma.
                </a>
            </h2>
            <p>Hepatocellular carcinoma (HCC) is characterized by widespread epidemiological and molecular heterogeneity. Previous work showed that in the western part of North Africa, a region of low incidence of HCC, mutations are scarce for this tumor type. <b>As epigenetic changes are considered possible surrogates to mutations in human cancers, we decided, thus, to characterize DNA methylation in HCC from North-African patients.A set of 11 loci was investigated in a series of 45 tumor specimens using methylation-specific and combined-bisulfite restriction assay PCR.</b> Results obtained on clinical samples were subsequently validated in liver cancer cell lines.DNA methylation at tumor suppressor loci is significantly higher in samples displaying chromosome instability. <b>More importantly, DNA methylation was significantly higher in Arg/Arg when compared to Pro/Pro genotype carriers at codon 72 rs1042522 of TP53 (65% vs 20% methylated loci, p = 0.0006), a polymorphism already known to affect somatic mutation rate in human carcinomas.</b> <b>In vitro experiments in cell lines indicated that enzymes controlling DNA methylation were differentially regulated by codon 72 Arg or Pro isoforms of p53.</b> <b>Furthermore, the Arg72-carrying version of p53 was shown to re-methylate DNA more rapidly than the pro-harboring isoform.</b> Finally, Pro-carrying cell lines were shown to be significantly more resistant to decitabine treatment (two-fold, p = 0.005).Our data suggest that Arg72Pro polymorphism in a WT p53 context may act as a primary driver of epigenetic changes in HCC. It suggests, in addition, that rs1042522 genotype may predict sensitivity to epigenetic-targeted therapy. This model of liver tumorigenesis that associates low penetrance genetic predisposition to epigenetic changes emerges from a region of low HCC incidence and it may, therefore, apply essentially to population living in similar areas. Surveys on populations submitted to highly mutagenic conditions as perinatally-acquired chronic hepatitis B or aflatoxin B1 exposure remained to be conducted to validate our observations as a general model.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25889032" target="_blank">
                    Life-course socioeconomic status and DNA methylation of genes regulating inflammation.
                </a>
            </h2>
            <p>In humans, low socioeconomic status (SES) across the life course is associated with greater diurnal cortisol production, increased inflammatory activity and higher circulating antibodies for several pathogens, all suggesting a dampened immune response. Recent evidence suggests that DNA methylation of pro-inflammatory genes may be implicated in the biological embedding of the social environment.The present study examines the association between life-course SES and DNA methylation of candidate genes, selected on the basis of their involvement in SES-related inflammation, in the context of a genome-wide methylation study. <b>Participants were 857 healthy individuals sampled from the EPIC Italy prospective cohort study.Indicators of SES were associated with DNA methylation of genes involved in inflammation.</b> <b>NFATC1, in particular, was consistently found to be less methylated in individuals with low vs high SES, in a dose-dependent manner.</b> <b>IL1A, GPR132 and genes belonging to the MAPK family were also less methylated among individuals with low SES.</b> <b>In addition, associations were found between SES and CXCL2 and PTGS2, but these genes were consistently more methylated among low SES individuals.Our findings support the hypothesis that the social environment leaves an epigenetic signature in cells.</b> <b>Although the functional significance of SES-related DNA methylation is still unclear, we hypothesize that it may link SES to chronic disease risk.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25889027" target="_blank">
                    Clinical implications of the quantitative detection of ID4 gene methylation in myelodysplastic syndrome.
                </a>
            </h2>
            <p>Myelodysplastic syndrome (MDS) eventually transforms into acute leukemia (AL) in about 30% of patients. Hypermethylation of the inhibitor of DNA binding 4 (ID4) gene may play an important role in the initiation and development of MDS and AL. The aim of this study was to quantitatively assess ID4 gene methylation in MDS and to establish if it could be an effective method of evaluating MDS disease progression.We examined 142 bone marrow samples from MDS patients, healthy donors and MDS-AL patients using bisulfite sequencing PCR and quantitative real-time methylation-specific PCR. The ID4 methylation rates and levels were assessed.ID4 methylation occurred in 27 patients (27/100). ID4 gene methylation was more frequent and at higher levels in patients with advanced disease stages and in high-risk subgroups according to WHO (P &lt; 0.001, P &lt; 0.001, respectively) and International Prognostic Scoring System (IPSS) (P = 0.002, P = 0.007, respectively) classifications. <b>ID4 methylation levels changed during disease progression.</b> <b>Both methylation rates and methylation levels were significantly different between healthy donor, MDS patients and patients with MDS-AL (P &lt; 0.001, P &lt; 0.001, respectively).</b> <b>Multivariate analysis indicated that the level of ID4 methylation was an independent factor influencing overall survival.</b> <b>Patients with MDS showed decreased survival time with increased ID4 methylation levels (P = 0.011, hazard ratio (HR) = 2.371).</b> <b>Patients with ID4 methylation had shorter survival time than those without ID4 methylation (P = 0.008).Our findings suggest that ID4 gene methylation might be a new biomarker for MDS monitoring and the detection of minimal residual disease.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25889020" target="_blank">
                    Methylation of NR3C1 and SLC6A4 and internalizing problems. The TRAILS study.
                </a>
            </h2>
            <p>The relationship between early adverse life events and later internalizing problems could be mediated by DNA methylation. Adversity has been associated with higher methylation levels in the glucocorticoid receptor gene (NR3C1) and the serotonin transporter gene (SLC6A4) in adolescents. We investigated cross-sectional and prospective associations of NR3C1 and SLC6A4 methylation with adolescents׳ clinical diagnoses of internalizing disorders and internalizing symptom scores.In a population sample (mean age=16.2) we measured DNA methylation in three regions of NR3C1 (NR3C1_1, N=454; NR3C1_2, N=904; NR3C1_3, N=412) and one region of SLC6A4 (N=939) at baseline. <b>Internalizing problems were operationalized as clinical DSM-IV diagnoses, assessed at 3 year follow-up with a diagnostic interview, and internalizing symptom scores, assessed with Self-Report questionnaires at baseline and follow-up.Only NR3C1_1 methylation was positively associated with risk of lifetime internalizing disorders, and with symptom scores at follow-up.</b> <b>However, after accounting for baseline symptom scores there was only a tendency for association with internalizing symptom scores at follow-up.</b> <b>There was no association between SLC6A4 methylation and risk of lifetime internalizing disorders.</b> <b>SLC6A4 methylation and internalizing symptom scores showed a tendency for association, also after accounting for baseline symptom scores.There was no repeated measure of DNA methylation to study causality between methylation and internalizing problems.</b> <b>Gene expression data were not available.Although the role of gene methylation in the development of internalizing problems remains unclear, our findings suggest that gene methylation, particularly of NR3C1, may be involved in the development of internalizing problems in adolescence.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25888794" target="_blank">
                    Identification and characterization of DNAzymes targeting DNA methyltransferase I for suppressing bladder cancer proliferation.
                </a>
            </h2>
            <p>Epigenetic inactivation of genes plays a critical role in many important human diseases, especially in cancer. <b>A core mechanism for epigenetic inactivation of the genes is methylation of CpG islands in genome DNA, which is catalyzed by DNA methyltransferases (DNMTs).</b> <b>The inhibition of DNMTs may lead to demethylation and expression of the silenced tumor suppressor genes.</b> Although DNMT inhibitors are currently being developed as potential anticancer agents, only limited success is achieved due to substantial toxicity. Here, we utilized a multiplex selection system to generate efficient RNA-cleaving DNAzymes targeting DNMT1. The lead molecule from the selection was shown to possess efficient kinetic profiles and high efficiency in inhibiting the enzyme activity. Transfection of the DNAzyme caused significant down-regulation of DNMT1 expression and reactivation of p16 gene, resulting in reduced cell proliferation of bladder cancers. <b>This study provides an alternative for targeting DNMTs for potential cancer therapy.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25888776" target="_blank">
                    Sensitivity and specificity of immunoprecipitation of DNA containing 5-Methylcytosine.
                </a>
            </h2>
            <p>Attempts to enrich or identify DNA with cytosine methylation have been commonly carried out using anti-5-methylcytosine or anti-MBD2 (methyl-CpG binding domain protein 2) antibody in immunoprecipitation (IP) assays. However, a careful and systematic control experiment to examine the sensitivity and specificity of this approach has not been reported. It is of critical importance to understand the potential pitfalls of this approach and to avoid potential misinterpretation of findings.We found that increased concentration of antibody used in the assay increased the amount of overall DNA captured as expected. <b>The increased number of methylated cytosines in/on the DNA fragment also increased the amount of DNA captured by the antibody.</b> <b>Importantly, the antibody can bind to some fully unmethylated DNA fragments, even when fully methylated DNA is present in the same experiment.The sensitivity of anti-5-methylcytosine antibody and anti-MBD2 antibody/MBD2 binding varies with the number of methylated cytosines on the DNA target.</b> <b>The specificity of these antibodies can also vary for different DNA target sequences.</b> <b>DNA fragments with fewer CpG sites may not bind to these antibodies even when all are methylated while DNA fragments with more CpG sites may bind to the antibodies when only some of these sites are methylated.</b> <b>More importantly, binding of DNA to these antibodies does not always indicate the presence of DNA methylation.</b> It is clear that false positive and false negative findings can be easily reached even though it does not nullify these convenient and simple methods completely. Great caution should be taken for the interpretation of IP results using these antibodies and rigorous confirmation by sodium bisulfite sequencing is essential.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25888628" target="_blank">
                    Manipulation of prostate cancer metastasis by locus-specific modification of the CRMP4 promoter region using chimeric TALE DNA methyltransferase and demethylase.
                </a>
            </h2>
            <p><b>Prostate cancer is the most commonly diagnosed non-cutaneous cancer and one of the leading causes of cancer death for North American men.</b> <b>Whereas localized prostate cancer can be cured, there is currently no cure for metastatic prostate cancer.</b> <b>Here we report a novel approach that utilizes designed chimeric transcription activator-like effectors (dTALEs) to control prostate cancer metastasis.</b> Transfection of dTALEs of DNA methyltransferase or demethylase induced artificial, yet active locus-specific CpG and subsequent histone modifications. These manipulations markedly altered expression of endogenous CRMP4, a metastasis suppressor gene. Remarkably, locus-specific CpG demethylation of the CRMP4 promoter in metastatic PC3 cells abolished metastasis, whereas locus-specific CpG methylation of the promoter in non-metastatic 22Rv1 cells induced metastasis. CRMP4-mediated metastasis suppression was found to require activation of Akt/Rac1 signaling and down-regulation of MMP-9 expression. <b>This proof-of-concept study with dTALEs for locus-specific epigenomic manipulation validates the selected CpG methylation of CRMP4 gene as an independent biomarker for diagnosis and prognosis of prostate cancer metastasis and opens up a novel avenue for mechanistic research on cancer biology.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25888305" target="_blank">
                    A network model for angiogenesis in ovarian cancer.
                </a>
            </h2>
            <p><b>We recently identified two robust ovarian cancer subtypes, defined by the expression of genes involved in angiogenesis, with significant differences in clinical outcome.</b> <b>To identify potential regulatory mechanisms that distinguish the subtypes we applied PANDA, a method that uses an integrative approach to model information flow in gene regulatory networks.We find distinct differences between networks that are active in the angiogenic and non-angiogenic subtypes, largely defined by a set of key transcription factors that, although previously reported to play a role in angiogenesis, are not strongly differentially-expressed between the subtypes.</b> <b>Our network analysis indicates that these factors are involved in the activation (or repression) of different genes in the two subtypes, resulting in differential expression of their network targets.</b> <b>Mechanisms mediating differences between subtypes include a previously unrecognized pro-angiogenic role for increased genome-wide DNA methylation and complex patterns of combinatorial regulation.The models we develop require a shift in our interpretation of the driving factors in biological networks away from the genes themselves and toward their interactions.</b> <b>The observed regulatory changes between subtypes suggest therapeutic interventions that may help in the treatment of ovarian cancer.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25888029" target="_blank">
                    Ageing-associated changes in the human DNA methylome: genomic locations and effects on gene expression.
                </a>
            </h2>
            <p>Changes in DNA methylation are among the mechanisms contributing to the ageing process. We sought to identify ageing-associated DNA methylation changes at single-CpG-site resolution in blood leukocytes and to ensure that the observed changes were not due to differences in the proportions of leukocytes. The association between DNA methylation changes and gene expression levels was also investigated in the same individuals.We identified 8540 high-confidence ageing-associated CpG sites, 46% of which were hypermethylated in nonagenarians. The hypermethylation-associated genes belonged to a common category: they were predicted to be regulated by a common group of transcription factors and were enriched in a related set of GO terms and canonical pathways. Conversely, for the hypomethylation-associated genes only a limited set of GO terms and canonical pathways were identified. <b>Among the 8540 CpG sites associated with ageing, methylation level of 377 sites was also associated with gene expression levels.</b> <b>These genes were enriched in GO terms and canonical pathways associated with immune system functions, particularly phagocytosis.We find that certain ageing-associated immune-system impairments may be mediated via changes in DNA methylation.</b> The results also imply that ageing-associated hypo- and hypermethylation are distinct processes: hypermethylation could be caused by programmed changes, whereas hypomethylation could be the result of environmental and stochastic processes.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25887996" target="_blank">
                    Global scale transcriptome analysis of Arabidopsis embryogenesis in vitro.
                </a>
            </h2>
            <p>Somatic embryogenesis (SE) in plants is a process by which embryos are generated directly from somatic cells, rather than from the fused products of male and female gametes. Despite the detailed expression analysis of several somatic-to-embryonic marker genes, a comprehensive understanding of SE at a molecular level is still lacking. The present study was designed to generate high resolution transcriptome datasets for early SE providing the way for future research to understand the underlying molecular mechanisms that regulate this process. We sequenced Arabidopsis thaliana somatic embryos collected from three distinct developmental time-points (5, 10 and 15 d after in vitro culture) using the Illumina HiSeq 2000 platform.This study yielded a total of 426,001,826 sequence reads mapped to 26,520 genes in the A. thaliana reference genome. <b>Analysis of embryonic cultures after 5 and 10 d showed differential expression of 1,195 genes; these included 778 genes that were more highly expressed after 5 d as compared to 10 d. Moreover, 1,718 genes were differentially expressed in embryonic cultures between 10 and 15 d. Our data also showed at least eight different expression patterns during early SE; the majority of genes are transcriptionally more active in embryos after 5 d. Comparison of transcriptomes derived from somatic embryos and leaf tissues revealed that at least 4,951 genes are transcriptionally more active in embryos than in the leaf; increased expression of genes involved in DNA cytosine methylation and histone deacetylation were noted in embryogenic tissues.</b> <b>In silico expression analysis based on microarray data found that approximately 5% of these genes are transcriptionally more active in somatic embryos than in actively dividing callus and non-dividing leaf tissues.</b> <b>Moreover, this identified 49 genes expressed at a higher level in somatic embryos than in other tissues.</b> <b>This included several genes with unknown function, as well as others related to oxidative and osmotic stress, and auxin signalling.The transcriptome information provided here will form the foundation for future research on genetic and epigenetic control of plant embryogenesis at a molecular level.</b> <b>In follow-up studies, these data could be used to construct a regulatory network for SE; the genes more highly expressed in somatic embryos than in vegetative tissues can be considered as potential candidates to validate these networks.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25887558" target="_blank">
                    A simple method for semi-random DNA amplicon fragmentation using the methylation-dependent restriction enzyme MspJI.
                </a>
            </h2>
            <p><b>Fragmentation at random nucleotide locations is an essential process for preparation of DNA libraries to be used on massively parallel short-read DNA sequencing platforms.</b> Although instruments for physical shearing, such as the Covaris S2 focused-ultrasonicator system, and products for enzymatic shearing, such as the Nextera technology and NEBNext dsDNA Fragmentase kit, are commercially available, a simple and inexpensive method is desirable for high-throughput sequencing library preparation. <b>MspJI is a recently characterised restriction enzyme which recognises the sequence motif CNNR (where R = G or A) when the first base is modified to 5-methylcytosine or 5-hydroxymethylcytosine.A semi-random enzymatic DNA amplicon fragmentation method was developed based on the unique cleavage properties of MspJI.</b> <b>In this method, random incorporation of 5-methyl-2'-deoxycytidine-5'-triphosphate is achieved through DNA amplification with DNA polymerase, followed by DNA digestion with MspJI.</b> <b>Due to the recognition sequence of the enzyme, DNA amplicons are fragmented in a relatively sequence-independent manner.</b> The size range of the resulting fragments was capable of control through optimisation of 5-methyl-2'-deoxycytidine-5'-triphosphate concentration in the reaction mixture. A library suitable for sequencing using the Illumina MiSeq platform was prepared and processed using the proposed method. Alignment of generated short reads to a reference sequence demonstrated a relatively high level of random fragmentation.The proposed method may be performed with standard laboratory equipment. Although the uniformity of coverage was slightly inferior to the Covaris physical shearing procedure, due to efficiencies of cost and labour, the method may be more suitable than existing approaches for implementation in large-scale sequencing activities, such as bacterial artificial chromosome (BAC)-based genome sequence assembly, pan-genomic studies and locus-targeted genotyping-by-sequencing.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25887538" target="_blank">
                    Implication of coronin 7 in body weight regulation in humans, mice and flies.
                </a>
            </h2>
            <p>Obesity is a growing global concern with strong associations with cardiovascular disease, cancer and type-2 diabetes. Although various genome-wide association studies have identified more than 40 genes associated with obesity, these genes cannot fully explain the heritability of obesity, suggesting there may be other contributing factors, including epigenetic effects.We performed genome wide DNA methylation profiling comparing normal-weight and obese 9-13 year old children to investigate possible epigenetic changes correlated with obesity. <b>Of note, obese children had significantly lower methylation levels at a CpG site located near coronin 7 (CORO7), which encodes a tryptophan-aspartic acid dipeptide (WD)-repeat containing protein most likely involved in Golgi complex morphology and function.</b> <b>Anatomical profiling of coronin 7 (Coro7) mRNA expression in mice revealed that it is highly expressed in appetite and energy balance regulating regions, including the hypothalamus, striatum and locus coeruleus, the main noradrenergic brain site.</b> <b>Interestingly, we found that food deprivation in mice downregulates hypothalamic Coro7 mRNA levels, and injecting ethanol, an appetite stimulant, increased the number of Coro7 expressing cells in the locus coeruleus.</b> <b>Finally, by employing the genetically-tractable Drosophila melanogaster model we were able to demonstrate an evolutionarily conserved metabolic function for the CORO7 homologue pod1.</b> Knocking down the pod1 in the Drosophila adult nervous system increased their resistance to starvation. Furthermore, feeding flies a high-calorie diet significantly increased pod1 expression.We conclude that coronin 7 is involved in the regulation of energy homeostasis and this role stems, to some degree, from the effect on feeding for calories and reward.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25887533" target="_blank">
                    Genome-wide analysis of DNA methylation in photoperiod- and thermo-sensitive male sterile rice Peiai 64S.
                </a>
            </h2>
            <p>Epigenetic modifications play important roles in the regulation of plant development. DNA methylation is an important epigenetic modification that dynamically regulates gene expression during developmental processes. <b>However, little studies have been reported about the methylation profiles of photoperiod- and thermo-sensitive genic male sterile (PTGMS) rice during the fertility transition.In this study, using methylated DNA immunoprecipitation sequencing (MeDIP-seq), the global DNA methylation patterns were compared in the rice PTGMS line PA64S under two different environments (different temperatures and day lengths).</b> <b>The profiling of the DNA methylation under two different phenotypes (sterility and fertility) revealed that hypermethylation was observed in PA64S (sterility), and 1258 differentially methylated regions (DMRs) were found between PA64S (sterility) and PA64S (fertility).</b> Twenty differentially methylated genes of them were further validated through bisulfite sequencing, and four of these genes were analyzed by qRT-PCR. Especially, a differentially methylated gene (LOC_Os08g38210), which encoded transcription factor BIM2, is a component of brassinosteroid signaling in rice. <b>The hypermethylated BIM2 gene may suppress some downstream genes in brassinosteroid signaling pathway, and thus affect the male fertility in PA64S.The results presented here indicated that hypermethylation was observed in PA64S (sterility).</b> <b>Gene Ontology (GO) analysis and KEGG analysis revealed that flavone and flavonol biosynthrsis, circadian rhythm, photosynthesis and oxidative phosphorylation pathways were involved in sterility-fertility transition of PA64S.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25887185" target="_blank">
                    Identification of radiation-induced aberrant hypomethylation in colon cancer.
                </a>
            </h2>
            <p>Exposure to ionizing radiation (IR) results in the simultaneous activation or downregulation of multiple signaling pathways that play critical roles in cell type-specific control of survival or death. IR is a well-known genotoxic agent and human carcinogen that induces cellular damage through direct and indirect mechanisms. <b>However, its impact on epigenetic mechanisms has not been elucidated, and more specifically, little information is available regarding genome-wide DNA methylation changes in cancer cells after IR exposure.</b> <b>Recently, genome-wide DNA methylation profiling technology using the Illumina HumanMethylation450K platform has emerged that allows us to query &gt;450,000 loci within the genome.</b> <b>This improved technology is capable of identifying genome-wide DNA methylation changes in CpG islands and other CpG island-associated regions.In this study, we employed this technology to test the hypothesis that exposure to IR not only induces differential DNA methylation patterns at a genome-wide level, but also results in locus- and gene-specific DNA methylation changes.</b> <b>We screened for differential DNA methylation changes in colorectal cancer cells after IR exposure with 2 and 5&nbsp;Gy.</b> Twenty-nine genes showed radiation-induced hypomethylation in colon cancer cells, and of those, seven genes showed a corresponding increase in gene expression by reverse transcriptase polymerase chain reaction (RT-PCR). In addition, we performed chromatin immunoprecipitation (ChIP) to confirm that the DNA-methyltransferase 1 (DNMT1) level associated with the promoter regions of these genes correlated with their methylation level and gene expression changes. <b>Finally, we used a gene ontology (GO) database to show that a handful of hypomethylated genes induced by IR are associated with a variety of biological pathways related to cancer.We identified alterations in global DNA methylation patterns and hypomethylation at specific cancer-related genes following IR exposure, which suggests that radiation exposure plays a critical role in conferring epigenetic alterations in cancer.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25887183" target="_blank">
                    Drinking beyond a lifetime: New and emerging insights into paternal alcohol exposure on subsequent generations.
                </a>
            </h2>
            <p>Alcohol-use disorder (AUD) is prevalent and associated with substantial socioeconomic costs. While heritability estimates of AUD are ∼50%, identifying specific gene variants associated with risk for AUD has proven challenging despite considerable investment. Emerging research into heritability of complex diseases has implicated transmission of epigenetic variants in the development of behavioral phenotypes, including drug preference and drug-induced behavior. <b>Several recent rodent studies have specifically focused on paternal transmission of epigenetic variants, which is especially relevant because sires are not present for offspring rearing and changes to offspring phenotype are assumed to result from modifications to the sperm epigenome.</b> <b>While considerable interest in paternal transmission of epigenetic variants has emerged recently, paternal alcohol exposures have been studied for 30+ years with interesting behavioral and physiologic effects noted on offspring.</b> <b>However, only recently, with improvements in technology to identify epigenetic modifications in germ cells, has it been possible to identify mechanisms by which paternal ethanol exposure alters offspring behavior.</b> <b>This review presents an overview of epigenetic inheritance in the context of paternal ethanol exposure and suggests future studies to identify specific effects of paternal ethanol exposure on offspring behavior and response to ethanol.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25887114" target="_blank">
                    Cell-composition effects in the analysis of DNA methylation array data: a mathematical perspective.
                </a>
            </h2>
            <p>The impact of cell-composition effects in analysis of DNA methylation data is now widely appreciated. <b>With the availability of a reference data set consisting of DNA methylation measurements on isolated cell types, it is possible to impute cell proportions and adjust for them, but there is increasing interest in methods that adjust for cell composition effects when reference sets are incomplete or unavailable.In this article we present a theoretical basis for one such method, showing that the total effect of a phenotype on DNA methylation can be decomposed into orthogonal components, one representing the effect of phenotype on proportions of major cell types, the other representing either subtle effects in composition or global effects at focused loci, and that it is possible to separate these two types of effects in a finite data set.</b> We demonstrate this principle empirically on nine DNA methylation data sets, showing that the first few principal components generally contain a majority of the information on cell-type present in the data, but that later principal components nevertheless contain information about a small number of loci that may represent more focused associations. We also present a new method for determining the number of linear terms to interpret as cell-mixture effects and demonstrate robustness to the choice of this parameter.Taken together, our work demonstrates that reference-free algorithms for cell-mixture adjustment can produce biologically valid results, separating cell-mediated epigenetic effects (i.e. <b>apparent effects arising from differences in cell composition) from those that are not cell mediated, and that in general the interpretation of associations evident from DNA methylation should be carefully considered.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25886910" target="_blank">
                    Loss of TET2 in hematopoietic cells leads to DNA hypermethylation of active enhancers and induction of leukemogenesis.
                </a>
            </h2>
            <p>DNA methylation is tightly regulated throughout mammalian development, and altered DNA methylation patterns are a general hallmark of cancer. <b>The methylcytosine dioxygenase TET2 is frequently mutated in hematological disorders, including acute myeloid leukemia (AML), and has been suggested to protect CG dinucleotide (CpG) islands and promoters from aberrant DNA methylation.</b> In this study, we present a novel Tet2-dependent leukemia mouse model that closely recapitulates gene expression profiles and hallmarks of human AML1-ETO-induced AML. <b>Using this model, we show that the primary effect of Tet2 loss in preleukemic hematopoietic cells is progressive and widespread DNA hypermethylation affecting up to 25% of active enhancer elements.</b> In contrast, CpG island and promoter methylation does not change in a Tet2-dependent manner but increases relative to population doublings. <b>We confirmed this specific enhancer hypermethylation phenotype in human AML patients with TET2 mutations.</b> Analysis of immediate gene expression changes reveals rapid deregulation of a large number of genes implicated in tumorigenesis, including many down-regulated tumor suppressor genes. <b>Hence, we propose that TET2 prevents leukemic transformation by protecting enhancers from aberrant DNA methylation and that it is the combined silencing of several tumor suppressor genes in TET2 mutated hematopoietic cells that contributes to increased stem cell proliferation and leukemogenesis.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25886813" target="_blank">
                    Identification and Functional Characterizations of N-terminal α-N-methylation and Phosphorylation of Serine 461 in Human Poly(ADP-ribose) Polymerase 3.
                </a>
            </h2>
            <p>Poly(ADP-ribose) polymerase 3 (PARP3) is a member of the PARP family enzymes which catalyze the ADP-ribosylation of proteins. <b>PARP3 plays an important role in DNA damage repair and mitotic progression.</b> <b>In this study, we identified, using mass spectrometric techniques, two novel post-translational modification sites in PARP3, α-N-methylation and phosphorylation of serine 461 (S461).</b> <b>We found that the N-terminal α-amino group of PARP3 is heavily methylated in human cells and N-terminal RCC1 methyltransferase (NRMT) is a key enzyme required for this methylation.</b> <b>Additionally, we observed that the phosphorylation level of S461 in PARP3 could be reduced in human cells upon treatment with flavopiridol, a cyclin-dependent kinase inhibitor.</b> <b>Moreover, we demonstrated that S461 phosphorylation, but not α-N-methylation of PARP3, may be involved in the cellular response toward DNA double-strand breaks.</b> These findings provided novel insights into the post-translational regulation of PARP3.ααα.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25886572" target="_blank">
                    Whole genome bisulfite sequencing of cell-free DNA and its cellular contributors uncovers placenta hypomethylated domains.
                </a>
            </h2>
            <p><b>Circulating cell-free fetal DNA has enabled non-invasive prenatal fetal aneuploidy testing without direct discrimination of the maternal and fetal DNA.</b> <b>Testing may be improved by specifically enriching the sample material for fetal DNA.</b> <b>DNA methylation may allow for such a separation of DNA; however, this depends on knowledge of the methylomes of circulating cell-free DNA and its cellular contributors.We perform whole genome bisulfite sequencing on a set of unmatched samples including circulating cell-free DNA from non-pregnant and pregnant female donors and genomic DNA from maternal buffy coat and placenta samples.</b> We find CpG cytosines within longer fragments are more likely to be methylated. <b>Comparison of the methylomes of placenta and non-pregnant circulating cell-free DNA reveal many of the 51,259 identified differentially methylated regions are located in domains exhibiting consistent placenta hypomethylation across millions of consecutive bases.</b> We find these placenta hypomethylated domains are consistently located within regions exhibiting low CpG and gene density. <b>Differentially methylated regions identified when comparing placenta to non-pregnant circulating cell-free DNA are recapitulated in pregnant circulating cell-free DNA, confirming the ability to detect differential methylation in circulating cell-free DNA mixtures.We generate methylome maps for four sample types at single-base resolution, identify a link between DNA methylation and fragment length in circulating cell-free DNA, identify differentially methylated regions between sample groups, and uncover the presence of megabase-size placenta hypomethylated domains.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25886480" target="_blank">
                    Age and sun exposure-related widespread genomic blocks of hypomethylation in nonmalignant skin.
                </a>
            </h2>
            <p><b>Aging and sun exposure are the leading causes of skin cancer.</b> <b>It has been shown that epigenetic changes, such as DNA methylation, are well established mechanisms for cancer, and also have emerging roles in aging and common disease.</b> <b>Here, we directly ask whether DNA methylation is altered following skin aging and/or chronic sun exposure in humans.We compare epidermis and dermis of both sun-protected and sun-exposed skin derived from younger subjects (under 35&nbsp;years old) and older subjects (over 60&nbsp;years old), using the Infinium HumanMethylation450 array and whole genome bisulfite sequencing.</b> We observe large blocks of the genome that are hypomethylated in older, sun-exposed epidermal samples, with the degree of hypomethylation associated with clinical measures of photo-aging. We replicate these findings using whole genome bisulfite sequencing, comparing epidermis from an additional set of younger and older subjects. These blocks largely overlap known hypomethylated blocks in colon cancer and we observe that these same regions are similarly hypomethylated in squamous cell carcinoma samples.These data implicate large scale epigenomic change in mediating the effects of environmental damage with photo-aging.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25886310" target="_blank">
                    DNA methyltransferase 1 functions through C/ebpa to maintain hematopoietic stem and progenitor cells in zebrafish.
                </a>
            </h2>
            <p>DNA methyltransferase 1 (Dnmt1) regulates expression of many critical genes through maintaining parental DNA methylation patterns on daughter DNA strands during mitosis. It is essential for embryonic development and diverse biological processes, including maintenance of hematopoietic stem and progenitor cells (HSPCs). <b>However, the precise molecular mechanism of how Dnmt1 is involved in HSPC maintenance remains unexplored.An N-ethyl-N-nitrosourea (ENU)-based genetic screening was performed to identify putative mutants with defects in definitive HSPCs during hematopoiesis in zebrafish.</b> The expression of hematopoietic markers was analyzed via whole mount in situ hybridization assay (WISH). Positional cloning approach was carried out to identify the gene responsible for the defective definitive hematopoiesis in the mutants. Analyses of the mechanism were conducted by morpholino-mediated gene knockdown, mRNA injection rescue assays, anti-phosphorylated histone H3 (pH3) immunostaining and TUNEL assay, quantitative real-time PCR, and bisulfite sequencing analysis.A heritable mutant line with impaired HSPCs of definitive hematopoiesis was identified. <b>Positional cloning demonstrated that a stop codon mutation was introduced in dnmt1 which resulted in a predicted truncated Dnmt1 lacking the DNA methylation catalytic domain.</b> <b>Molecular analysis revealed that expression of CCAAT/enhancer-binding protein alpha (C/ebpa) was upregulated, which correlated with hypomethylation of CpG islands in the regulation regions of cebpa gene in Dnmt1 deficient HSPCs.</b> <b>Overexpression of a transcriptional repressive SUMO-C/ebpa fusion protein could rescue hematological defects in the dnmt1 mutants.</b> <b>Finally, dnmt1 and cebpa double null embryos exhibited no obvious abnormal hematopoiesis indicated that the HSPC defects triggered by dnmt1 mutation were C/ebpa dependent.Dnmt1 is required for HSPC maintenance via cebpa regulation during definitive hematopoiesis in zebrafish.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25886188" target="_blank">
                    Methylation of RASSF1A gene promoter and the correlation with DNMT1 expression that may contribute to esophageal squamous cell carcinoma.
                </a>
            </h2>
            <p>Esophageal squamous cell carcinoma is one of the most common malignancies in the world. <b>Studies have confirmed that there are many genes abnormally hypermethylated in esophageal squamous cell carcinoma.</b> <b>The objective is to detect methylation of the RASSF1A gene promoter and the expression of the DNA methyltransferase 1 (DNMT1) protein in esophageal cancer tissue and discuss their relationship with esophageal squamous cell carcinoma.The CpG island methylation status of RASSF1A genes were analyzed in 100 cases of tumor specimens as well as their adjacent tissues which was used for methylation-specific polymerase chain reaction (MSP).</b> The expression of DNMT1 protein was determined by immunohistochemistry. Difference between measurement data and categorical data was compared through analysis of t test and chi-square test. All the statistics were taken with a bilateral test. <b>The difference was statistically significant (P &lt; 0.05).The promoter methylation of the RASSF1A gene promoter has been detected in 45 out of 100 (45%) esophageal squamous carcinoma cases, while methylation of RASSF1A gene has been detected in 2 out of 100 adjacent normal tissues (2%).</b> <b>The RASSF1A gene promoter was highly methylated in cancer tissues, and there were significant differences between normal esophagus tissues and esophageal squamous carcinoma (P &lt; 0.05).</b> <b>The expression of DNMT1 protein has been detected in 61 out of 100 (61%) esophageal squamous carcinoma cases, including 41 cases in the above 45 methylated samples of RASSF1A gene promoter, and none in adjacent tissues.</b> DNMT1 proteins are highly expressed in cancer tissues, and there were significant differences (P &lt; 0.05). In positive cases for methylation of RASSF1A, the DNMT1 protein had been detected in 41 out of 45 (91%), while in non-methylated cancer cases, 20 out of 55(36.3%), and the difference is significant (P &lt; 0.05).Esophageal squamous carcinoma tumorigenesis may be related with hypermethylation of DNMT1 and RASSF1A promoter CpG island due to their high expression and also their hypermethylation.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25886061" target="_blank">
                    ABT-888 enhances cytotoxic effects of temozolomide independent of MGMT status in serum free cultured glioma cells.
                </a>
            </h2>
            <p>The current standard of care for Glioblastoma Multiforme (GBM) consists of fractionated focal irradiation with concomitant temozolomide (TMZ) chemotherapy. A promising strategy to increase the efficacy of TMZ is through interference with the DNA damage repair machinery, by poly(ADP-ribose) polymerase protein inhibition(PARPi). The objective of the present study was to investigate the therapeutic benefit of combination therapy in patient-derived glioma stem-like cells (GSC).Combination therapy feasibility was tested on established GBM cell lines U373 and T98. We developed an in vitro drug-screening assay based on GSC cultures derived from a panel of primary patient tissue samples (n = 20) to evaluate the effect of PARPi (ABT-888) monotherapy and combination therapy with TMZ. Therapeutic effect was assessed by viability, double stranded breaks, apoptosis and autophagy assays and longitudinal microscopic cell monitoring was performed. O-6-methylguanine-DNA methyltransferase (MGMT) status was determined by methylation assay and protein expression by western blots.PARPi monotherapy was found to decrease viability by more than 25% in 4 of the 20 GSCs (20%) at 10 μM. <b>TMZ monotherapy at 50 μM and 100 μM was effective in 12 and 14 of the 20 GSCs, respectively.</b> <b>TMZ resistance to 100 μM was found in 7 of 8 MGMT protein positive cultures.</b> <b>Potentiation of TMZ therapy through PARPi was found in 90% (n = 20) of GSCs, of which 6 were initially resistant and 7 were sensitive to TMZ monotherapy.</b> Increased induction of double stranded breaks and apoptosis were noted in responsive GSCs. <b>There was a trend noted, albeit statistically insignificant, of increased autophagy both in western blots and accumulation of autophagosomes.PARPi mediated potentiation of TMZ is independent of TMZ sensitivity and can override MGMT(-) mediated resistance when administered simultaneously.</b> Response to combination therapy was associated with increased double strand breaks induction, and coincided by increased apoptosis and autophagy. <b>PARPi addition potentiates TMZ treatment in primary GSCs.</b> PARPi could potentially enhance the therapeutic efficacy of the standard of care in GBM.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25885555" target="_blank">
                    Methylation-dependent SOX9 expression mediates invasion in human melanoma cells and is a negative prognostic factor in advanced melanoma.
                </a>
            </h2>
            <p>Melanoma is the most fatal skin cancer displaying a high degree of molecular heterogeneity. Phenotype switching is a mechanism that contributes to melanoma heterogeneity by altering transcription profiles for the transition between states of proliferation/differentiation and invasion/stemness. As phenotype switching is reversible, epigenetic mechanisms, like DNA methylation, could contribute to the changes in gene expression.Integrative analysis of methylation and gene expression datasets of five proliferative and five invasion melanoma cell cultures reveal two distinct clusters. SOX9 is methylated and lowly expressed in the highly proliferative group. SOX9 overexpression results in decreased proliferation but increased invasion in vitro. In a B16 mouse model, sox9 overexpression increases the number of lung metastases. Transcriptional analysis of SOX9-overexpressing melanoma cells reveals enrichment in epithelial to mesenchymal transition (EMT) pathways. Survival analysis of The Cancer Genome Atlas melanoma dataset shows that metastatic patients with high expression levels of SOX9 have significantly worse survival rates. <b>Additional survival analysis on the targets of SOX9 reveals that most SOX9 downregulated genes have survival benefit for metastatic patients.Our genome-wide DNA methylation and gene expression study of 10 early passage melanoma cell cultures reveals two phenotypically distinct groups.</b> <b>One of the genes regulated by DNA methylation between the two groups is SOX9.</b> <b>SOX9 induces melanoma cell invasion and metastasis and decreases patient survival.</b> <b>A number of genes downregulated by SOX9 have a negative impact on patient survival.</b> <b>In conclusion, SOX9 is an important gene involved in melanoma invasion and negatively impacts melanoma patient survival.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25885113" target="_blank">
                    AFLP and MS-AFLP Analysis of the Variation within Saffron Crocus (Crocus sativus L.) Germplasm.
                </a>
            </h2>
            <p>The presence and extent of genetic variation in saffron crocus are still debated, as testified by several contradictory articles providing contrasting results about the monomorphism or less of the species. Remarkably, phenotypic variations have been frequently observed in the field, such variations are usually unstable and can change from one growing season to another. Considering that gene expression can be influenced both by genetic and epigenetic changes, epigenetics could be a plausible cause of the alternative phenotypes. <b>In order to obtain new insights into this issue, we carried out a molecular marker analysis of 112 accessions from the World Saffron and Crocus Collection.</b> <b>The accessions were grown for at least three years in the same open field conditions.</b> The same samples were analysed using Amplified Fragment Length Polymorphism (AFLP) and Methyl Sensitive AFLP in order to search for variation at the genetic (DNA sequence) and epigenetic (cytosine methylation) level. While the genetic variability was low (4.23% polymorphic peaks and twelve (12) effective different genotypes), the methyl sensitive analysis showed the presence of high epigenetic variability (33.57% polymorphic peaks and twenty eight (28) different effective epigenotypes). <b>The pattern obtained by Factorial Correspondence Analysis of AFLP and, in particular, of MS-AFLP data was consistent with the geographical provenance of the accessions.</b> <b>Very interestingly, by focusing on Spanish accessions, it was observed that the distribution of the accessions in the Factorial Correspondence Analysis is not random but tends to reflect the geographical origin.</b> <b>Two clearly defined clusters grouping accessions from the West (Toledo and Ciudad Real) and accessions from the East (Cuenca and Teruel) were clearly recognised.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25884154" target="_blank">
                    DNA methylation of the LIN28 pseudogene family.
                </a>
            </h2>
            <p>DNA methylation directs the epigenetic silencing of selected regions of DNA, including the regulation of pseudogenes, and is widespread throughout the genome. Pseudogenes are decayed copies of duplicated genes that have spread throughout the genome by transposition. Pseudogenes are transcriptionally silenced by DNA methylation, but little is known about how pseudogenes are targeted for methylation or how methylation levels are maintained in different tissues.We employed bisulfite next generation sequencing to examine the methylation status of the LIN28 gene and four processed pseudogenes derived from LIN28. <b>The objective was to determine whether LIN28 pseudogenes maintain the same pattern of methylation as the parental gene or acquire a methylation pattern independent of the gene of origin.</b> <b>In this study, we determined that the methylation status of LIN28 pseudogenes does not resemble the pattern evident for the LIN28 gene, but rather these pseudogenes appear to acquire methylation patterns independent of the parental gene.</b> <b>Furthermore, we observed that methylation levels of the examined pseudogenes correlate to the location of insertion within the genome.</b> <b>LIN28 pseudogenes inserted into gene bodies were highly methylated in all tissues examined.</b> <b>In contrast, pseudogenes inserted into genomic regions that are not proximal to genes were differentially methylated in various tissue types.Our analysis suggests that Lin28 pseudogenes do not aquire patterns of tissue-specific methylation as for the parental gene, but rather are methylated in patterns specific to the local genomic environment into which they were inserted.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25882495" target="_blank">
                    Absence of keratins 8 and 18 expression in rodent epithelial cell lines associates with keratin gene mutation and DNA methylation: Cell line selective effects on cell invasion.
                </a>
            </h2>
            <p>Epithelial-mesenchymal transition (EMT) in carcinoma is associated with dramatic up-regulation of vimentin and down-regulation of the simple-type keratins 8 and 18 (K8/K18), but the mechanisms of these changes are poorly understood. We demonstrate that two commonly-studied murine (CT26) and rat (IEC-6) intestinal cell lines have negligible K8/K18 but high vimentin protein expression. <b>Proteasome inhibition led to a limited increase in K18 but not K8 stabilization, thereby indicating that K8/K18 absence is not due, in large part, to increased protein turnover.</b> <b>CT26 and IEC-6 cells had &lt;10% of normal K8/K18 mRNA and exhibited decreased mRNA stability, with K8 being higher in IEC-6 versus CT26 and K18 being higher in CT26 versus IEC-6 cells.</b> <b>Keratin gene sequencing showed that KRT8 in CT26 cells had a 21-nucleotide deletion while K18 in IEC-6 cells had a 9-amino acid in-frame insertion.</b> Furthermore, the KRT8 promoter in CT26 and the KRT18 promoter in IEC-6 are hypermethylated. <b>Inhibition of DNA methylation using 5-azacytidine increased K8 or K18 in some but all the tested rodent epithelial cell lines.</b> <b>Restoring K8 and K18 by lentiviral transduction reduced CT26 but not IEC-6 cell matrigel invasion.</b> K8/K18 re-introduction also decreased E-cadherin expression in IEC-6 but not CT26 cells, suggesting that the effect of keratin expression on epithelial to mesenchymal transition is cell-line dependent. Therefore, some commonly utilized rodent epithelial cell lines, unexpectedly, manifest barely detectable keratin expression but have high levels of vimentin. In the CT26 and IEC-6 intestinal cell lines, keratin expression correlates with keratin gene insertion or deletion and with promoter methylation, which likely suppress keratin transcription or mRNA stability.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25882215" target="_blank">
                    Epigenetics in Cystic Fibrosis: Epigenetic Targeting of a Genetic Disease.
                </a>
            </h2>
            <p>Cystic fibrosis (CF) is a deadly genetic disease that affects the lungs and digestive system. A mutation in the CF transmembrane conductance regulator (CFTR) gene is the cause of the disease. How epigenetics contributes to CFTR expression is still poorly understood. Epigenetics is a mechanism that alters gene expression without changing the underlying DNA sequence. <b>Epigenetic mechanisms include DNA methylation and histone modification.</b> Both mechanisms have been implicated in CFTR gene regulation. <b>Here we review epigenetic regulation of CFTR transcription while discussing potential epigenetic targeting strategies including DNA methyltransferase, histone deacetylase, and histone methyltransferase and demethylase inhibition.</b> <b>Because of the reversibility of epigenetics, targeting epigenetic mechanisms has been an attractive therapeutic approach.</b> <b>However, epigenetic targeting of CF disease is still at its infant stage.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25881900" target="_blank">
                    MIRA-seq for DNA methylation analysis of CpG islands.
                </a>
            </h2>
            <p><b>To develop a reliable method for whole genome analysis of DNA methylation.Genome-scale analysis of DNA methylation includes affinity-based approaches such as enrichment using methyl-CpG-binding proteins.</b> <b>One of these methods, the methylated-CpG island recovery assay (MIRA), is based on the high affinity of the MBD2b-MBD3L1 complex for CpG-methylated DNA.</b> <b>Here we provide a detailed description of MIRA and combine it with next generation sequencing platforms (MIRA-seq).We assessed the performance of MIRA-seq and compared the data with whole genome bisulfite sequencing.MIRA-seq is a reliable, genome-scale DNA methylation analysis platform for scoring DNA methylation differences at CpG-rich genomic regions.</b> The method is not limited by primer or probe design and is cost effective.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25881700" target="_blank">
                    Long non-coding RNAs in plants.
                </a>
            </h2>
            <p><b>Long non-coding RNAs (lncRNAs), which are longer than 200 nucleotides in length, widely exist in organisms and function in a variety of biological processes.</b> <b>Currently, most of lncRNAs found in plants are transcribed by RNA polymerase Ⅱ and mediate gene expression through multiple mechanisms, such as target mimicry, transcription interference, histone methylation and DNA methylation, and play important roles in flowering, male sterility, nutrition metabolism, biotic and abiotic stress and other biological processes as regulators in plants.</b> <b>In this review, we summarize the databases, prediction methods, and possible functions of plant lncRNAs discovered in recent years.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25881205" target="_blank">
                    Endopolyploidy Changes with Age-Related Polyethism in the Honey Bee, Apis mellifera.
                </a>
            </h2>
            <p>Honey bees (Apis mellifera) exhibit age polyethism, whereby female workers assume increasingly complex colony tasks as they age. While changes in DNA methylation accompany age polyethism, other DNA modifications accompanying age polyethism are less known. Changes in endopolyploidy (DNA amplification in the absence of cell division) with increased larval age are typical in many insect cells and are essential in adults for creating larger cells, more copies of essential loci, or greater storage capacity in secretory cells. <b>However, changes in endopolyploidy with increased adult worker age and polyethism are unstudied.</b> <b>In this study, we examined endopolyploidy in honey bee workers ranging in age from newly emerged up to 55 days old.</b> <b>We found a nonsignificant increase in ploidy levels with age (P &lt; 0.1) in the most highly endopolyploid secretory cells, the Malpighian tubules.</b> <b>All other cell types decreased ploidy levels with age.</b> Endopolyploidy decreased the least amount (nonsignificant) in neural (brain) cells and the stinger (P &lt; 0.1). <b>There was a significant reduction of endopolyploidy with age in leg (P &lt; 0.05) and thoracic (P &lt; 0.001) muscles.</b> Ploidy in thoracic muscle dropped from an average of 0.5 rounds of replication in newly emerged workers to essentially no rounds of replication (0.125) in the oldest workers. Ploidy reduction in flight muscle cells is likely due to the production of G1 (2C) nuclei by amitotic division in the multinucleate striated flight muscles that are essential to foragers, the oldest workers. We suggest that ploidy is constrained by the shape, size and makeup of the multinucleate striated muscle cells. Furthermore, the presence of multiple 2C nuclei might be optimal for cell function, while higher ploidy levels might be a dead-end strategy of some aging adult tissues, likely used to increase cell size and storage capacity in secretory cells.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25881176" target="_blank">
                    DNA methylation dynamics at imprinted genes during bovine pre-implantation embryo development.
                </a>
            </h2>
            <p>In mammals, maternal differentially methylated regions (DMRs) acquire DNA methylation during the postnatal growth stage of oogenesis, with paternal DMRs acquiring DNA methylation in the perinatal prospermatagonia. <b>Following fusion of the male and female gametes, it is widely accepted that murine DNA methylation marks at the DMRs of imprinted genes are stable through embryogenesis and early development, until they are reprogrammed in primordial germ cells.</b> <b>However, the DNA methylation dynamics at DMRs of bovine imprinted genes during early stages of development remains largely unknown.</b> <b>The objective of this investigation was to analyse the methylation dynamics at imprinted gene DMRs during bovine embryo development, from blastocyst stage until implantation.To this end, pyrosequencing technology was used to quantify DNA methylation at DMR-associated CpG dinucleotides of six imprinted bovine genes (SNRPN, MEST, IGF2R, PLAGL1, PEG10 and H19) using bisulfite-modified genomic DNA isolated from individual blastocysts (Day 7); ovoid embryos (Day 14); filamentous embryos (Day 17) and implanting conceptuses (Day 25).</b> <b>For all genes, the degree of DNA methylation was most variable in Day 7 blastocysts compared to later developmental stages (P &lt; 0.05).</b> <b>Furthermore, mining of RNA-seq transcriptomic data and western blot analysis revealed a specific window of expression of DNA methylation machinery genes (including DNMT3A, DNMT3B, TRIM28/KAP1 and DNMT1) and proteins (DNMT3A, DNMT3A2 and DNMT3B) by bovine embryos coincident with imprint stabilization.The findings of this study suggest that the DNA methylation status of bovine DMRs might be variable during the early stages of embryonic development, possibly requiring an active period of imprint stabilization.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25881171" target="_blank">
                    The DNA methylation drift of the atherosclerotic aorta increases with lesion progression.
                </a>
            </h2>
            <p><b>Atherosclerosis severity-independent alterations in DNA methylation, a reversible and highly regulated DNA modification, have been detected in aortic atheromas, thus supporting the hypothesis that epigenetic mechanisms participate in the pathogenesis of atherosclerosis.</b> <b>One yet unaddressed issue is whether the progression of atherosclerosis is associated with an increase in DNA methylation drift in the vascular tissue.</b> <b>The purpose of the study was to identify CpG methylation profiles that vary with the progression of atherosclerosis in the human aorta.We interrogated a set of donor-matched atherosclerotic and normal aortic samples ranging from histological grade III to VII, with a high-density (&gt;450,000 CpG sites) DNA methylation microarray.We detected a correlation between histological grade and intra-pair differential methylation for 1,985 autosomal CpGs, the vast majority of which drifted towards hypermethylation with lesion progression.</b> The identified CpG loci map to genes that are regulated by known critical transcription factors involved in atherosclerosis and participate in inflammatory and immune responses. <b>Functional relevance was corroborated by crossing the DNA methylation profiles with expression data obtained in the same human aorta sample set, by a transcriptome-wide analysis of murine atherosclerotic aortas and from available public databases.Our work identifies for the first time atherosclerosis progression-specific DNA methylation profiles in the vascular tissue.</b> These findings provide potential novel markers of lesion severity and targets to counteract the progression of the atheroma.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25881116" target="_blank">
                    Hypermethylation of BDNF and SST Genes in the Orbital Frontal Cortex of Older Individuals: A Putative Mechanism for Declining Gene Expression with Age.
                </a>
            </h2>
            <p>Expression of brain-derived neurotrophic factor (BDNF) and somatostatin (SST) mRNAs in the brain decreases progressively and robustly with age, and lower BDNF and SST expression in the brain has been observed in many brain disorders. <b>BDNF is known to regulate SST expression; however, the mechanisms underlying decreased expression of both genes are not understood.</b> DNA methylation (DNAm) is an attractive candidate mechanism. <b>To investigate the contribution of DNAm to the age-related decline in BDNF and SST expression, the Illumina Infinium HumanMethylation450 Beadchip Array was used to quantify DNAm of BDNF (26 CpG loci) and SST (9 CpG loci) in the orbital frontal cortices of postmortem brains from 22 younger (age&lt;42 years) and 22 older individuals (age&gt;60 years) with known age-dependent BDNF and SST expression differences.</b> <b>Relative to the younger individuals, 10 of 26 CpG loci in BDNF and 8 of 9 CpG loci in SST were significantly hypermethylated in the older individuals.</b> <b>DNAm in BDNF exons/promoters I, II, &amp; IV negatively correlated with BDNF expression (r=-0.37, p&lt;0.05; r=-0.40, p&lt;0.05; r=-0.24, p=0.07) and DNAm in SST 5' UTR &amp; 1st exon/intron negatively correlated with SST expression (r=-0.48, p&lt;0.01; r=-0.63, p&lt;0.001), respectively.</b> An expanded set of BDNF- and GABA-related genes exhibited similar age-related changes in DNAm and correlation with gene expression. <b>These results suggest that DNAm may be a proximal mechanism for decreased expression of BDNF, SST, and other BDNF- and GABA-related genes with brain aging and, by extension, for brain disorders in which their expression is decreased.Neuropsychopharmacology accepted article preview online, 16 April 2015. doi:10.1038/npp.2015.107.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25880792" target="_blank">
                    Polycomb Repressive Complex 2 epigenomic signature defines age-associated hypermethylation and gene expression changes.
                </a>
            </h2>
            <p>Although age-associated gene expression and methylation changes have been reported throughout the literature, the unifying epigenomic principles of aging remain poorly understood. Recent explosion in availability and resolution of functional/regulatory genome annotation data (epigenomic data), such as that provided by the ENCODE and Roadmap Epigenomics projects, provides an opportunity for the identification of epigenomic mechanisms potentially altered by age-associated differentially methylated regions (aDMRs) and to find regulatory signatures in the promoters of age-associated genes (aGENs). In this study we found that aDMRs and aGENs identified in multiple independent studies share a common Polycomb Repressive Complex 2&nbsp;signature marked by EZH2, SUZ12, CTCF binding sites, repressive H3K27me3, and activating H3K4me1 histone modification marks, and a "poised promoter" chromatin state. This signature is depleted in RNAP II-associated transcription factor binding sites, activating H3K79me2, H3K36me3, H3K27ac marks, and an "active promoter" chromatin state. The PRC2 signature was shown to be generally stable across cell types. <b>When considering the directionality of methylation changes, we found the PRC2 signature to be associated with aDMRs hypermethylated with age, while hypomethylated aDMRs were associated with enhancers.</b> <b>In contrast, aGENs were associated with the PRC2 signature independently of the directionality of gene expression changes.</b> <b>In this study we demonstrate that the PRC2 signature is the common epigenomic context of genomic regions associated with hypermethylation and gene expression changes in aging.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25880665" target="_blank">
                    Genome wide comparative analysis of the effects of PRMT5 and PRMT4/CARM1 arginine methyltransferases on the Arabidopsis thaliana transcriptome.
                </a>
            </h2>
            <p>Methylation at arginine residues (R) is an important post-translational modification that regulates a myriad of essential cellular processes in eukaryotes, such as transcriptional regulation, RNA processing, signal transduction and DNA repair. Arginine methylation is catalyzed by a family of enzymes known as protein arginine methyltransferases (PRMTs). PRMTs are classified as Type I or Type II, depending on the position of the methyl group on the guanidine of the methylated arginine. Previous reports have linked symmetric R methylation to transcriptional repression, while asymmetric R methylation is generally associated with transcriptional activation. <b>However, global studies supporting this conclusion are not available.Here we compared side by side the physiological and molecular roles of the best characterized plant PRMTs, the Type II PRMT5 and the Type I PRMT4, also known as CARM1 in mammals.</b> <b>We found that prmt5 and prmt4a;4b mutants showed similar alterations in flowering time, photomorphogenic responses and salt stress tolerance, while only prmt5 mutants exhibited alterations in circadian rhythms.</b> <b>An RNA-seq analysis revealed that expression and splicing of many differentially regulated genes was similarly enhanced or repressed by PRMT5 and PRMT4s.</b> <b>Furthermore, PRMT5 and PRMT4s co-regulated the expression and splicing of key regulatory genes associated with transcription, RNA processing, responses to light, flowering, and abiotic stress tolerance, being candidates to mediate the physiological alterations observed in the mutants.Our global analysis indicates that two of the most important Type I and Type II arginine methyltransferases, PRTM4 and PRMT5, have mostly overlapping as well as specific, but not opposite, roles in the global regulation of gene expression in plants.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25880212" target="_blank">
                    DNA methylation and histone modifications regulate SOX11 expression in lymphoid and solid cancer cells.
                </a>
            </h2>
            <p>The neural transcription factor SOX11 is present at specific stages during embryo development with a very restricted expression in adult tissue, indicating precise regulation of transcription. SOX11 is strongly up-regulated in some malignancies and have a functional role in tumorgenesis. With the aim to explore differences in epigenetic regulation of SOX11 expression in normal versus neoplastic cells, we investigated methylation and histone modifications related to the SOX11 promoter and the possibility to induce re-expression using histone deacetylase (HDAC) or EZH2 inhibitors.The epigenetic regulation of SOX11 was investigated in distinct non-malignant cell populations (n = 7) and neoplastic cell-lines (n = 42) of different cellular origins. DNA methylation was assessed using bisulfite sequencing, methylation-specific melting curve analysis, MethyLight and pyrosequencing. The presence of H3K27me3 was assessed using ChIP-qPCR. The HDAC inhibitors Vorinostat and trichostatin A were used to induce SOX11 in cell lines with no endogenous expression.The SOX11 promoter shows a low degree of methylation and strong enrichment of H3K27me3 in non-malignant differentiated cells, independent of cellular origin. Cancers of the B-cell lineage are strongly marked by de novo methylation at the SOX11 promoter in SOX11 non-expressing cells, while solid cancer entities display a more varying degree of SOX11 promoter methylation. <b>The silencing mark H3K27me3 was generally present at the SOX11 promoter in non-expressing cells, and an increased enrichment was observed in cancer cells with a low degree of SOX11 methylation compared to cells with dense methylation.</b> <b>Finally, we demonstrate that the HDAC inhibitors (vorinostat and trichostatin A) induce SOX11 expression in cancer cells with low levels of SOX11 methylation.We show that SOX11 is strongly marked by repressive histone marks in non-malignant cells.</b> <b>In contrast, SOX11 regulation in neoplastic tissues is more complex involving both DNA methylation and histone modifications.</b> <b>The possibility to re-express SOX11 in non-methylated tissue is of clinical relevance, and was successfully achieved in cell lines with low levels of SOX11 methylation.</b> <b>In breast cancer patients, methylation of the SOX11 promoter was shown to correlate with estrogen receptor status, suggesting that SOX11 may be functionally re-expressed during treatment with HDAC inhibitors in specific patient subgroups.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25879881" target="_blank">
                    An automated method for efficient, accurate and reproducible construction of RNA-seq libraries.
                </a>
            </h2>
            <p>Integration of RNA-seq expression data with knowledge on chromatin accessibility, histone modifications, DNA methylation, and transcription factor binding has been instrumental for the unveiling of cell-specific local and long-range regulatory patterns, facilitating further investigation on the underlying rules of transcription regulation at an individual and allele-specific level. However, full genome transcriptome characterization has been partially limited by the complexity and increased time-requirements of available RNA-seq library construction protocols.Use of the SX-8G IP-Star® Compact System significantly reduces the hands-on time for RNA-seq library synthesis, adenylation, and adaptor ligation providing with high quality RNA-seq libraries tailored for Illumina high-throughput next-generation sequencing. Generated data exhibits high technical reproducibility compared to data from RNA-seq libraries synthesized manually for the same samples. Obtained results are consistent regardless the researcher, day of the experiment, and experimental run.Overall, the SX-8G IP-Star® Compact System proves an efficient, fast and reliable tool for the construction of next-generation RNA-seq libraries especially for trancriptome-based annotation of larger genomes.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25879481" target="_blank">
                    Factors to preserve CpG-rich sequences in methylated CpG islands.
                </a>
            </h2>
            <p>Mammalian CpG islands (CGIs) normally escape DNA methylation in all adult tissues and developmental stages. However, in our previous study we unexpectedly identified many methylated CGIs in human peripheral blood leukocytes. Methylated CpG dinucleotides convert to TpG dinucleotides through deaminization of their cytosine bases more frequently than hypomethylated CpG dinucleotides. Therefore, we wondered how methylated CGIs in germline or non-germline cells maintain their CpG-rich sequences. <b>It is known that events such as germline hypomethylation, CpG selection, biased gene conversion (BGC), and frequent CpG fixation can contribute to the maintenance of CpG-rich sequences in methylated CGIs in germline or non-germline cells.</b> However, it has not been investigated which of the processes maintain CpG-rich sequences of methylated CGIs in each genomic position.In this study, we comprehensively examined the contribution of the processes described above to the maintenance of CpG-rich sequences in methylated CGIs in germline and non-germline cells which were classified by genomic positions. <b>Approximately 60-80% of CGIs with high methylation in H1 cell line (H1-HM) in all the genomic positions showed a low average CpG → TpG/CpA substitution rate.</b> <b>In contrast, fewer than half the numbers of CGIs with H1-HM in all the genomic positions showed a low average CpG → TpG/CpA substitution rate and low levels of methylation in sperm cells (SPM-LM).</b> <b>Furthermore, a small fraction of CGIs with a low average CpG → TpG/CpA substitution rate and high levels of methylation in sperm cells (SPM-HM) showed CpG selection.</b> <b>On the other hand, independent of the positions in genes, most CGIs with SPM-HM showed a slightly higher average TpG/CpA → CpG substitution rate compared with those with SPM-LM.Relatively high numbers (approximately 60-80%) of CGIs with H1-HM in all the genomic positions preserve their CpG-rich sequences by a low CpG → TpG/CpA substitution rate caused mainly by their SPM-LM, and for those with SPM-HM partly by CpG selection and TpG/CpA → CpG fixation.</b> BGC has little contribution to the maintenance of CpG-rich sequences of CGIs with SPM-HM which were classified by genomic positions.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25878287" target="_blank">
                    DNA methylation in the medial prefrontal cortex regulates alcohol-induced behavior and plasticity.
                </a>
            </h2>
            <p>Recent studies have suggested an association between alcoholism and DNA methylation, a mechanism that can mediate long-lasting changes in gene transcription. Here, we examined the contribution of DNA methylation to the long-term behavioral and molecular changes induced by a history of alcohol dependence. <b>In search of mechanisms underlying persistent rather than acute dependence-induced neuroadaptations, we studied the role of DNA methylation regulating medial prefrontal cortex (mPFC) gene expression and alcohol-related behaviors in rats 3 weeks into abstinence following alcohol dependence.</b> <b>Postdependent rats showed escalated alcohol intake, which was associated with increased DNA methylation as well as decreased expression of genes encoding synaptic proteins involved in neurotransmitter release in the mPFC.</b> <b>Infusion of the DNA methyltransferase inhibitor RG108 prevented both escalation of alcohol consumption and dependence-induced downregulation of 4 of the 7 transcripts modified in postdependent rats.</b> Specifically, RG108 treatment directly reversed both downregulation of synaptotagmin 2 (Syt2) gene expression and hypermethylation on CpG#5 of its first exon. <b>Lentiviral inhibition of Syt2 expression in the mPFC increased aversion-resistant alcohol drinking, supporting a mechanistic role of Syt2 in compulsive-like behavior.</b> <b>Our findings identified a functional role of DNA methylation in alcohol dependence-like behavioral phenotypes and a candidate gene network that may mediate its effects.</b> Together, these data provide novel evidence for DNA methyltransferases as potential therapeutic targets in alcoholism.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25878217" target="_blank">
                    Paternal sperm DNA methylation associated with early signs of autism risk in an autism-enriched cohort.
                </a>
            </h2>
            <p><b>Epigenetic mechanisms such as altered DNA methylation have been suggested to play a role in autism, beginning with the classical association of Prader-Willi syndrome, an imprinting disorder, with autistic features.Here we tested for the relationship of paternal sperm DNA methylation with autism risk in offspring, examining an enriched-risk cohort of fathers of autistic children.We examined genome-wide DNA methylation (DNAm) in paternal semen biosamples obtained from an autism spectrum disorder (ASD) enriched-risk pregnancy cohort, the Early Autism Risk Longitudinal Investigation (EARLI) cohort, to estimate associations between sperm DNAm and prospective ASD development, using a 12-month ASD symptoms assessment, the Autism Observation Scale for Infants (AOSI).</b> We analysed methylation data from 44 sperm samples run on the CHARM 3.0 array, which contains over 4 million probes (over 7 million CpG sites), including 30 samples also run on the Illumina Infinium HumanMethylation450 (450K) BeadChip platform (∼485&nbsp;000 CpG sites). <b>We also examined associated regions in an independent sample of post-mortem human brain ASD and control samples for which Illumina 450K DNA methylation data were available.Using region-based statistical approaches, we identified 193 differentially methylated regions (DMRs) in paternal sperm with a family-wise empirical P-value [family-wise error rate (FWER)] &lt;0.05 associated with performance on the Autism Observational Scale for Infants (AOSI) at 12 months of age in offspring.</b> The DMRs clustered near genes involved in developmental processes, including many genes in the SNORD family, within the Prader-Willi syndrome gene cluster. These results were consistent among the 75 probes on the Illumina 450K array that cover AOSI-associated DMRs from CHARM. <b>Further, 18 of 75 (24%) 450K array probes showed consistent differences in the cerebellums of autistic individuals compared with controls.These data suggest that epigenetic differences in paternal sperm may contribute to autism risk in offspring, and provide evidence that directionally consistent, potentially related epigenetic mechanisms may be operating in the cerebellum of individuals with autism.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25878004" target="_blank">
                    The epigenetics of multiple sclerosis and other related disorders.
                </a>
            </h2>
            <p><b>Multiple Sclerosis (MS) is a demyelinating disease characterized by chronic inflammation of the central nervous system (CNS) gray and white matter.</b> <b>Although the cause of MS is unknown, it is widely appreciated that innate and adaptive immune processes contribute to its pathogenesis.</b> These include microglia/macrophage activation, pro-inflammatory T-cell (Th1) responses and humoral responses. <b>Additionally, there is evidence indicating that MS has a neurodegenerative component since neuronal and axonal loss occurs even in the absence of overt inflammation.</b> These aspects also form the rationale for clinical management of the disease. However, the currently available therapies to control the disease are only partially effective at best indicating that more effective therapeutic solutions are urgently needed. It is appreciated that in the immune-driven and neurodegenerative processes MS-specific deregulation of gene expressions and resulting protein dysfunction are thought to play a central role. These deviations in gene expression patterns contribute to the inflammatory response in the CNS, and to neuronal or axonal loss. Epigenetic mechanisms control transcription of most, if not all genes, in nucleated cells including cells of the CNS and in haematopoietic cells. MS-specific alterations in epigenetic regulation of gene expression may therefore lie at the heart of the deregulation of gene expression in MS. As such, epigenetic mechanisms most likely play an important role in disease pathogenesis. In this review we discuss a role for MS-specific deregulation of epigenetic features that control gene expression in the CNS and in the periphery. <b>Furthermore, we discuss the application of small molecule inhibitors that target the epigenetic machinery to ameliorate disease in experimental animal models, indicating that such approaches may be applicable to MS patients.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25875827" target="_blank">
                    Differentially Expressed MicroRNAs in Postpartum Breast Cancer in Hispanic Women.
                </a>
            </h2>
            <p>The risk of breast cancer transiently increases immediately following pregnancy; peaking between 3-7 years. The biology that underlies this risk window and the effect on the natural history of the disease is unknown. MicroRNAs (miRNAs) are small non-coding RNAs that have been shown to be dysregulated in breast cancer. We conducted miRNA profiling of 56 tumors from a case series of multiparous Hispanic women and assessed the pattern of expression by time since last full-term pregnancy. A data-driven splitting analysis on the pattern of 355 miRNAs separated the case series into two groups: a) an early group representing women diagnosed with breast cancer ≤ 5.2 years postpartum (n = 12), and b) a late group representing women diagnosed with breast cancer ≥ 5.3 years postpartum (n = 44). We identified 15 miRNAs with significant differential expression between the early and late postpartum groups; 60% of these miRNAs are encoded on the X chromosome. Ten miRNAs had a two-fold or higher difference in expression with miR-138, miR-660, miR-31, miR-135b, miR-17, miR-454, and miR-934 overexpressed in the early versus the late group; while miR-892a, miR-199a-5p, and miR-542-5p were underexpressed in the early versus the late postpartum group. The DNA methylation of three out of five tested miRNAs (miR-31, miR-135b, and miR-138) was lower in the early versus late postpartum group, and negatively correlated with miRNA expression. Here we show that miRNAs are differentially expressed and differentially methylated between tumors of the early versus late postpartum, suggesting that potential differences in epigenetic dysfunction may be operative in postpartum breast cancers.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25875376" target="_blank">
                    Placental DNA methylation of peroxisome proliferator-activated receptor-gamma coactivator-1 alpha promoter is associated with maternal gestational glucose level.
                </a>
            </h2>
            <p>Intrauterine exposure to hyperglycemia may increase the risk for later-life metabolic disorders. Although the underlying mechanism is not fully understood, epigenetic dysregulationin fetal programminghas been implicated. <b>Apropos to energy homeostasis, peroxisome proliferator-activated receptor gamma coactivator-1 alpha (PPARGC1A) plays a regulatory role in several biochemical processes.</b> <b>We hypothesized that maternal gestational glucose levels would positively correlate with DNA methylation of PPARGC1A promoter in placental tissue.This was a cross-sectional study of 58 mothers who underwent uncomplicated Cesarean delivery in a university hospital.</b> Maternal gestational glucose concentration was determined after a 75-g OGTT at 24 -28 weeks gestation. Placenta tissue and cord blood were collected immediately post delivery. Genomic DNA was extracted and, thereafter, bisulfite conversion was performed. <b>After PCR amplification, the DNA methylation of the PPARGC1A promoter was quantified by pyrosequencing technique.</b> <b>The protein level of PPARGC1A was evaluated by Western blotting.For all participants as a whole, including the gestational diabetes mellitus (GDM) and normoglycemia groups, the maternal gestationalglucose level was positively correlated with placental DNA methylation, while negatively correlated with cord blood DNA methylation of PPARGC1A promoter in a CpG site-specific manner.</b> <b>In the GDM group alone, the placental CpG site-specific methylation of PPARGC1A promoter strongly correlated with gestational 2h post-OGTT glycemia.Epigenetic alteration in PPAGRC1A promoter may be one of the potential mechanisms underlying the metabolic programming in offspring exposed to intrauterine hyperglycemia.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25875334" target="_blank">
                    Maternal prenatal depressive symptoms predict infant NR3C1 1F and BDNF IV DNA methylation.
                </a>
            </h2>
            <p>Prenatal maternal psychological distress increases risk for adverse infant outcomes. However, the biological mechanisms underlying this association remain unclear. Prenatal stress can impact fetal epigenetic regulation that could underlie changes in infant stress responses. It has been suggested that maternal glucocorticoids may mediate this epigenetic effect. We examined this hypothesis by determining the impact of maternal cortisol and depressive symptoms during pregnancy on infant NR3C1 and BDNF DNA methylation. Fifty-seven pregnant women were recruited during the second or third trimester. Participants self-reported depressive symptoms and salivary cortisol samples were collected diurnally and in response to a stressor. Buccal swabs for DNA extraction and DNA methylation analysis were collected from each infant at two months of age, and mothers were assessed for postnatal depressive symptoms. <b>Prenatal depressive symptoms significantly predicted increased NR3C1 1F DNA methylation in male infants (β = 2.147, P = 0.044).</b> <b>Prenatal depressive symptoms also significantly predicted decreased BDNF IV DNA methylation in both male and female infants (β = -3.244, P = 0.013).</b> <b>No measure of maternal cortisol during pregnancy predicted infant NR3C1 1F or BDNF promoter IV DNA methylation.</b> <b>Our findings highlight the susceptibility of males to changes in NR3C1 DNA methylation and present novel evidence for altered BDNF IV DNA methylation in response to maternal depression during pregnancy.</b> <b>The lack of association between maternal cortisol and infant DNA methylation suggests that effects of maternal depression may not be mediated directly by glucocorticoids.</b> Future studies should consider other potential mediating mechanisms in the link between maternal mood and infant outcomes.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25875202" target="_blank">
                    Epigenetic Modulations in Activated Cells Early after HIV-1 Infection and Their Possible Functional Consequences.
                </a>
            </h2>
            <p>Epigenetic modifications refer to a number of biological processes which alter the structure of chromatin and its transcriptional activity such as DNA methylation and histone post-translational processing. Studies have tried to elucidate how the viral genome and its products are affected by epigenetic modifications imposed by cell machinery and how it affects the ability of the virus to either, replicate and produce a viable progeny or be driven to latency. The purpose of this study was to evaluate epigenetic modifications in PBMCs and CD4+ cells after HIV-1 infection analyzing three approaches: (i) global DNA- methylation; (ii) qPCR array and (iii) western blot. HIV-1 infection led to methylation increases in the cellular DNA regardless the activation status of PBMCs. The analysis of H3K9me3 and H3K27me3 suggested a trend towards transcriptional repression in activated cells after HIV-1 infection. Using a qPCR array, we detected genes related to epigenetic processes highly modulated in activated HIV-1 infected cells. SETDB2 and RSK2 transcripts showed highest up-regulation levels. SETDB2 signaling is related to transcriptional silencing while RSK2 is related to either silencing or activation of gene expression depending on the signaling pathway triggered down-stream. In addition, activated cells infected by HIV-1 showed lower CD69 expression and a decrease of IL-2, IFN-γ and metabolism-related factors transcripts indicating a possible functional consequence towards global transcriptional repression found in HIV-1 infected cells. Conversely, based on epigenetic markers studied here, non-stimulated cells infected by HIV-1, showed signs of global transcriptional activation. Our results suggest that HIV-1 infection exerts epigenetic modulations in activated cells that may lead these cells to transcriptional repression with important functional consequences. Moreover, non-stimulated cells seem to increase gene transcription after HIV-1 infection. Based on these observations, it is possible to speculate that the outcome of viral infections may be influenced by the cellular activation status at the moment of infection.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25875118" target="_blank">
                    Considering Maternal Dietary Modulators for Epigenetic Regulation and Programming of the Fetal Epigenome.
                </a>
            </h2>
            <p>Fetal life is characterized by a tremendous plasticity and ability to respond to various environmental and lifestyle factors, including maternal nutrition. <b>Identification of the role of dietary factors that can modulate and reshape the cellular epigenome during development, including methyl group donors (e.g., folate, choline) and bioactive compounds (e.g., polyphenols) is of great importance; however, there is insufficient knowledge of a particular effect of each type of modulator and/or their combination on fetal life.</b> <b>To enhance the quality and safety of food products for proper fetal health and disease prevention in later life, a better understanding of the underlying mechanisms of dietary epigenetic modulators during the critical prenatal period is necessary.</b> <b>This review focuses on the influence of maternal dietary components on DNA methylation, histone modification, and microRNAs, and summarizes current knowledge of the effect and importance of dietary components on epigenetic mechanisms that control the proper expression of genetic information.</b> Evidence reveals that some components in the maternal diet can directly or indirectly affect epigenetic mechanisms. Understanding the underlying mechanisms of how early-life nutritional environment affects the epigenome during development is of great importance for the successful prevention of adult chronic diseases through optimal maternal nutrition.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25874772" target="_blank">
                    Deregulation between miR-29b/c and DNMT3A Is Associated with Epigenetic Silencing of the CDH1 Gene, Affecting Cell Migration and Invasion in Gastric Cancer.
                </a>
            </h2>
            <p>The de-regulation of the miR-29 family and DNA methyltransferase 3A (DNMT3A) is associated with gastric cancer (GC). While increasing evidence indicates miR-29b/c could regulate DNA methylation by targeting DNMT3A, it is currently unknown if epigenetic silencing of miR-29b/c via promoter hypermethylation in GC is caused by abnormal expression of DNMT3A. Thus, we aimed to evaluate whether cross-talk regulation exists between miR-29b/c and DNMT3A and whether it is associated with a malignant phenotype in GC. First, wound healing and Transwell assays revealed that miR-29b/c suppresses tumor metastasis in GC. A luciferase reporter assay demonstrated that DNMT3A is a direct target of miR-29b/c. We used bisulfite genomic sequencing to analyze the DNA methylation status of miR-29b/c. The percentage of methylated CpGs was significantly decreased in DNMT3A-depleted cells compared to the controls. Furthermore, the involvement of DNMT3A in promoting GC cell migration was associated with the promoter methylation-mediated repression of CDH1. In 50 paired clinical GC tissue specimens, decreased miR-29b/c was significantly correlated with the degree of differentiation and invasion of the cells and was negatively correlated with DNMT3A expression. Together, our preliminary results suggest that the following process may be involved in GC tumorigenesis. miR-29b/c suppresses the downstream gene DNMT3A, and in turn, miR-29b/c is suppressed by DNMT3A in a DNA methylation-dependent manner. The de-regulation of both of miR-29b/c and DNMT3A leads to the epigenetic silencing of CDH1 and contributes to the metastasis phenotype in GC. This finding reveals that DNA methylation-associated silencing of miR-29b/c is critical for GC development and thus may be a therapeutic target.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25874027" target="_blank">
                    Influence of toxicologically relevant metals on human epigenetic regulation.
                </a>
            </h2>
            <p><b>Environmental toxicants such as toxic metals can alter epigenetic regulatory features such as DNA methylation, histone modification, and non-coding RNA expression.</b> <b>Heavy metals influence gene expression by epigenetic mechanisms and by directly binding to various metal response elements in the target gene promoters.</b> <b>Given the role of epigenetic alterations in regulating genes, there is potential for the integration of toxic metal-induced epigenetic alterations as informative factors in the risk assessment process.</b> <b>Here, we focus on recent advances in understanding epigenetic changes, gene expression, and biological effects induced by toxic metals.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25874017" target="_blank">
                    Age-associated DNA methylation changes in immune genes, histone modifiers and chromatin remodeling factors within 5&nbsp;years after birth in human blood leukocytes.
                </a>
            </h2>
            <p>Age-related changes in DNA methylation occurring in blood leukocytes during early childhood may reflect epigenetic maturation. We hypothesized that some of these changes involve gene networks of critical relevance in leukocyte biology and conducted a prospective study to elucidate the dynamics of DNA methylation. Serial blood samples were collected at 3, 6, 12, 24, 36, 48 and 60&nbsp;months after birth in ten healthy girls born in Finland and participating in the Type 1 Diabetes Prediction and Prevention Study. <b>DNA methylation was measured using the HumanMethylation450 BeadChip.After filtering for the presence of polymorphisms and cell-lineage-specific signatures, 794 CpG sites showed significant DNA methylation differences as a function of age in all children (41.6% age-methylated and 58.4% age-demethylated, Bonferroni-corrected P value &lt;0.01).</b> Age-methylated CpGs were more frequently located in gene bodies and within +5 to +50 kilobases (kb) of transcription start sites (TSS) and enriched in developmental, neuronal and plasma membrane genes. Age-demethylated CpGs were associated to promoters and DNAse-I hypersensitivity sites, located within -5 to +5&nbsp;kb of the nearest TSS and enriched in genes related to immunity, antigen presentation, the polycomb-group protein complex and cytoplasm.This study reveals that susceptibility loci for complex inflammatory diseases (for example, IRF5, NOD2, and PTGER4) and genes encoding histone modifiers and chromatin remodeling factors (for example, HDAC4, KDM2A, KDM2B, JARID2, ARID3A, and SMARCD3) undergo DNA methylation changes in leukocytes during early childhood. <b>These results open new perspectives to understand leukocyte maturation and provide a catalogue of CpG sites that may need to be corrected for age effects when performing DNA methylation studies in children.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25873928" target="_blank">
                    Association between DNA Methylation of the BDNF Promoter Region and Clinical Presentation in Alzheimer's Disease.
                </a>
            </h2>
            <p><b>In the present study, we examined whether DNA methylation of the brain-derived neurotrophic factor (BDNF) promoter is associated with the manifestation and clinical presentation of Alzheimer's disease (AD).Of 20 patients with AD and 20 age-matched normal controls (NCs), the DNA methylation of the BDNF promoter (measured using peripheral blood samples) was completely analyzed in 12 patients with AD and 6 NCs.</b> <b>The resulting methylation levels were compared statistically.</b> <b>Next, we investigated the correlation between the DNA methylation levels and the clinical presentation of AD.The total methylation ratio (in %) of the 20 CpG sites was significantly higher in the AD patients (5.08 ± 5.52%) than in the NCs (2.09 ± 0.81%; p &lt; 0.05).</b> <b>Of the 20 CpG sites, the methylation level at the CpG4 site was significantly higher in the AD subjects than in the NCs (p &lt; 0.05).</b> <b>Moreover, the methylation level was significantly and negatively correlated with some neuropsychological test subscores (registration, recall, and prehension behavior scores; p &lt; 0.05).These results suggest that the DNA methylation of the BDNF promoter may significantly influence the manifestation of AD and might be associated with its neurocognitive presentation.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25873363" target="_blank">
                    Targeted Histone Peptides: Insights into the Spatial Regulation of the Methyltransferase PRC2 by using a Surrogate of Heterotypic Chromatin.
                </a>
            </h2>
            <p>Eukaryotic genomes are dynamically regulated through a host of epigenetic stimuli. <b>The substrate for these epigenetic transactions, chromatin, is a polymer of nucleosome building blocks.</b> <b>In native chromatin, each nucleosome can differ from its neighbors as a result of covalent modifications to both the DNA and the histone packaging proteins.</b> <b>The heterotypic nature of chromatin presents a formidable obstacle to biochemical studies seeking to understand the role of context on epigenetic regulation.</b> <b>A chemical approach to the production of heterotypic chromatin that can be used in such studies is introduced.</b> This method involves the attachment of a user-defined modified histone peptide to a designated nucleosome within the polymer by using a peptide nucleic acid (PNA) targeting compound. <b>This strategy was applied to dissect the effect of chromatin context on the activity of the histone methyltransferase PRC2.</b> The results show that PRC2 can be stimulated to produce histone H3 methylation from a defined nucleation site.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25873173" target="_blank">
                    Mutational cooperativity linked to combinatorial epigenetic gain of function in acute myeloid leukemia.
                </a>
            </h2>
            <p><b>Specific combinations of acute myeloid leukemia (AML) disease alleles, including FLT3 and TET2 mutations, confer distinct biologic features and adverse outcome.</b> <b>We generated mice with mutations in Tet2 and Flt3, which resulted in fully penetrant, lethal AML.</b> <b>Multipotent Tet2(-/-);Flt3(ITD) progenitors (LSK CD48(+)CD150(-)) propagate disease in secondary recipients and were refractory to standard AML chemotherapy and FLT3-targeted therapy.</b> <b>Flt3(ITD) mutations and Tet2 loss cooperatively remodeled DNA methylation and gene expression to an extent not seen with either mutant allele alone, including at the Gata2 locus.</b> Re-expression of Gata2 induced differentiation in AML stem cells and attenuated leukemogenesis. <b>TET2 and FLT3 mutations cooperatively induce AML, with a defined leukemia stem cell population characterized by site-specific changes in DNA methylation and gene expression.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25871514" target="_blank">
                    Genome-wide screen of DNA methylation identifies novel markers in childhood obesity.
                </a>
            </h2>
            <p>Epigenetic modifications have been highlighted in chronic non-communicable diseases. <b>The aim of this study was to investigate genome-wide DNA methylation for the identification of methylation markers in obesity.</b> <b>With obese Chinese preschool children, we performed comprehensive DNA methylation profiling of gene promoters and CpG islands to determine the differentially methylated genes using methylated DNA immunoprecipitation followed by hybridization to the NimbleGen Human DNA Methylation 385K Promoter Plus CpG Island Microarray.</b> We found that compared to lean children, 251 promoters and 575 CGIs were demethylated, and 141 promoters and 277 CGIs were hypermethylated in obese children, and their distribution on chromosomes was imbalanced, showing more promoters and CGIs with demethylation on chromosomes 3, 16, 17 and 19 and more differentially methylated promoters and CGIs on chromosome X compared with chromosome Y. Further analysis indicated that aberrant methylations occurred mostly in HCP promoters and promoter CGIs. <b>Among the top 80 promoters and CGIs that had differentiated methylation between obese and lean children, nearly half have been previously studied, and almost all of them are involved in the pathogenesis of cancers that are associated with many organs.</b> <b>Furthermore, four genes (FZD7, PRLHR, EXOSC4, and EIF6) with differential promoter methylation were validated, and their associations with obesity must be clarified.</b> <b>In conclusion, this study represents the first effort to determine methylation markers in obese Chinese children, which has potential relevance for identifying markers that are useful in elucidating the mechanisms of obesity pathogenesis and its complications.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25870836" target="_blank">
                    Different Methylation Patterns of RUNX2, OSX, DLX5 and BSP in Osteoblastic Differentiation of Mesenchymal Stem Cells.
                </a>
            </h2>
            <p>Runt-related transcription factor 2 (RUNX2) and osterix (OSX) as two specific osteoblast transcription factors and distal-less homeobox 5 (DLX5) as a non-specific one are of paramount importance in regulating osteoblast related genes including osteocalcin, bone sialoprotein (BSP), osteopontin and collagen type Iα1. <b>The present study sets out to investigate whether epigenetic regulation of these genes is important in osteoblastic differentiation of mesenchymal stem cells (MSCs).In this experimental study, MSCs were differentiated to osteoblasts under the influence of the osteogenic differentiation medium.</b> DNA and RNA were extracted at days 0, 7, 14 and 21 from MSCs differentiating to osteoblasts. Promoter regions of RUNX2, OSX, DLX5 and BSP were analyzed by methylation-specific PCR (MSP). <b>Gene expression was analyzed during osteoblastic differentiation by quantitative real-time polymerase chain reaction (PCR).MSP analysis revealed that promoter methylation status did not change in RUNX2, DLX5 and BSP during MSC osteoblastic differentiation.</b> In contrast, OSX promoter showed a dynamic change in methylation pattern. <b>Moreover, RUNX2, OSX, DLX5 and BSP promoter regions showed three different methylation patterns during MSC differentiation.</b> <b>Gene expression analyses confirmed these results.The results show that in differentiation of MSCs to osteoblasts, epigenetic regulation of OSX may play a leading role.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25870586" target="_blank">
                    The potential use of DNA methylation biomarkers to identify risk and progression of type 2 diabetes.
                </a>
            </h2>
            <p>Type 2 diabetes mellitus (T2D) is a slowly progressive disease that can be postponed or even avoided through lifestyle changes. <b>Recent data demonstrate highly significant correlations between DNA methylation and the most important risk factors of T2D, including age and body mass index, in blood and human tissues relevant to insulin resistance and T2D.</b> <b>Also, T2D patients and individuals with increased risk of the disease display differential DNA methylation profiles and plasticity compared to controls.</b> <b>Accordingly, the novel clues to DNA methylation fingerprints in blood and tissues with deteriorated metabolic capacity indicate that blood-borne epigenetic biomarkers of T2D progression might become a reality.</b> This Review will address the most recent associations between DNA methylation and diabetes-related traits in human tissues and blood. <b>The overall focus is on the potential of future epigenome-wide studies, carried out across tissues and populations with correlations to pre-diabetes and T2D risk factors, to build up a library of epigenetic markers of risk and early progression of T2D.</b> <b>These markers may, tentatively in combination with other predictors of T2D development, increase the possibility of individual-based lifestyle prevention of T2D and associated metabolic diseases.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25870282" target="_blank">
                    MeCP2 binds to non-CG methylated DNA as neurons mature, influencing transcription and the timing of onset for Rett syndrome.
                </a>
            </h2>
            <p>Epigenetic mechanisms, such as DNA methylation, regulate transcriptional programs to afford the genome flexibility in responding to developmental and environmental cues in health and disease. A prime example involving epigenetic dysfunction is the postnatal neurodevelopmental disorder Rett syndrome (RTT), which is caused by mutations in the gene encoding methyl-CpG binding protein 2 (MeCP2). Despite decades of research, it remains unclear how MeCP2 regulates transcription or why RTT features appear 6-18 months after birth. Here we report integrated analyses of genomic binding of MeCP2, gene-expression data, and patterns of DNA methylation. <b>In addition to the expected high-affinity binding to methylated cytosine in the CG context (mCG), we find a distinct epigenetic pattern of substantial MeCP2 binding to methylated cytosine in the non-CG context (mCH, where H = A, C, or T) in the adult brain.</b> <b>Unexpectedly, we discovered that genes that acquire elevated mCH after birth become preferentially misregulated in mouse models of MeCP2 disorders, suggesting that MeCP2 binding at mCH loci is key for regulating neuronal gene expression in vivo.</b> <b>This pattern is unique to the maturing and adult nervous system, as it requires the increase in mCH after birth to guide differential MeCP2 binding among mCG, mCH, and nonmethylated DNA elements.</b> <b>Notably, MeCP2 binds mCH with higher affinity than nonmethylated identical DNA sequences to influence the level of Bdnf, a gene implicated in the pathophysiology of RTT.</b> <b>This study thus provides insight into the molecular mechanism governing MeCP2 targeting and sheds light on the delayed onset of RTT symptoms.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25870251" target="_blank">
                    Nuclear TAZ expression associates with the triple-negative phenotype in breast cancer.
                </a>
            </h2>
            <p>The Hippo signaling pathway, a conserved regulator of organ size, has emerged as an important regulatory pathway in cancer. The final transducer effectors of this pathway in mammals are the oncoproteins TAZ and YAP1, transcriptional coactivators of target genes involved in cell proliferation and survival. TAZ has been previously reported to have a role in tumorigenesis in breast cancer, but detailed analyses in the different breast cancer phenotypes have not been conducted so far. We have analyzed TAZ expression by immunohistochemistry in a retrospective series of 640 invasive breast carcinomas, comprising estrogen/progesterone receptor positive, HER2 positive, and triple-negative tumors. <b>We found a strong association of TAZ nuclear expression with the triple-negative phenotype (60.5% TAZ positive, p&lt;0.001), which was even strengthened when stratifying into the basal-like subtype (70.8% TAZ positive, p&lt;0.001).</b> <b>Moreover, 90 % of metaplastic breast carcinomas with morphological epithelial to mesenchymal transition features were TAZ positive.</b> <b>We have also investigated whether amplification or differential DNA methylation of TAZ-encoding locus could account for the observed enhanced TAZ protein expression in triple-negative/basal phenotype.</b> <b>Amplification of TAZ locus was analyzed by FISH in 30 triple-negative tumors, and we found gene amplification in some cases (6.45%).</b> DNA methylation analysis were performed in the Sequenom MassArray® MALDI-TOF platform, and we observed similar low methylation levels both in triple-negative (n=25) and estrogen/progesterone receptor positive (n= 26) tumors. These data were further confirmed in a panel of breast cancer cell lines and in the TCGA dataset. <b>Finally, patients with strong TAZ expression showed poorer clinical outcome regarding both recurrence and overall survival.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25870236" target="_blank">
                    MPP8 and SIRT1 crosstalk in E-cadherin gene silencing and epithelial-mesenchymal transition.
                </a>
            </h2>
            <p>As a critical developmental process, epithelial-mesenchymal transition (EMT) involves complex transcriptional reprogramming and has been closely linked to malignant progression. Although various epigenetic modifications, such as histone deacetylation and H3K9 methylation, have been implicated in this process, how they are coordinated remains elusive. We recently revealed that MPP8 couples H3K9 methylation and DNA methylation for E-cadherin gene silencing and promotes tumor cell migration, invasion, and EMT. <b>Here, we show that MPP8 cooperates with the class III HDAC SIRT1 in this process through their physical interaction.</b> <b>SIRT1 antagonizes PCAF-catalyzed MPP8-K439 acetylation to protect MPP8 from ubiquitin-proteasome-mediated proteolysis.</b> <b>Conversely, MPP8 recruits SIRT1 for H4K16 deacetylation after binding to methyl-H3K9 on target promoters.</b> <b>Consequently, disabling either MPP8 methyl-H3K9 binding or SIRT1 interaction de-represses E-cadherin and reduces EMT phenotypes, as does knockdown of MPP8 or SIRT1 in prostate cancer cells.</b> <b>These results illustrate how SIRT1 and MPP8 reciprocally promote each other's function and coordinate epithelial gene silencing and EMT.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25869828" target="_blank">
                    Longitudinal analysis of DNA methylation associated with birth weight and gestational age.
                </a>
            </h2>
            <p>Gestational age (GA) and birth-weight have been implicated in the determination of long-term health. It has been hypothesised that changes in DNA methylation may mediate these long-term effects. We obtained DNA methylation profiles from cord blood and peripheral blood at ages 7 and 17 in the same children from the Avon Longitudinal Study of Parents and Children. Repeated measures data were used to investigate changes in birth-related methylation during childhood and adolescence. Ten developmental phenotypes (e.g. height) were analysed to identify possible mediation of health effects by DNA methylation. In cord blood, methylation at 224 CpG sites was found to be associated with GA and 23 CpG sites with birth-weight. <b>Methylation changed in the majority of these sites over time, but neither birth characteristic was strongly associated with methylation at age 7 or 17 (using a conservative correction for multiple testing of p&lt;1.03x10(-7)) suggesting resolution of differential methylation by early childhood.</b> Associations were observed between birth-weight associated CpG sites and phenotypic characteristics in childhood. <b>One strong association involved birth weight, methylation of a CpG site proximal to the NFIX locus and bone mineral density at age 17.</b> <b>Analysis of serial methylation from birth to adolescence provided evidence for a lack of persistence of methylation differences beyond early childhood.</b> <b>Sites associated with birth weight were linked to developmental genes and have methylation levels which are associated with developmental phenotypes.</b> <b>Replication and interrogation of causal relationships are needed to substantiate whether methylation differences at birth influence the association between birth-weight and development.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25869796" target="_blank">
                    Joint effects between urinary selenium and polymorphisms in methylation related genes on breast cancer risk.
                </a>
            </h2>
            <p>The aim of this study was to explore the associations of urinary selenium and polymorphisms in methylation related genes with breast cancer risk and the interactions on the risk. The present study involved in 240 female patients with incident breast cancer and 246 age-matched controls in two affiliated hospitals of Sun Yat-sen University in Guangzhou, China, from October 2009 to July 2010. DNMT1 rs2228611, MTHFR rs1801133, and MTR rs1805087 were genotyped using a&nbsp;matrix-assisted laser desorption/ionization time-of-flight mass spectrometry platform. Urinary concentration of selenium was measured by inductively coupled plasma mass spectrometry. Women with urinary selenium in the second tertile had a&nbsp;significant reduced breast cancer risk compared to those with urinary selenium in the lowest tertile [OR (95%CI): 0.50 (0.30, 0.81)]. DNMT1 rs2228611, MTHFR rs1801133, and MTR rs1805087 were not associated with breast cancer risk. Women with the third tertile of urinary selenium had a&nbsp;significant reduced breast cancer risk compared to those with the lowest tertile among women only with CC genotype [OR (95%CI): 0.55 (0.30, 1.00)] but not CT/TT genotypes [OR (95%CI): 1.58 (0.73, 3.42)] of MTHFR rs1801133 (P for interaction=0.044). Our results suggested that selenium was associated with a&nbsp;decreased risk of breast cancer and this beneficial effect was limited to women with CC genotype of MTHFR rs1801133.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25869653" target="_blank">
                    Examining the causes and consequences of context-specific differential DNA methylation in maize.
                </a>
            </h2>
            <p>DNA methylation is a stable modification of chromatin that can contribute to epigenetic variation through the regulation of genes or transposons. <b>Profiling of DNA methylation in five maize (Zea mays) inbred lines found that while DNA methylation levels for &gt;99% of the analyzed genomic regions are similar there are still 5,000 to 20,000 context-specific differentially methylated regions (DMRs) between any two genotypes.</b> <b>The analysis of identical-by-state genomic regions that have limited genetic variation provided evidence that DMRs can occur without local sequence variation but they are less common than in regions with genetic variation.</b> <b>Characterization of the sequence-specificity of DMRs, location of DMRs relative to genes and transposons and patterns of DNA methylation in regions flanking DMRs reveals distinct subset of DMRs.</b> <b>RNAseq profiling of the same tissue revealed that only ~20% of genes with qualitative (on-off) differences in gene expression are associated with DMRs and there is little evidence for association of DMRs with genes that show quantitative differences in gene expression.</b> We also identify a set of genes that may represent cryptic information that is silenced by DNA methylation in the reference B73 genome. Many of these genes exhibit natural variation in other genotypes suggesting the potential for selection to act upon existing epigenetic natural variation. <b>This study provides insights into the origin and influences of DMRs in a crop species with a complex genome organization.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25868845" target="_blank">
                    Increased DNA methylation of scavenger receptor class B type I contributes to inhibitory effects of prenatal caffeine ingestion on cholesterol uptake and steroidogenesis in fetal adrenals.
                </a>
            </h2>
            <p>Steroid hormones synthesized from cholesterol in the fetal adrenal are crucial for fetal development. We have observed the inhibited fetal adrenal corticosterone synthesis and increased intrauterine growth retardation (IUGR) rate in rats under prenatal caffeine ingestion. The aim of this study is to evaluate the effects of prenatal caffeine ingestion on cholesterol supply in fetal adrenal steroidogenesis in rats and explore the underlying epigenetic mechanisms. <b>Pregnant Wistar rats were treated with 60mg/kg·d caffeine from gestational day (GD) 7 to GD17.</b> <b>Histological changes of fetal adrenals and increased IUGR rates were observed in the caffeine group.</b> There were significantly decreased steroid hormone contents and cholesterol supply in caffeine-treated fetal adrenals. <b>Data from the gene expression array suggested that prenatal caffeine ingestion caused increased expression of genes related to DNA methylation and decreased expression of genes related to cholesterol uptake.</b> The following conjoint analysis of DNA methylation array with these differentially expressed genes suggested that scavenger receptor class B type I (SR-BI) may play an important role in caffeine-induced cholesterol supply deficiency. <b>Moreover, real-time RT-PCR and immunohistochemical detection certified the inhibitory effects of caffeine on both mRNA and protein expression of SR-BI in the fetal adrenal.</b> And the increased DNA methylation frequency in the proximal promoter of SR-BI was confirmed by bisulfite-sequencing PCR. <b>In conclusion, prenatal caffeine ingestion can induce DNA hypermethylation of the SR-BI promoter in the rat fetal adrenal.</b> These effects may lead to decreased SR-BI expression and cholesterol uptake, which inhibits steroidogenesis in the fetal adrenal.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25867767" target="_blank">
                    Formalin-fixed, paraffin-embedded (FFPE) tissue epigenomics using Infinium HumanMethylation450 BeadChip assays.
                </a>
            </h2>
            <p>Current genome-wide methods to detect DNA-methylation in healthy and diseased tissue require high-quality DNA from fresh-frozen (FF) samples. However, well-annotated clinical samples are mostly available as formalin-fixed, paraffin-embedded (FFPE) tissues containing poor-quality DNA. <b>To overcome this limitation, we here aimed to evaluate a DNA restoration protocol for usage with the genome-wide Infinium HumanMethylation450 BeadChip assay (HM-450K).</b> <b>Sixty-six DNA samples from normal colon (n=9) and breast cancer (n=11) were interrogated separately using HM-450K.</b> Analyses included matched FF/FFPE samples and technical duplicates. <b>FFPE DNA was processed with (FFPEr) or without a DNA restoration protocol (Illumina).</b> Differentially methylated genes were finally validated in 24 additional FFPE tissues using nested methylation-specific PCR (MSP). In summary, β-values correlation between FFPEr duplicates was high (ρ=0.9927 (s.d. ±0.0015)). Matched FF/FFPEr correlation was also high (ρ=0.9590 (s.d. ±0.0184)) compared with matched FF/FFPE (ρ=0.8051 (s.d. ±0.1028). Probe detection rate in FFPEr samples (98.37%, s.d. ±0.66) was comparable to FF samples (99.98%, s.d. ±0.019) and substantially lower in FFPE samples (82.31%, s.d. ±18.65). Assay robustness was not decreased by sample archival age up to 10 years. <b>We could also demonstrate no decrease in assay robustness when using 100 ng of DNA input only.</b> Four out of the five selected differentially methylated genes could be validated by MSP. The gene failing validation by PCR showed high variation of CpG β-values in primer-binding sites. <b>In conclusion, by using the FFPE DNA restoration protocol, HM-450K assays provide robust, accurate and reproducible results with FFPE tissue-derived DNA, which are comparable to those obtained with FF tissue.</b> Most importantly, differentially methylated genes can be validated using more sensitive techniques, such as nested MSP, altogether providing an epigenomics platform for molecular pathological epidemiology research on archived samples with limited tissue amount.Laboratory Investigation advance online publication, 13 April 2015; doi:10.1038/labinvest.2015.53.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25867117" target="_blank">
                    Age dynamics of DNA damage and CpG methylation in the peripheral blood leukocytes of mice.
                </a>
            </h2>
            <p>Certain types of DNA damage are known to accumulate with age. Here we present the quantitative data describing the extent of the spontaneous DNA damage in 144 SHK mice of various ages. In each animal, we assessed double strand breaks, single strand breaks and alkali-labile sites, as well as amounts of oxidized purines, oxidized pyrimidines and misincorporated uracils. In addition, overall levels of genome DNA methylation were evaluated. The amounts of oxidized pyrimidines were correlated with age in males only, while the amounts of double strand breaks (DSB) in DNA samples were correlated with age in females only (R=0.26; p&lt;0.035). No age-related accumulation of single-strand breaks (SSB) was observed. The hypomethylation of DNA was significant in aging females, but not in aging males. <b>Various types of DNA damage were correlated to each other.</b> <b>In attempt to develop more stable indicator of age-dependent alterations in DNA, the DNA Damage Level Differential (DDLD) indices was developed for comet assaying of peripheral blood leukocytes.</b> <b>As expected, DDLD index was shown to be better correlated with age than any single quantitative measure reflecting certain type of DNA damage.</b> <b>A variability of effectiveness of various kinds of DNA repair in individual animals was larger than expected.</b> This conclusion may have a substantial impact on subsequent studies of the mutagens and other kinds of environmental stressors in animal populations. <b>Nor DDLD neither individual quantitative measures of DNA damage were capable of prediction post-sampling survival time.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25866970" target="_blank">
                    TALE-mediated epigenetic suppression of CDKN2A increases replication in human fibroblasts.
                </a>
            </h2>
            <p><b>Current strategies to alter disease-associated epigenetic modifications target ubiquitously expressed epigenetic regulators.</b> <b>This approach does not allow specific genes to be controlled in specific cell types; therefore, tools to selectively target epigenetic modifications in the desired cell type and strategies to more efficiently correct aberrant gene expression in disease are needed.</b> Here, we have developed a method for directing DNA methylation to specific gene loci by conjugating catalytic domains of DNA methyltransferases (DNMTs) to engineered transcription activator-like effectors (TALEs). We demonstrated that these TALE-DNMTs direct DNA methylation specifically to the targeted gene locus in human cells. Further, we determined that minimizing direct nucleotide sequence repeats within the TALE moiety permits efficient lentivirus transduction, allowing easy targeting of primary cell types. Finally, we demonstrated that directed DNA methylation with a TALE-DNMT targeting the CDKN2A locus, which encodes the cyclin-dependent kinase inhibitor p16, decreased CDKN2A expression and increased replication of primary human fibroblasts, as intended. <b>Moreover, overexpression of p16 in these cells reversed the proliferative phenotype, demonstrating the specificity of our epigenetic targeting.</b> <b>Together, our results demonstrate that TALE-DNMTs can selectively target specific genes and suggest that this strategy has potential application for the development of locus-specific epigenetic therapeutics.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25866590" target="_blank">
                    Genome-wide blood DNA methylation alterations at regulatory elements and heterochromatic regions in monozygotic twins discordant for obesity and liver fat.
                </a>
            </h2>
            <p><b>The current epidemic of obesity and associated diseases calls for swift actions to better understand the mechanisms by which genetics and environmental factors affect metabolic health in humans.</b> <b>Monozygotic (MZ) twin pairs showing discordance for obesity suggest that epigenetic influences represent one such mechanism.</b> We studied genome-wide leukocyte DNA methylation variation in 30 clinically healthy young adult MZ twin pairs discordant for body mass index (BMI; average within-pair BMI difference: 5.4 ± 2.0&nbsp;kg/m(2)).There were no differentially methylated cytosine-guanine (CpG) sites between the co-twins discordant for BMI. However, stratification of the twin pairs based on the level of liver fat accumulation revealed two epigenetically highly different groups. Significant DNA methylation differences (n = 1,236 CpG sites (CpGs)) between the co-twins were only observed if the heavier co-twins had excessive liver fat (n = 13 twin pairs). <b>This unhealthy pattern of obesity was coupled with insulin resistance and low-grade inflammation.</b> <b>The differentially methylated CpGs included 23 genes known to be associated with obesity, liver fat, type 2 diabetes mellitus (T2DM) and metabolic syndrome, and potential novel metabolic genes.</b> Differentially methylated CpG sites were overrepresented at promoters, insulators, and heterochromatic and repressed regions. Based on predictions by overlapping histone marks, repressed and weakly transcribed sites were significantly more often hypomethylated, whereas sites with strong enhancers and active promoters were hypermethylated. <b>Further, significant clustering of differentially methylated genes in vitamin, amino acid, fatty acid, sulfur, and renin-angiotensin metabolism pathways was observed.The methylome in leukocytes is altered in obesity associated with metabolic disturbances, and our findings indicate several novel candidate genes and pathways in obesity and obesity-related complications.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25866341" target="_blank">
                    Human papillomavirus type 16 L1/L2 DNA methylation shows weak association with cervical disease grade in young women.
                </a>
            </h2>
            <p>Persistent infection with human papillomavirus (HPV) type 16 causes the majority of cervical cancers. Genital HPV infection is very common, but neoplastic progression is uncommon. <b>There is an urgent need for biomarkers associated with cervical neoplasia, to enable triage of women who test positive for HPV.To assess the ability of quantitative measurement of HPV16 DNA methylation to separate samples of different cytological and histological grades from young women, among whom rates of HPV infection are high.DNA methylation was quantified by pyrosequencing of bisulphite converted DNA from liquid based cytology samples from 234 women (mean age 20.6 years) who tested positive for HPV16 and showed varying degrees of neoplasia.</b> <b>Methylation was assessed at CpGs in the HPV E2 and L1/L2 regions.The performance of methylation-based classifiers was assessed by ROC curve analyses.</b> <b>The best combination of CpGs (5600 and 5609) achieved AUCs of 0.656 (95% CI=0.520-0.792) for separation of cytologically normal and severely dyskaryotic samples, and 0.639 (95% CI=0.547-0.731) for separation of samples with or without high-grade neoplasia (CIN2+/-).The data are consistent with HPV L1/L2 methylation being a marker of the duration of infection in a specific host.</b> <b>Assessment of HPV DNA methylation is hence a promising biomarker to triage HPV-positive cytology samples, but may have limited utility in young women.</b> <b>Future studies assessing the likely utility of HPV DNA methylation as a potential triage biomarker must take account of women's age.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25866208" target="_blank">
                    Structures of the Ets Domains of Transcription Factors ETV1, ETV4, ETV5 and FEV: Determinants of DNA Binding and Redox Regulation by Disulfide bond formation.
                </a>
            </h2>
            <p>Ets transcription factors, which share the conserved Ets DNA-binding domain, number nearly 30 members in humans and are particularly involved in developmental processes. Their deregulation following changes in expression, transcriptional activity, or by chromosomal translocation, plays a critical role in carcinogenesis. Ets DNA-binding, selectivity and regulation have been extensively studied, although questions still arise regarding binding specificity outside the core GGA recognition sequence, and the mode of action of Ets post-translational modifications. <b>Here we report the crystal structures of ETV1, ETV4, ETV5 and FEV, alone and in complex with DNA.</b> We identify previously unrecognized features of the protein-DNA interface. <b>Interactions with the DNA backbone account for most of the binding affinity.</b> We describe a highly coordinated network of water molecules acting in base selection upstream of the GGAA core, and the structural features that may account for discrimination against methylated cytidine residues. <b>Unexpectedly, all proteins crystallized as disulfide-linked dimers, exhibiting a novel interface (distant to the DNA recognition helix).</b> <b>Homodimers of ETV1, ETV4 and ETV5 could be reduced to monomers, leading to a 40-200-fold increase in DNA binding affinity.</b> Hence, we present the first indication of a redox-dependent regulatory mechanism which may control the activity of this subset of oncogenic Ets transcription factors.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25865601" target="_blank">
                    Identification of functionally methylated regions based on discriminant analysis through integrating methylation and gene expression data.
                </a>
            </h2>
            <p>DNA methylation is essential not only in cellular differentiation but also in diseases. Identification of differentially methylated patterns between case and control groups is important in understanding the mechanism and possible functionality of complex diseases. We propose a method to find possible functionally methylated regions which not only are differentially methylated but also have an effect on gene expression. It integrates methylation and gene expression data and is based on distance discriminant analysis (DDA). <b>In the procedure of identifying differentially methylated regions (DMRs), we do not need to cluster methylation sites or partition the genome in advance.</b> <b>Therefore, the identified DMRs have a larger coverage than those of bump hunting and Ong's methods.</b> <b>Furthermore, through incorporating gene expression data as a complementary source, whether these DMRs are functional is determined through estimating the difference of the corresponding genes.</b> <b>Through a comparison of our approach with bump hunting and Ong's methods for simulation data, it is shown that our method is more powerful in identifying DMRs which have a larger distance in the genome, or only consist of a few sites and have higher sensitivity and specificity.</b> Also, our method is more robust to heterogeneity of data. <b>Applied to different real datasets, we find that most of the functional DMRs are hyper-methylated and located at CpG rich regions (e.g.</b> islands, TSS200 and TSS1500), consistent with the fact that the methylation levels of CpG islands are higher in tumors than normal. Through comparing and analyzing the results of different datasets, we find that the change of methylation in some regions may be related to diseases through changing expression of the corresponding genes, and show the effectiveness of our method.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25865577" target="_blank">
                    Multiple levels of epigenetic control for bone biology and pathology.
                </a>
            </h2>
            <p><b>Multiple dimensions of epigenetic control contribute to regulation of gene expression that governs bone biology and pathology.</b> <b>Once confined to DNA methylation and a limited number of post-translational modifications of histone proteins, the definition of epigenetic mechanisms is expanding to include contributions of non-coding RNAs and mitotic bookmarking, a mechanism for retaining phenotype identity during cell proliferation.</b> <b>Together these different levels of epigenetic control of physiological processes and their perturbations that are associated with compromised gene expression during the onset and progression of disease, have contributed to an unprecedented understanding of the activities (operation) of the genomic landscape.</b> <b>Here, we address general concepts that explain the contribution of epigenetic control to the dynamic regulation of gene expression during eukaryotic transcription.</b> This article is part of a Special Issue entitled Epigenetics and Bone.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25865119" target="_blank">
                    Decoding the regulatory landscape of melanoma reveals TEADS as regulators of the invasive cell state.
                </a>
            </h2>
            <p><b>Transcriptional reprogramming of proliferative melanoma cells into a phenotypically distinct invasive cell subpopulation is a critical event at the origin of metastatic spreading.</b> Here we generate transcriptome, open chromatin and histone modification maps of melanoma cultures; and integrate this data with existing transcriptome and DNA methylation profiles from tumour biopsies to gain insight into the mechanisms underlying this key reprogramming event. <b>This shows thousands of genomic regulatory regions underlying the proliferative and invasive states, identifying SOX10/MITF and AP-1/TEAD as regulators, respectively.</b> <b>Knockdown of TEADs shows a previously unrecognized role in the invasive gene network and establishes a causative link between these transcription factors, cell invasion and sensitivity to MAPK inhibitors.</b> Using regulatory landscapes and in silico analysis, we show that transcriptional reprogramming underlies the distinct cellular states present in melanoma. Furthermore, it reveals an essential role for the TEADs, linking it to clinically relevant mechanisms such as invasion and resistance.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25864559" target="_blank">
                    Genome-wide sperm DNA methylation changes after 3 months of exercise training in humans.
                </a>
            </h2>
            <p>DNA methylation programs gene expression and is involved in numerous biological processes. <b>Accumulating evidence supports transgenerational inheritance of DNA methylation changes in mammals via germ cells.</b> <b>Our aim was to determine the effect of exercise on sperm DNA methylation.Twenty-four men were recruited and assigned to an exercise intervention or control group.</b> Clinical parameters were measured and sperm samples were donated by subjects before and after the 3-month time-period. <b>Mature sperm global and genome-wide DNA methylation was assessed using an ELISA assay and the 450K BeadChip (Illumina).Global and genome-wide sperm DNA methylation was altered after 3 months of exercise training.</b> <b>DNA methylation changes occurred in genes related to numerous diseases such as Schizophrenia and Parkinson's disease.Our study provides the first evidence showing exercise training reprograms the sperm methylome.</b> <b>Whether these DNA methylation changes are inherited to future generations warrants attention.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25864488" target="_blank">
                    Genome-wide differentially methylated genes in prostate cancer tissues from African-American and Caucasian men.
                </a>
            </h2>
            <p>Increasing evidence suggests that aberrant DNA methylation changes may contribute to prostate cancer (PCa) ethnic disparity. To comprehensively identify DNA methylation alterations in PCa disparity, we used the Illumina 450K methylation platform to interrogate the methylation status of 485,577 CpG sites focusing on gene-associated regions of the human genome. Genomic DNA from African-American (AA; 7 normal and 3 cancers) and Caucasian (Cau; 8 normal and 3 cancers) was used in the analysis. Hierarchical clustering analysis identified probe-sets unique to AA and Cau samples, as well as common to both. We selected 25 promoter-associated novel CpG sites most differentially methylated by race (fold change &gt; 1.5-fold; adjusted P &lt; 0.05) and compared the β-value of these sites provided by the Illumina, Inc. array with quantitative methylation obtained by pyrosequencing in 7 prostate cell lines. We found very good concordance of the methylation levels between β-value and pyrosequencing. Gene expression analysis using qRT-PCR in a subset of 8 genes after treatment with 5-aza-2'-deoxycytidine and/or trichostatin showed up-regulation of gene expression in PCa cells. <b>Quantitative analysis of 4 genes, SNRPN, SHANK2, MST1R, and ABCG5, in matched normal and PCa tissues derived from AA and Cau PCa patients demonstrated differential promoter methylation and concomitant differences in mRNA expression in prostate tissues from AA vs. Cau.</b> <b>Regression analysis in normal and PCa tissues as a function of race showed significantly higher methylation prevalence for SNRPN (P = 0.012), MST1R (P = 0.038), and ABCG5 (P &lt; 0.0002) for AA vs. Cau samples.</b> <b>We selected the ABCG5 and SNRPN genes and verified their biological functions by Western blot analysis and siRNA gene knockout effects on cell proliferation and invasion in 4 PCa cell lines (2 AA and 2 Cau patients-derived lines).</b> <b>Knockdown of either ABCG5 or SNRPN resulted in a significant decrease in both invasion and proliferation in Cau PCa cell lines but we did not observe these remarkable loss-of-function effects in AA PCa cell lines.</b> <b>Our study demonstrates how differential genome-wide DNA methylation levels influence gene expression and biological functions in AA and Cau PCa.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25863144" target="_blank">
                    Does DNA methylation regulate metamorphosis? The case of the sea lamprey (Petromyzon marinus) as an example.
                </a>
            </h2>
            <p>Lampreys represent one of the most ancient vertebrate lineages enclosing a special interest for genetic and epigenetic studies. The sea lamprey (Petromyzon marinus) is an anadromous species that experiences metamorphosis all the way up to the adult stage. Although representing a gradual process, metamorphosis in this species involves dramatic conversions with regard to physiological together with structural body changes preparing individuals for a marine and parasitic life; in consequence, multiple gene expression modifications are expected. The implications of thyroid hormones and HOX gene expression changes have previously been reported in this species and also in other vertebrate species. Nonetheless, information lacks on how these genes are regulated in lampreys. <b>We here report about the existence of methylation pattern differences between the adult and the larvae sea lamprey life cycle stages making use of the Methylation-Sensitive Amplified Polymorphism (MSAP) technique.</b> Differentially methylated fragment sequencing allowed to establish homologous identities with HOX genes involved in morphogenesis, along with genes related to the water balance and to the osmotic homoeostasis, all associated to a marine environment adaptation. <b>These results provide evidences revealing that DNA methylation plays a role in the epigenetic regulation of the P. marinus post-natal development representing a starting point for future studies.</b> <b>To the best of our knowledge, this is the first study which detects DNA methylation changes associated with metamorphosis in lampreys.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25862956" target="_blank">
                    Effects of oral exposure to bisphenol A on gene expression and global genomic DNA methylation in the prostate, female mammary gland, and uterus of NCTR Sprague-Dawley rats.
                </a>
            </h2>
            <p>Bisphenol A (BPA), an industrial chemical used in the manufacture of polycarbonate and epoxy resins, binds to the nuclear estrogen receptor with an affinity 4-5 orders of magnitude lower than that of estradiol. <b>We reported previously that "high BPA" (100,000 and 300,000 µg/kg body weight (bw)/day), but not "low BPA" [2.5-2700 µg/kg bw/day], induced clear adverse effects in NCTR Sprague-Dawley rats gavaged daily from gestation day 6 through postnatal day 90.</b> <b>The "high BPA" effects partially overlapped those of ethinyl estradiol (EE2, 0.5 and 5.0 µg/kg bw/day).</b> <b>To evaluate further the potential of "low BPA" to induce biological effects, here we assessed the global genomic DNA methylation and gene expression in the prostate and female mammary glands, tissues identified previously as potential targets of BPA, and uterus, a sensitive estrogen-responsive tissue.</b> <b>Both doses of EE2 modulated gene expression, including of known estrogen-responsive genes, and PND 4 global gene expression data showed a partial overlap of the "high BPA" effects with those of EE2.</b> <b>The "low BPA" doses modulated the expression of several genes; however, the absence of a dose response reduces the likelihood that these changes were causally linked to the treatment.</b> These results are consistent with the toxicity outcomes.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25862628" target="_blank">
                    DNA methylation mediates the effect of maternal smoking during pregnancy on birthweight of the offspring.
                </a>
            </h2>
            <p>We examined whether the effect of maternal smoking during pregnancy on birthweight of the offspring was mediated by smoking-induced changes to DNA methylation in cord blood.First, we used cord blood of 129 Dutch children exposed to maternal smoking vs 126 unexposed to maternal and paternal smoking (53% male) participating in the GECKO Drenthe birth cohort. DNA methylation was measured using the Illumina HumanMethylation450 Beadchip. We performed an epigenome-wide association study for the association between maternal smoking and methylation followed by a mediation analysis of the top signals [false-discovery rate (FDR) &lt; 0.05]. We adjusted both analyses for maternal age, education, pre-pregnancy BMI, offspring's sex, gestational age and white blood cell composition. Secondly, in 175 exposed and 1248 unexposed newborns from two independent birth cohorts, we replicated and meta-analysed results of eight cytosine-phosphate-guanine (CpG) sites in the GFI1 gene, which showed the most robust mediation. <b>Finally, we performed functional network and enrichment analysis.We found 35 differentially methylated CpGs (FDR &lt; 0.05) in newborns exposed vs unexposed to smoking, of which 23 survived Bonferroni correction (P &lt; 1 × 10(-7)).</b> These 23 CpGs mapped to eight genes: AHRR, GFI1, MYO1G, CYP1A1, NEUROG1, CNTNAP2, FRMD4A and LRP5. We observed partial confirmation as three of the eight CpGs in GFI1 replicated. <b>These CpGs partly mediated the effect of maternal smoking on birthweight (Sobel P &lt; 0.05) in meta-analysis of GECKO and the two replication cohorts.</b> <b>Differential methylation of these three GFI1 CpGs explained 12-19% of the 202 g lower birthweight in smoking mothers.</b> <b>Functional enrichment analysis pointed towards activation of cell-mediated immunity.Maternal smoking during pregnancy was associated with cord blood methylation differences.</b> <b>We observed a potentially mediating role of methylation in the association between maternal smoking during pregnancy and birthweight of the offspring.</b> Functional network analysis suggested a role in activating the immune system.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25862579" target="_blank">
                    Genetic and epigenetic alterations in hepatitis B virus-associated hepatocellular carcinoma.
                </a>
            </h2>
            <p><b>Hepatitis B virus (HBV) is a major cause of hepatocellular carcinoma (HCC).</b> Its chronic infection can lead to chronic liver inflammation and the accumulation of genetic alterations to result in the oncogenic transformation of hepatocytes. <b>HBV can also sensitize hepatocytes to oncogenic transformation by causing genetic and epigenetic changes of the host chromosomes.</b> <b>HBV DNA can insert into host chromosomes and recent large-scale whole-genome sequencing studies revealed recurrent HBV DNA integrations sites that may play important roles in the initiation of hepatocellular carcinogenesis.</b> <b>HBV can also cause epigenetic changes by altering the methylation status of cellular DNA, the post-translational modification of histones, and the expression of microRNAs.</b> These changes can also lead to the eventual hepatocellular transformation. <b>These recent findings on the genetic and epigenetic alterations of the host chromosomes induced by HBV opened a new avenue for the development of novel diagnosis and treatments for HBV-induced HCC.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25862382" target="_blank">
                    Germline and somatic imprinting in the nonhuman primate highlights species differences in oocyte methylation.
                </a>
            </h2>
            <p>Genomic imprinting is an epigenetic mechanism resulting in parental allele-specific gene expression. Defects in normal imprinting are found in cancer, assisted reproductive technologies, and several human syndromes. <b>In mouse models, germline-derived DNA methylation is shown to regulate imprinting.</b> <b>Though imprinting is largely conserved between mammals, species- and tissue-specific domains of imprinted expression exist.</b> <b>Using the cynomolgus macaque (Macaca fascicularis) to assess primate-specific imprinting, we present a comprehensive view of tissue-specific imprinted expression and DNA methylation at established imprinted gene clusters.</b> <b>For example, like mouse and unlike human, macaque IGF2R is consistently imprinted, and the PLAGL1, INPP5F transcript variant 2, and PEG3 imprinting control regions are not methylated in the macaque germline but acquire this post-fertilization.</b> Methylome data from human early embryos appear to support this finding. <b>These suggest fundamental differences in imprinting control mechanisms between primate species and rodents at some imprinted domains, with implications for our understanding of the epigenetic programming process in humans and its influence on disease.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25862379" target="_blank">
                    DAT1 methylation changes in alcohol-dependent individuals vs. controls.
                </a>
            </h2>
            <p><b>The dopaminergic system plays a crucial role in the development of alcohol dependence.</b> Regulation of the extracellular dopamine concentration is driven by dopamine transporter. <b>Both, the expression and function of dopamine transporter, are influenced by chronic alcohol intake.</b> Dopamine transporter (DAT) gene is also supposed to be differentially methylated in alcohol-dependent patients than in controls.DNA was extracted from peripheral blood leukocytes. <b>We analyzed the methylation status in 23 CpG islands of DAT gene promoter in alcohol dependent subjects (n&nbsp;=&nbsp;171) and control (n&nbsp;=&nbsp;160) group.No statistical differences in the general frequency of DAT CpG islands was revealed between patients (altogether 175 methylated islands) and control subjects (170 methylated islands (p&nbsp;=&nbsp;0.86).</b> <b>However it was revealed that one of analyzed positions is significantly more often methylated in control subjects than in alcohol dependent individuals (p&nbsp;=&nbsp;0.0296).Further subsequent studies are necessary to determine whether the methylation change of one (out of 23) CpG site results in DAT expression changes.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25861997" target="_blank">
                    The Clinical Course of an Overgrowth Syndrome, From Diagnosis in Infancy Through Adulthood: The Case of Beckwith-Wiedemann Syndrome.
                </a>
            </h2>
            <p>Beckwith-Wiedemann syndrome (BWS) is the most common genetic overgrowth syndrome, and it is frequently clinically recognizable because of characteristic features. These features include macrosomia, hemihypertrophy, macroglossia, facial nevus flammeus, earlobe creases and pits, omphalocele, and organomegaly. <b>The most common molecular cause is hypomethylation of the maternal imprinting control region 2 (ICR2) in 11p15.</b> <b>Other molecular causes include hypermethylation of the maternal ICR1 in 11p15, mutations in CDKN1C, mosaic uniparental disomy 11p15, and chromosomal abnormalities involving 11p15.</b> <b>Some of these abnormalities are testable, and DNA methylation tests of 11p15 confirm about 60% of cases with BWS.</b> The main management issues in pediatrics are hypoglycemia at birth, macroglossia, and surveillance for embryonal tumors, especially Wilms and hepatoblastoma.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25861810" target="_blank">
                    Impact of age, BMI and HbA1c levels on the genome-wide DNA methylation and mRNA expression patterns in human adipose tissue and identification of epigenetic biomarkers in blood.
                </a>
            </h2>
            <p>Increased age, BMI and HbA1c levels are risk factors for several non-communicable diseases. However, the impact of these factors on the genome-wide DNA methylation pattern in human adipose tissue remains unknown. We analyzed DNA methylation of ∼480,000 sites in human adipose tissue from 96 males and 94 females, and related methylation to age, BMI and HbA1c. We also compared epigenetic signatures in adipose tissue and blood. Age was significantly associated with both altered DNA methylation and expression of 1,050 genes (e.g. FHL2, NOX4 and PLG). Interestingly, many reported epigenetic biomarkers of ageing in blood, including ELOVL2, FHL2, KLF14 and GLRA1, also showed significant correlations between adipose tissue DNA methylation and age in our study. <b>The most significant association between age and adipose tissue DNA methylation was found upstream of ELOVL2.</b> We identified 2,825 genes (e.g. <b>FTO, ITIH5, CCL18, MTCH2, IRS1 and SPP1) where both DNA methylation and expression correlated with BMI.</b> <b>Methylation at previously reported HIF3A sites correlated significantly with BMI in females only.</b> <b>HbA1c (range 28-46 mmol/mol) correlated significantly with methylation of 711 sites, annotated to e.g.</b> RAB37, TICAM1 and HLA-DPB1. <b>Pathway analyses demonstrated that methylation levels associated with age and BMI are overrepresented among genes involved in cancer, type 2 diabetes and cardiovascular disease.</b> Our results highlight the impact of age, BMI and HbA1c on epigenetic variation of candidate genes for metabolic diseases and cancer in human adipose tissue. Importantly, we demonstrate that epigenetic biomarkers in blood can mirror age-related epigenetic signatures in target tissues for metabolic diseases such as adipose tissue.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25861393" target="_blank">
                    From inflammaging to healthy aging by dietary lifestyle choices: is epigenetics the key to personalized nutrition?
                </a>
            </h2>
            <p>The progressively older population in developed countries is reflected in an increase in the number of people suffering from age-related chronic inflammatory diseases such as metabolic syndrome, diabetes, heart and lung diseases, cancer, osteoporosis, arthritis, and dementia. <b>The heterogeneity in biological aging, chronological age, and aging-associated disorders in humans have been ascribed to different genetic and environmental factors (i.e., diet, pollution, stress) that are closely linked to socioeconomic factors.</b> The common denominator of these factors is the inflammatory response. <b>Chronic low-grade systemic inflammation during physiological aging and immunosenescence are intertwined in the pathogenesis of premature aging also defined as 'inflammaging.</b> ' The latter has been associated with frailty, morbidity, and mortality in elderly subjects. However, it is unknown to what extent inflammaging or longevity is controlled by epigenetic events in early life. Today, human diet is believed to have a major influence on both the development and prevention of age-related diseases. Most plant-derived dietary phytochemicals and macro- and micronutrients modulate oxidative stress and inflammatory signaling and regulate metabolic pathways and bioenergetics that can be translated into stable epigenetic patterns of gene expression. <b>Therefore, diet interventions designed for healthy aging have become a hot topic in nutritional epigenomic research.</b> Increasing evidence has revealed that complex interactions between food components and histone modifications, DNA methylation, non-coding RNA expression, and chromatin remodeling factors influence the inflammaging phenotype and as such may protect or predispose an individual to many age-related diseases. Remarkably, humans present a broad range of responses to similar dietary challenges due to both genetic and epigenetic modulations of the expression of target proteins and key genes involved in the metabolism and distribution of the dietary constituents. Here, we will summarize the epigenetic actions of dietary components, including phytochemicals, and macro- and micronutrients as well as metabolites, that can attenuate inflammaging. <b>We will discuss the challenges facing personalized nutrition to translate highly variable interindividual epigenetic diet responses to potential individual health benefits/risks related to aging disease.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25861392" target="_blank">
                    Multiplex bisulfite PCR resequencing of clinical FFPE DNA.
                </a>
            </h2>
            <p>The clinical utility of DNA methylation as a predictive or prognostic biomarker requires scalable resequencing protocols for bisulfite-converted DNA. Key features of any validation method should be adaptability for low- or high-throughput needs and high reproducibility, and should only require minimal amounts of precious clinical sample as input material. Critically, this method should also deliver robust results when working with bisulfite-converted DNA extracted from formalin-fixed, paraffin-embedded (FFPE) blocks.We report here for the first time on comparison studies between the Fluidigm Access Array system and multiplex assays for multiplex bisulfite PCR resequencing. <b>The requirement of the Fluidigm Access Array system for high template amounts and its sensitivity to variations in template quality rendered it unsuitable for bisulfite PCR applications utilizing FFPE DNA.</b> <b>In response to this limitation, we established a multiplex bisulfite PCR assay capable of delivering robust methylation data using minimal amounts of FFPE clinical DNA.</b> To evaluate the parameters and reproducibility of this assay, 57 amplicons were used to prepare sequencing libraries in triplicate for 13 FFPE tumour samples, as well as a series of 5 methylated controls (0%, 25%, 50%, 75%, and 100%). <b>Analysis of this data demonstrated that this multiplex assay had high reproducibility (mean standard deviation of 1.4% for methylation values), was low cost, required low sample input (50 ng of DNA or less), and could be scaled for both low- and high-throughput needs.</b> <b>Notably, ExoSAP-IT (exonuclease I) treatment to remove residual primers in bisulfite resequencing libraries appeared to degrade the library and generate a high-molecular weight smear which may impact on the degree of methylation assessed.Multiplex bisulfite PCR assays represent a convenient and scalable method for validation and screening of methylated DNA regions from archival FFPE DNA.</b> <b>Moreover, the library construction process detailed here can be rapidly optimized and implemented with a minimal amount of work, can be performed using the standard equipment found in any molecular biology laboratory, and can be easily adapted for use on both genomic DNA and bisulfite DNA applications.</b> However, in preparing bisulfite libraries for sequencing, the use of ExoSAP-IT is not recommended due to potential off-target nuclease effects which may impact downstream methylation analysis.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25861255" target="_blank">
                    Genome-Wide Expression of MicroRNAs Is Regulated by DNA Methylation in Hepatocarcinogenesis.
                </a>
            </h2>
            <p>Background. Previous studies, including ours, have examined the regulation of microRNAs (miRNAs) by DNA methylation, but whether this regulation occurs at a genome-wide level in hepatocellular carcinoma (HCC) is unclear. Subjects/Methods. <b>Using a two-phase study design, we conducted genome-wide screening for DNA methylation and miRNA expression to explore the potential role of methylation alterations in miRNAs regulation.</b> Results. <b>We found that expressions of 25 miRNAs were statistically significantly different between tumor and nontumor tissues and perfectly differentiated HCC tumor from nontumor.</b> <b>Six miRNAs were overexpressed, and 19 were repressed in tumors.</b> <b>Among 133 miRNAs with inverse correlations between methylation and expression, 8 miRNAs (6%) showed statistically significant differences in expression between tumor and nontumor tissues.</b> <b>Six miRNAs were validated in 56 additional paired HCC tissues, and significant inverse correlations were observed for miR-125b and miR-199a, which is consistent with the inactive chromatin pattern found in HepG2 cells.</b> Conclusion. These data suggest that the expressions of miR-125b and miR-199a are dramatically regulated by DNA hypermethylation that plays a key role in hepatocarcinogenesis.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25860644" target="_blank">
                    Epigenetic mechanisms contribute to the expression of immune related genes in the livers of dairy cows fed a high concentrate diet.
                </a>
            </h2>
            <p>Epigenetic modifications critically regulate the expression of immune-related genes in response to inflammatory stimuli. <b>It has been extensively reported that a high concentrate (HC) diet can trigger systemic inflammation in dairy cows, yet it is unclear whether epigenetic regulation is involved in the expression of immune genes in the livers of dairy cows.</b> This study aimed to investigate the impact of epigenetic modifications on the expression of immune-related genes.In eight mid-lactating cows, we installed a rumen cannula and catheters of the portal and hepatic veins. Cows were randomly assigned to either the treatment group fed a high concentrate (HC) diet (60% concentrate + 40% forage, n = 4) or a control group fed a low concentrate (LC) diet (40% concentrate + 60% forage, n = 4).After 10 weeks of feeding, the rumen pH was reduced, and levels of lipopolysaccharide (LPS) in the rumen, and portal and hepatic veins were notably increased in the HC group compared with the LC group. <b>The expression levels of detected immune response-related genes, including Toll-like receptor 4 (TLR4), cytokines, chemokines, and acute phase proteins, were significantly up-regulated in the livers of cows fed a HC diet.</b> <b>Chromatin loosening at the promoter region of four candidate immune-related genes (TLR4, LPS-binding protein, haptoglobin, and serum amyloid A3) was elicited, and was strongly correlated with enhanced expression of these genes in the HC group.</b> <b>Demethylation at the promoter region of all four candidate immune-related genes was accompanied by chromatin decompaction.After HC diet feeding, LPS derived from the digestive tract translocated to the liver via the portal vein, enhancing hepatic immune gene expression.</b> <b>The up-regulation of these immune genes was mediated by epigenetic mechanisms, which involve chromatin remodeling and DNA methylation.</b> Our findings suggest that modulating epigenetic mechanisms could provide novel ways to treat systemic inflammatory responses elicited by the feeding of a HC diet.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25860502" target="_blank">
                    In Ovo Injection of Betaine Affects Hepatic Cholesterol Metabolism through Epigenetic Gene Regulation in Newly Hatched Chicks.
                </a>
            </h2>
            <p>Betaine is reported to regulate hepatic cholesterol metabolism in mammals. Chicken eggs contain considerable amount of betaine, yet it remains unknown whether and how betaine in the egg affects hepatic cholesterol metabolism in chicks. In this study, eggs were injected with betaine at 2.5 mg/egg and the hepatic cholesterol metabolism was investigated in newly hatched chicks. Betaine did not affect body weight or liver weight, but significantly increased the serum concentration (P &lt; 0.05) and the hepatic content (P &lt; 0.01) of cholesterol. Accordingly, the cholesterol biosynthetic enzyme HMGCR was up-regulated (P &lt; 0.05 for both mRNA and protein), while CYP7A1 which converts cholesterol to bile acids was down-regulated (P &lt; 0.05 for mRNA and P = 0.07 for protein). Moreover, hepatic protein content of the sterol-regulatory element binding protein 1 which regulates cholesterol and lipid biosynthesis, and the mRNA abundance of ATP binding cassette sub-family A member 1 (ABCA1) which mediates cholesterol counter transport were significantly (P &lt; 0.05) increased in betaine-treated chicks. Meanwhile, hepatic protein contents of DNA methyltransferases 1 and adenosylhomocysteinase-like 1 were increased (P &lt; 0.05), which was associated with global genomic DNA hypermethylation (P &lt; 0.05) and diminished gene repression mark histone H3 lysine 27 trimethylation (P &lt; 0.05). Furthermore, CpG methylation level on gene promoters was found to be increased (P &lt; 0.05) for CYP7A1 yet decreased (P &lt; 0.05) for ABCA1. These results indicate that in ovo betaine injection regulates hepatic cholesterol metabolism in chicks through epigenetic mechanisms including DNA and histone methylations.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25860442" target="_blank">
                    Bone Marrow Stromal Antigen 2 (BST-2) DNA Is Demethylated in Breast Tumors and Breast Cancer Cells.
                </a>
            </h2>
            <p><b>Bone marrow stromal antigen 2 (BST-2) is a known anti-viral gene that has been recently identified to be overexpressed in many cancers, including breast cancer.</b> BST-2 is critical for the invasiveness of breast cancer cells and the formation of metastasis in vivo. Although the regulation of BST-2 in immune cells is unraveling, it is unknown how BST-2 expression is regulated in breast cancer. We hypothesized that meta-analyses of BST-2 gene expression and BST-2 DNA methylation profiles would illuminate mechanisms regulating elevated BST-2 expression in breast tumor tissues and cells.We performed comprehensive meta-analyses of BST-2 gene expression and BST-2 DNA methylation in The Cancer Genome Atlas (TCGA) and various Gene Expression Omnibus (GEO) datasets. BST-2 expression levels and BST-2 DNA methylation status at specific CpG sites on the BST-2 gene were compared for various breast tumor molecular subtypes and breast cancer cell lines.We show that BST-2 gene expression is inversely associated with the methylation status at specific CpG sites in primary breast cancer specimens and breast cancer cell lines. BST-2 demethylation is significantly more prevalent in primary tumors and cancer cells than in normal breast tissues or normal mammary epithelial cells. Demethylation of the BST-2 gene significantly correlates with its mRNA expression. These studies provide the initial evidence that significant differences exist in BST-2 DNA methylation patterns between breast tumors and normal breast tissues, and that BST-2 expression patterns in tumors and cancer cells correlate with hypomethylated BST-2 DNA.Our study suggests that the DNA methylation pattern and expression of BST-2 may play a role in disease pathogenesis and could serve as a biomarker for the diagnosis of breast cancer.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25860431" target="_blank">
                    The Expression of DNMT1 in Pathologic Scar Fibroblasts and the Effect of 5-aza-2-Deoxycytidine on Cytokines of Pathologic Scar Fibroblasts.
                </a>
            </h2>
            <p>This study aimed to investigate the expression and significance of DNA methyltransferase 1 (DNMT1) in pathologic scar fibroblasts, as well as the influence of methylase inhibitor, 5-aza-2-deoxycytidine, on pathological scars.Samples of 31 keloids, 20 hypertrophic scars, and 25 normal skins were taken to test the expression rate of DNMT1 by immunohistochemistry. Primary fibroblasts were cultured with the monoplast method. <b>Samples were categorized into the keloid group (K group), 5-aza-2-deoxycytidine keloid intervention group (K+ group), normal skin group (N group), hypertrophic scar group (H group), and 5-aza-2-deoxycytidine hypertrophic scar intervention group (H+ group).</b> <b>The expressions of DNMT1, transforming growth factor-β (TGF-β), and Smad7 mRNA in each group were detected with realtime polymerase chain reaction.</b> The effect of 5-aza-2-deoxycytidine on the cell cycle and apoptosis of pathologic scar fibroblasts were analyzed with flow cytometry.The expression rate of DNMT1 was 100% in keloid fibroblasts, 90% in hyperplastic scar fibroblasts, and 8% in normal skin fibroblasts. <b>After the intervention with 5-aza-2-deoxycytidine in the K+ group, the expression of DNMT1 and TGF-β1 mRNA was lower, Smad7 mRNA was elevated in pathological scar fibroblasts, the flow cytometry showed the proportion of cells in G0/G1 phase were increased, and the proportion of apoptosis cells were also increased, with similar changes in the cells in the H and H+ groups.DNMT1 may play a vital role on the generation of pathological scars.</b> Methylaze inhibitors 5-aza-2-deoxycytidine may influence the related cytokines of pathological scars, inhibit proliferation, and promote apoptosis of pathological scar fibroblasts. The generation of pathological scars may be related with methylation of certain genes. 5-aza-2-deoxycytidine be a new choice for the treatment of pathological scars.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25860355" target="_blank">
                    Genome-Wide Methylation Patterns in Salmonella enterica Subsp. enterica Serovars.
                </a>
            </h2>
            <p>The methylation of DNA bases plays an important role in numerous biological processes including development, gene expression, and DNA replication. Salmonella is an important foodborne pathogen, and methylation in Salmonella is implicated in virulence. Using single molecule real-time (SMRT) DNA-sequencing, we sequenced and assembled the complete genomes of eleven Salmonella enterica isolates from nine different serovars, and analysed the whole-genome methylation patterns of each genome. <b>We describe 16 distinct N6-methyladenine (m6A) methylated motifs, one N4-methylcytosine (m4C) motif, and one combined m6A-m4C motif.</b> <b>Eight of these motifs are novel, i.e., they have not been previously described.</b> <b>We also identified the methyltransferases (MTases) associated with 13 of the motifs.</b> <b>Some motifs are conserved across all Salmonella serovars tested, while others were found only in a subset of serovars.</b> <b>Eight of the nine serovars contained a unique methylated motif that was not found in any other serovar (most of these motifs were part of Type I restriction modification systems), indicating the high diversity of methylation patterns present in Salmonella.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25860294" target="_blank">
                    Genetic and epigenetic profiling of CLL disease progression reveals limited somatic evolution and suggests a relationship to memory-cell development.
                </a>
            </h2>
            <p><b>We examined genetic and epigenetic changes that occur during disease progression from indolent to aggressive forms of chronic lymphocytic leukemia (CLL) using serial samples from 27 patients.</b> Analysis of DNA mutations grouped the leukemia cases into three categories: evolving (26%), expanding (26%) and static (47%). Thus, approximately three-quarters of the CLL cases had little to no genetic subclonal evolution. <b>However, we identified significant recurrent DNA methylation changes during progression at 4752 CpGs enriched for regions near Polycomb 2 repressive complex (PRC2) targets.</b> <b>Progression-associated CpGs near the PRC2 targets undergo methylation changes in the same direction during disease progression as during normal development from naive to memory B cells.</b> <b>Our study shows that CLL progression does not typically occur via subclonal evolution, but that certain CpG sites undergo recurrent methylation changes.</b> <b>Our results suggest CLL progression may involve developmental processes shared in common with the generation of normal memory B cells.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25860220" target="_blank">
                    Value of quantitative and qualitative analyses of circulating cell-free DNA as diagnostic tools for hepatocellular carcinoma: a meta-analysis.
                </a>
            </h2>
            <p>Qualitative and quantitative analyses of circulating cell-free DNA (cfDNA) are potential methods for the detection of hepatocellular carcinoma (HCC). Many studies have evaluated these approaches, but the results have been variable. This meta-analysis is the first to synthesize these published results and evaluate the use of circulating cfDNA values for HCC diagnosis.All articles that met our inclusion criteria were assessed using QUADAS guidelines after the literature research. We also investigated 3 subgroups in this meta-analysis: qualitative analysis of abnormal concentrations of circulating cfDNA; qualitative analysis of single-gene methylation alterations; and multiple analyses combined with alpha-fetoprotein (AFP). Statistical analyses were performed using the software Stata 12.0. We synthesized these published results and calculated accuracy measures (pooled sensitivity and specificity, positive/negative likelihood ratios [PLRs/NLRs], diagnostic odds ratios [DORs], and corresponding 95% confidence intervals [95% CIs]). Data were pooled using bivariate generalized linear mixed model. Furthermore, summary receiver operating characteristic curves and area under the curve (AUC) were used to summarize overall test performance. Heterogeneity and publication bias were also examined.A total of 2424 subjects included 1280 HCC patients in 22 studies were recruited in this meta-analysis. Pooled sensitivity and specificity, PLR, NLR, DOR, AUC, and CIs of quantitative analysis were 0.741 (95% CI: 0.610-0.840), 0.851 (95% CI: 0.718-0.927), 4.970 (95% CI: 2.694-9.169), 0.304 (95% CI: 0.205-0.451), 16.347 (95% CI: 8.250-32.388), and 0.86 (95% CI: 0.83-0.89), respectively. For qualitative analysis, the values were 0.538 (95% CI: 0.401-0.669), 0.944 (95% CI: 0.889-0.972), 9.545 (95% CI: 5.298-17.196), 0.490 (95% CI: 0.372-0.646), 19.491 (95% CI: 10.458-36.329), and 0.87 (95% CI: 0.84-0.90), respectively. After combining with AFP assay, the values were 0.818 (95% CI: 0.676-0.906), 0.960 (95% CI: 0.873-0.988), 20.195 (95% CI: 5.973-68.282), 0.190 (95% CI: 0.100-0.359), 106.270 (95% CI: 22.317-506.055), and 0.96 (95% CI: 0.94-0.97), respectively.The results in this meta-analysis suggest that circulating cfDNA have potential value for HCC diagnosis. However, it would not be recommended for using independently, which is based on the nonrobust results. After combining with AFP, the diagnostic performance will be improved. Further investigation with more data is needed.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25859847" target="_blank">
                    Genome wide DNA methylation profiles provide clues to the origin and pathogenesis of germ cell tumors.
                </a>
            </h2>
            <p>The cell of origin of the five subtypes (I-V) of germ cell tumors (GCTs) are assumed to be germ cells from different maturation stages. This is (potentially) reflected in their methylation status as fetal maturing primordial germ cells are globally demethylated during migration from the yolk sac to the gonad. Imprinted regions are erased in the gonad and later become uniparentally imprinted according to fetal sex. Here, 91 GCTs (type I-IV) and four cell lines were profiled (Illumina's HumanMethylation450BeadChip). Data was pre-processed controlling for cross hybridization, SNPs, detection rate, probe-type bias and batch effects. The annotation was extended, covering snRNAs/microRNAs, repeat elements and imprinted regions. A Hidden Markov Model-based genome segmentation was devised to identify differentially methylated genomic regions. <b>Methylation profiles allowed for separation of clusters of non-seminomas (type II), seminomas/dysgerminomas (type II), spermatocytic seminomas (type III) and teratomas/dermoid cysts (type I/IV).</b> The seminomas, dysgerminomas and spermatocytic seminomas were globally hypomethylated, in line with previous reports and their demethylated precursor. Differential methylation and imprinting status between subtypes reflected their presumed cell of origin. <b>Ovarian type I teratomas and dermoid cysts showed (partial) sex specific uniparental maternal imprinting.</b> The spermatocytic seminomas showed uniparental paternal imprinting while testicular teratomas exhibited partial imprinting erasure. <b>Somatic imprinting in type II GCTs might indicate a cell of origin after global demethylation but before imprinting erasure.</b> <b>This is earlier than previously described, but agrees with the totipotent/embryonic stem cell like potential of type II GCTs and their rare extra-gonadal localization.</b> <b>The results support the common origin of the type I teratomas and show strong similarity between ovarian type I teratomas and dermoid cysts.</b> In conclusion, we identified specific and global methylation differences between GCT subtypes, providing insight into their developmental timing and underlying developmental biology. Data and extended annotation are deposited at GEO (GSE58538 and GPL18809).</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25859290" target="_blank">
                    Differences in DNA methylation profile of Th1 and Th2 cytokine genes are associated with tolerance acquisition in children with IgE-mediated cow's milk allergy.
                </a>
            </h2>
            <p>Epigenetic changes in DNA methylation could regulate the expression of several allergy-related genes. We investigated whether tolerance acquisition in children with immunoglobulin E (IgE)-mediated cow's milk allergy (CMA) is characterized by a specific DNA methylation profile of Th2 (IL-4, IL-5) and Th1 (IL-10, IFN-γ)-associated cytokine genes.DNA methylation of CpGs in the promoting regions of genes from peripheral blood mononuclear cells and serum level of IL-4, IL-5, IL-10 and INF-γ were assessed in children with active IgE-mediated CMA (group 1), in children who acquired tolerance to cow's milk proteins (group 2) and in healthy children (group 3). Forty children (24 boys, aged 3 to 18&nbsp;months) were enrolled: 10 in group 1, 20 in group 2, and 10 in the control group. <b>The DNA methylation profiles clearly separated active CMA patients from healthy controls.</b> We observed an opposite pattern comparing subjects with active IgE-mediated CMA with healthy controls and group 2 children who outgrew CMA. <b>The IL-4 and IL-5 DNA methylation was significantly lower, and IL-10 and INF-γ DNA methylation was higher in active IgE-mediated CMA patients.</b> <b>Gene promoter DNA methylation rates of all cytokines and respective serum levels were strongly correlated.</b> <b>Formula selection significantly influenced cytokine DNA methylation profiles in group 2.Tolerance acquisition in children with IgE-mediated CMA is characterized by a distinct Th1 and Th2 cytokine gene DNA methylation pattern.</b> <b>These results suggest that DNA methylation may be a target for CMA prevention and treatment.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25859289" target="_blank">
                    Differential DNA methylation of genes involved in fibrosis progression in non-alcoholic fatty liver disease and alcoholic liver disease.
                </a>
            </h2>
            <p><b>Chronic liver injury can lead to the development of liver fibrosis and cirrhosis but only in a minority of patients.</b> Currently, it is not clear which factors determine progression to fibrosis. <b>We investigated whether DNA\methylation profile as determined by pyrosequencing can distinguish patients with mild from those with advanced/severe fibrosis in non-alcoholic liver disease (NAFLD) and alcoholic liver disease (ALD).</b> <b>To this end, paraffin-embedded liver biopsies were collected from patients with biopsy-proven NAFLD or ALD, as well as paraffin-embedded normal liver resections, genomic DNA isolated, bisulfite converted and pyrosequencing assays used to quantify DNA methylation at specific CpGs within PPARα, PPARα, TGFβ1, Collagen 1A1 and PDGFα genes.</b> <b>Furthermore, we assessed the impact of age, gender and anatomical location within the liver on patterns of DNA methylation in the same panel of genes.DNA methylation at specific CpGs within genes known to affect fibrogenesis distinguishes between patients with mild from those with severe fibrosis in both NAFLD and ALD, although same CpGs are not equally represented in both etiologies.</b> <b>In normal liver, age, gender or anatomical location had no significant impact on DNA methylation patterns in the liver.DNA methylation status at specific CpGs may be useful as part of a wider set of patient data for predicting progression to liver fibrosis.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25858139" target="_blank">
                    Role of fumonisin B1 on DNA methylation changes in rat kidney and liver cells.
                </a>
            </h2>
            <p><b>Fumonisin B1 (FB1) is a mycotoxin produced by Fusarium verticillioides (Sacc.</b> ) Nirenberg (Nectriaceae) mold that contaminates maize and other agricultural products. <b>Although the effects of FB1 on sphingolipid metabolism are clear, little is known about early molecular changes associated with FB1 carcinogenicity.Alteration on DNA methylation, as an early event in non-genotoxic carcinogenesis, may play an important role in the mechanism of FB1 toxiciy.Dose-related effects of FB1 (1-50 µM for 24 h) on global DNA methylation by using high-performance liquid chromatography with UV-diode array detection (HPLC-UV/DAD) and CpG promoter methylation by methylation-specific PCR (MSP) were performed in rat liver (Clone 9) and rat kidney (NRK-52E) epithelial cells.Cell viability reduction is 39% and 34% by the XTT test and LDH release in the growth medium is 32% and 26% at 200 µM of FB1 treatment in Clone 9 and NRK-52E cells, respectively.</b> <b>No significant dose-related effects of FB1 on global DNA methylation which ranged from 4 to 5% were observed in both cells compared with controls.</b> <b>Promoter regions of c-myc gene were methylated (&gt;33%) at 10 and 50 µM of FB1 treatment in Clone 9 cells while it was unmethylated in NRK-52E cells.</b> <b>Promoter regions of p15 gene were unmethylated while VHL gene were found to be methylated (&gt;33%) at 10, 25, and 50 µM and 10 and 50 µM of FB1 treatment in Clone 9 and NRK-52E cells, respectively.Alteration in DNA methylation might play an important role in the toxicity of FB1 in risk assessment process.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25857739" target="_blank">
                    Association of in vitro fertilization with global and IGF2/H19 methylation variation in newborn twins.
                </a>
            </h2>
            <p>In vitro fertilization (IVF) and its subset intracytoplasmic sperm injection (ICSI), are widely used medical treatments for conception. There has been controversy over whether IVF is associated with adverse short- and long-term health outcomes of offspring. As with other prenatal factors, epigenetic change is thought to be a molecular mediator of any in utero programming effects. Most studies focused on DNA methylation at gene-specific and genomic level, with only a few on associations between DNA methylation and IVF. Using buccal epithelium from 208 twin pairs from the Peri/Postnatal Epigenetic Twin Study (PETS), we investigated associations between IVF and DNA methylation on a global level, using the proxies of Alu and LINE-1 interspersed repeats in addition to two locus-specific regulatory regions within IGF2/H19, controlling for 13 potentially confounding factors. <b>Using multiple correction testing, we found strong evidence that IVF-conceived twins have lower DNA methylation in Alu, and weak evidence of lower methylation in one of the two IGF2/H19 regulatory regions and LINE-1, compared with naturally conceived twins.</b> <b>Weak evidence of a relationship between ICSI and DNA methylation within IGF2/H19 regulatory region was found, suggesting that one or more of the processes associated with IVF/ICSI may contribute to these methylation differences.</b> <b>Lower within- and between-pair DNA methylation variation was also found in IVF-conceived twins for LINE-1, Alu and one IGF2/H19 regulatory region.</b> <b>Although larger sample sizes are needed, our results provide additional insight to the possible influence of IVF and ICSI on DNA methylation.</b> <b>To our knowledge, this is the largest study to date investigating the association of IVF and DNA methylation.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25857738" target="_blank">
                    Maternal diet as a modifier of offspring epigenetics.
                </a>
            </h2>
            <p><b>There has been a substantial body of evidence, which has shown that genetic variation is an important determinant of disease risk.</b> <b>However, there is now increasing evidence that alterations in epigenetic processes also play a role in determining susceptibility to disease.</b> Epigenetic processes, which include DNA methylation, histone modifications and non-coding RNAs play a central role in regulating gene expression, determining when and where a gene is expressed as well as the level of gene expression. The epigenome is highly sensitive to a variety of environmental factors, especially in early life. One factor that has been shown consistently to alter the epigenome is maternal diet. <b>This review will focus on how maternal diet can modify the epigenome of the offspring, producing different phenotypes and altered disease susceptibilities.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25857663" target="_blank">
                    The recently identified modifier of murine metastable epialleles, Rearranged L-Myc Fusion, is involved in maintaining epigenetic marks at CpG island shores and enhancers.
                </a>
            </h2>
            <p><b>We recently identified a novel protein, Rearranged L-myc fusion (Rlf), that is required for DNA hypomethylation and transcriptional activity at two specific regions of the genome known to be sensitive to epigenetic gene silencing.</b> <b>To identify other loci affected by the absence of Rlf, we have now analysed 12 whole genome bisulphite sequencing datasets across three different embryonic tissues/stages from mice wild-type or null for Rlf.Here we show that the absence of Rlf results in an increase in DNA methylation at thousands of elements involved in transcriptional regulation and many of the changes occur at enhancers and CpG island shores.</b> <b>ChIP-seq for H3K4me1, a mark generally found at regulatory elements, revealed associated changes at many of the regions that are differentially methylated in the Rlf mutants.</b> <b>RNA-seq showed that the numerous effects of the absence of Rlf on the epigenome are associated with relatively subtle effects on the mRNA population.</b> <b>In vitro studies suggest that Rlf's zinc fingers have the capacity to bind DNA and that the protein interacts with other known epigenetic modifiers.This study provides the first evidence that the epigenetic modifier Rlf is involved in the maintenance of DNA methylation at enhancers and CGI shores across the genome.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25857630" target="_blank">
                    NF-κB and STAT3 transcription factor signatures differentiate HPV-positive and HPV-negative Head and neck squamous cell carcinoma.
                </a>
            </h2>
            <p>Using high-throughput analyses and the TRANSFAC database, we characterized TF signatures of head and neck squamous cell carcinoma (HNSCC) subgroups by inferential analysis of target gene expression, correcting for the effects of DNA methylation and copy number. <b>Using this discovery pipeline, we determined that human papillomavirus-related (HPV+) and HPV- HNSCC differed significantly based on the activity levels of key TFs including AP1, STATs, NF-κB, and p53.</b> <b>Immunohistochemical analysis confirmed that HPV- HNSCC is characterized by co-activated STAT3 and NF-κB pathways, and functional studies demonstrate that this phenotype can be effectively targeted with combined anti-NF-κB and anti-STAT therapies.</b> <b>These discoveries correlate strongly with previous findings connecting STATs, NF-κB, and AP1 in HNSCC.</b> <b>We identified 5 top-scoring pair biomarkers from STATs, NF-κB and AP1 pathways that distinguish HPV+ from HPV- HNSCC based on TF activity, and validated these biomarkers on TCGA and on independent validation cohorts.</b> <b>We conclude that a novel approach to TF pathway analysis can provide insight into therapeutic targeting of patient subgroup for heterogeneous disease such as HNSCC.</b> This article is protected by copyright. All rights reserved.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25857453" target="_blank">
                    Epigenetic targets and drug discovery Part 2: Histone demethylation and DNA methylation.
                </a>
            </h2>
            <p>Chromatin structure is dynamically modulated by various chromatin modifications, such as histone/DNA methylation and demethylation. <b>We have reviewed histone methyltransferases and methyllysine binders in terms of their small 7molecule screening and drug discovery in the first part of this review series.</b> <b>In this part, we will summarize recent progress in chemical probe and drug discovery of histone demethylases and DNA methyltransferases.</b> <b>Histone demethylation and DNA methylation has attracted a lot of attention regarding their biology and disease implications.</b> <b>Correspondingly, many small molecule compounds have been designed to modulate the activity of histone demethylases and DNA methyltransferases, and some of them have been developed into therapeutic drugs or put into clinical trials.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25857441" target="_blank">
                    Establishing pluripotency in early development.
                </a>
            </h2>
            <p><b>The earliest steps of embryonic development involve important changes in chromatin and transcription factor networks, which are orchestrated to establish pluripotent cells that will form the embryo.</b> DNA methylation, histone modifications, the pluripotency regulatory network of transcription factors, maternal factors and newly translated proteins all contribute to these transitions in dynamic ways. Moreover, these dynamics are linked to the onset of zygotic transcription. <b>We will review recent progress in our understanding of chromatin state and regulation of gene expression in the context of embryonic development in vertebrates, in particular mouse, Xenopus and zebrafish.</b> <b>We include work on mouse embryonic stem cells and highlight work that illustrates how early embryonic dynamics establish gene regulatory networks and the state of pluripotency.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25856389" target="_blank">
                    A Comparison of Genome-Wide DNA Methylation Patterns between Different Vascular Tissues from Patients with Coronary Heart Disease.
                </a>
            </h2>
            <p>Epigenetic mechanisms of gene regulation in context of cardiovascular diseases are of considerable interest. So far, our current knowledge of the DNA methylation profiles for atherosclerosis affected and healthy human vascular tissues is still limited. <b>Using the Illumina Infinium Human Methylation27 BeadChip, we performed a genome-wide analysis of DNA methylation in right coronary artery in the area of advanced atherosclerotic plaques, atherosclerotic-resistant internal mammary arteries, and great saphenous veins obtained from same patients with coronary heart disease.</b> The resulting DNA methylation patterns were markedly different between all the vascular tissues. <b>The genes hypomethylated in athero-prone arteries to compare with atherosclerotic-resistant arteries were predominately involved in regulation of inflammation and immune processes, as well as development.</b> <b>The great saphenous veins exhibited an increase of the DNA methylation age in comparison to the internal mammary arteries.</b> Gene ontology analysis for genes harboring hypermethylated CpG-sites in veins revealed the enrichment for biological processes associated with the development. <b>Four CpG-sites located within the MIR10B gene sequence and about 1 kb upstream of the HOXD4 gene were also confirmed as hypomethylated in the independent dataset of the right coronary arteries in the area of advanced atherosclerotic plaques in comparison with the other vascular tissues.</b> The DNA methylation differences observed in vascular tissues of patients with coronary heart disease can provide new insights into the mechanisms underlying the development of pathology and explanation for the difference in graft patency after coronary artery bypass grafting surgery.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25855964" target="_blank">
                    PHRF1 promotes genome integrity by modulating non-homologous end-joining.
                </a>
            </h2>
            <p>Methylated histone readers are critical for chromatin dynamics, transcription, and DNA repair. Human PHRF1 contains a plant homeodomain (PHD) that recognizes methylated histones and a RING domain, which ubiquitinates substrates. A recent study reveals that PHRF1 is a tumor suppressor that promotes TGF-β cytostatic signaling through TGIF ubiquitination. Also, PHRF1 is a putative phosphorylation substrate of ataxia telangiectasia-mutated/ataxia telangiectasia and Rad3-related kinases; however, the role of PHRF1 in DNA damage response is unclear. Here we report a novel function of PHRF1 in modulating non-homologous end-joining (NHEJ). PHRF1 quickly localizes to DNA damage lesions upon genotoxic insults. <b>Ablation of PHRF1 decreases the efficiency of plasmid-based end-joining, whereas PHRF1 overexpression leads to an elevated NHEJ in H1299 reporter cells.</b> <b>Immunoprecipitation and peptide pull-down assays verify that PHRF1 constitutively binds to di- and trimethylated histone H3 lysine 36 (H3K36) (H3K36me2 and H3K36me3) via its PHD domain.</b> <b>Substitution of S(915)DT(917)E to ADAE in PHRF1 decreases its affinity for NBS1.</b> Both PHD domain and SDTE motif are required for its NHEJ-promoting activity. <b>Furthermore, PHRF1 mediates PARP1 polyubiquitination for proteasomal degradation.</b> <b>These results suggest that PHRF1 may combine with H3K36 methylation and NBS1 to promote NHEJ and stabilize genomic integrity upon DNA damage insults.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25855800" target="_blank">
                    Infrequent DNA methylation of miR-9-1 and miR-9-3 in multiple myeloma.
                </a>
            </h2>
            <p>The miR-9 family microRNAs (miRNAs) are tumour suppressor miRNAs implicated in carcinogenesis. We postulated that miR-9-1, miR-9-2 and miR-9-3 may be inactivated by aberrant promoter methylation in multiple myeloma (MM).Methylation of miR-9-1, miR-9-2 and miR-9-3 was studied by methylation-specific PCR (MSP) in six normal controls, including three each of healthy peripheral blood (PB) or bone marrow buffy coat, 10 MM cell lines, 62 primary MM marrow samples at diagnosis and 22 at relapse/progression.MSP, verified by quantitative pyrosequencing, showed that the promoters of miR-9-3 and miR-9-1 were unmethylated in normal controls but methylated in 4 (40%) and 5 (50%) of 10 MM cell lines, respectively. However, the promoter of miR-9-2 was methylated in three normal PB buffy coat controls and in CD138-sorted healthy marrow plasma cells, indicating possibly tissue specific rather than tumour-specific methylation of miR-9-2, which was thus not studied further. In WL-2 cells, which were completely methylated for miR-9-3, 5-aza-2'-deoxycytidine treatment caused miR-9-3 promoter demethylation and pri-miR-9-3 re-expression. In primary samples, methylation of miR-9-3 was detected in 1 of 62 patients at diagnosis and 1 of 22 patients at relapse/progression. However, miR-9-1 methylation was absent in both primary samples at diagnosis and at relapse/progression.Hypermethylation of miR-9-3 and miR-9-1 is tumour-specific in MM, leading to reversible miRNA silencing. Frequent methylation of miR-9-3 and miR-9-1 in cell lines, but not in primary samples, may be acquired during in vitro culture, and indicates an unimportant role of miR-9 methylation in myelomagenesis.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25855720" target="_blank">
                    Maternal pre-pregnancy BMI and gestational weight gain, offspring DNA methylation and later offspring adiposity: findings from the Avon Longitudinal Study of Parents and Children.
                </a>
            </h2>
            <p>Evidence suggests that in utero exposure to undernutrition and overnutrition might affect adiposity in later life. <b>Epigenetic modification is suggested as a plausible mediating mechanism.We used multivariable linear regression and a negative control design to examine offspring epigenome-wide DNA methylation in relation to maternal and offspring adiposity in 1018 participants.Compared with neonatal offspring of normal weight mothers, 28 and 1621 CpG sites were differentially methylated in offspring of obese and underweight mothers, respectively [false discovert rate (FDR)-corrected P-value &lt; 0.05), with no overlap in the sites that maternal obesity and underweight relate to.</b> A positive association, where higher methylation is associated with a body mass index (BMI) outside the normal range, was seen at 78.6% of the sites associated with obesity and 87.9% of the sites associated with underweight. <b>Associations of maternal obesity with offspring methylation were stronger than associations of paternal obesity, supporting an intrauterine mechanism.</b> There were no consistent associations of gestational weight gain with offspring DNA methylation. <b>In general, sites that were hypermethylated in association with maternal obesity or hypomethylated in association with maternal underweight tended to be positively associated with offspring adiposity, and sites hypomethylated in association with maternal obesity or hypermethylated in association with maternal underweight tended to be inversely associated with offspring adiposity.Our data suggest that both maternal obesity and, to a larger degree, underweight affect the neonatal epigenome via an intrauterine mechanism, but weight gain during pregnancy has little effect.</b> <b>We found some evidence that associations of maternal underweight with lower offspring adiposity and maternal obesity with greater offspring adiposity may be mediated via increased DNA methylation.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25855717" target="_blank">
                    The Inheritance of Acquired Epigenetic Variations.
                </a>
            </h2>
            <p>There is evidence that the functional history of a gene in one generation can influence its expression in the next. In somatic cells, changes in gene activity are frequently associated with changes in the pattern of methylation of the cytosines in DNA; these methylation patterns are stably inherited. Recent work suggests that information about patterns of methylation and other epigenetic states can also be transmitted from parents to offspring. This evidence is the basis of a model for the inheritance of acquired epigenetic variations. <b>According to the model, an environmental stimulus can induce heritable chromatin modifications which are very specific and predictable, and might result in an adaptive response to the stimulus.</b> This type of response probably has most significance for adaptive evolution in organisms such as fungi and plants, which lack distinct segregation of the soma and germ line. <b>However, in all organisms, the accumulation of specific and random chromatin modifications in the germ line may be important in speciation, because these modifications could lead to reproductive isolation between populations.</b> <b>Heritable chromatin variations may also alter the frequency and distribution of classical mutations and meiotic recombination.</b> <b>Therefore, inherited epigenetic changes in the structure of chromatin can influence neo-Darwinian evolution as well as cause a type of "Lamarckian" inheritance.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25855716" target="_blank">
                    Exposure to aflatoxin B1 in utero is associated with DNA methylation in white blood cells of infants in The Gambia.
                </a>
            </h2>
            <p>Exposure to environmental toxins during embryonic development may lead to epigenetic changes that influence disease risk in later life. <b>Aflatoxin is a contaminant of staple foods in sub-Saharan Africa, is a known human liver carcinogen and has been associated with stunting in infants.We have measured aflatoxin exposure in 115 pregnant women in The Gambia and examined the DNA methylation status of white blood cells from their infants at 2-8 months old (mean 3.6 ± 0.9).</b> <b>Aflatoxin exposure in women was assessed using an ELISA method to measure aflatoxin albumin (AF-alb) adducts in plasma taken at 1-16 weeks of pregnancy.</b> Genome-wide DNA methylation of infant white blood cells was measured using the Illumina Infinium HumanMethylation450beadchip.AF-alb levels ranged from 3.9 to 458.4 pg/mg albumin. <b>We found that aflatoxin exposure in the mothers was associated to DNA methylation in their infants for 71 CpG sites (false discovery rate &lt; 0.05), with an average effect size of 1.7% change in methylation.</b> Aflatoxin-associated differential methylation was observed in growth factor genes such as FGF12 and IGF1, and immune-related genes such as CCL28, TLR2 and TGFBI. <b>Moreover, one aflatoxin-associated methylation region (corresponding to the miR-4520b locus) was identified.This study shows that maternal exposure to aflatoxin during the early stages of pregnancy is associated with differential DNA methylation patterns of infants, including in genes related to growth and immune function.</b> <b>This reinforces the need for interventions to reduce aflatoxin exposure, especially during critical periods of fetal and infant development.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25855514" target="_blank">
                    Impact of a folic acid enriched diet on urinary tract function in mice treated with testosterone and estradiol.
                </a>
            </h2>
            <p>Aging men are susceptible to developing lower urinary tract symptoms, but the underlying etiology is unknown and the influence of dietary and environmental factors on them is unclear. We tested whether a folic acid enriched diet changed urinary tract physiology and biology in control male mice and male mice with urinary dysfunction induced by exogenous testosterone and estradiol (T+E2), which mimics changing hormone levels in aging humans. T+E2 treatment increased mouse urine output, time between voiding events, bladder capacity and compliance. <b>Consumption of a folic acid enriched diet moderated these changes without decreasing prostate wet weight or threshold voiding pressure.</b> One potential mechanism for these changes involves water balance. <b>T+E2 treatment increases plasma concentrations of anti-diuretic hormone, which is offset at least in part by a folic acid enriched diet.</b> Another potential mechanism involves neural control of micturition. <b>The folic acid enriched diet, fed to T+E2 treated mice, increased voiding frequency in response to intravesicular capsaicin infusion and increased mRNA abundance of capsaicin-sensitive cation channel, transient receptor potential vanilloid subfamily member 1 (Trpv1), in L6 and S1 dorsal root ganglia (DRG) neurons.</b> <b>T+E2 treatment and a folic acid enriched diet also modified DNA methylation, which is capable of altering gene expression.</b> We found the enriched diet increased global DNA methylation in dorsal and ventral prostate and L6 and S1 DRG. <b>Our results are consistent with folic acid acting to slow or reverse T+E2-mediated alteration in urinary function in part by normalizing water balance and enhancing or preserving afferent neuronal function.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25855280" target="_blank">
                    [Properties and biological roles of TET proteins during embryogenesis and in hematopoiesis].
                </a>
            </h2>
            <p><b>DNA methylation is associated with a large number of biological processes and mainly concerns the cytosine methylation at position 5 (5-mC).</b> An active demethylation mechanism was highlighted in 2009 following the discovery that TET proteins were enzymes implicated in the hydroxylation of 5-mC to 5-hydroxymethylcytosine. Simultaneously, other studies showed frequent acquired TET2 mutations in hematological malignancies and have depicted their role in their pathogenesis. An entire field of research has developed rapidly showing that these proteins are involved in many biological processes.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25855017" target="_blank">
                    Variants in MTHFR gene and neural tube defects susceptibility in China.
                </a>
            </h2>
            <p><b>Neural tube defect (NTD) is a severe congenital birth abnormalities involving incomplete neural tube closure.</b> 5, 10-methylenetetrahydrofolate reductase (MTHFR) gene plays key role in folate cycle and methylation cycle, which could affect the DNA synthesis, repair and methylation. In this study, we aim to investigate the correlation between MTHFR polymorphisms and NTD-affected pregnancy. There were 444 participants involved in our study. Tag-SNPs were identified in HapMap Databases. Blood samples were collected from all subjects to further extract the genomic DNAs by TaqMan Blood DNA kits. <b>We also carried out a meta-analysis based on previous published studies to further examine the association between MTHFR polymorphisms and NTD.</b> <b>In case-control study analysis, two SNPs were identified to be associated with NTD risk.</b> The 677 C &gt; T genetic variant was correlated with increased risk of NTD-affected pregnancy. However, the 1298 A &gt; C polymorphism was shown to lower the risk of NTD-affected pregnancy. The protective role of 1298 A &gt; C polymorphisms was further supported by the result of meta-analysis. <b>Our study revealed that the SNPs of 677C &gt; T and 1298A &gt; C in MTHFR were associated with NTD-affected pregnancy, in which 677C &gt; T was a risk factor and in contrast 1298A &gt; C was protective factor against NTD.</b> <b>Our results of meta-analysis also revealed the 1298A &gt; C MTHFR polymorphism play protective role in NTD.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25854584" target="_blank">
                    Differential DNA methylation and expression of inflammatory and zinc transporter genes defines subgroups of osteoarthritic hip patients.
                </a>
            </h2>
            <p>We have previously shown that the cartilage DNA methylome delineates two clusters of osteoarthritic (OA) hip patients, characterised by differential methylation of inflammatory genes, while others have demonstrated a link between zinc homeostasis and inflammation in OA. <b>We aimed to investigate these effects at the methylation and gene expression level.We used our previously generated methylation data while quantitative PCR was used to measure gene expression using RNA from the hip cartilage of members of both clusters and from control individuals without hip OA.One of the OA clusters is characterised by the promoter hypomethylation and increased expression of inflammation-associated genes including IL1A and TNF.</b> <b>Furthermore, we show that the increase in expression of these genes is accompanied by increased expression of several zinc transporter genes.</b> <b>In addition, the zinc responsive transcription factor MTF1 is also upregulated, which is accompanied by an increase in the expression of its targets the metalloproteinases MMP13 and ADAMTS5.We have identified a subgroup of OA hip patients that are epigenetically and transcriptiomically characterised by a cartilage inflammatory phenotype with concurrent differential regulation of zinc regulators.</b> The identification of subgroups enhances stratified phenotyping of OA patients and has important implications for future therapeutic applications.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25854397" target="_blank">
                    Application of multiplex nested methylated specific PCR in early diagnosis of epithelial ovarian cancer.
                </a>
            </h2>
            <p>To explore the application of multiplex nested methylated specific polymerase chain reaction (PCR) in the early diagnosis of epithelial ovarian carcinoma (EOC).Serum and fresh tissue samples were collected from 114 EOC patients. RUNX3, TFPI2 and OPCML served as target genes. <b>Methylation levels of tissues were assessed by multiplex nested methylated specific PCR, the results being compared with those for carcinoma antigen 125 (CA125).The serum free deoxyribose nucleic acid (DNA) methylation spectrum of EOC patients was completely contained in the DNA spectrum of cancer tissues, providing an accurate reflection of tumor DNA methylation conditions.</b> <b>Serum levels of CA125 and free DNA methylation in the EOC group were evidently higher than those in benign lesion and control groups (p&lt;0.05).</b> <b>Patients with early EOC had markedly lower serum CA125 than those with advanced EOC (p&lt;0.05), but there was no significant difference in free DNA methylation (p&gt;0.05).</b> <b>The sensitivity, specificity and positive predicative value (PPV) of multiplex nested methylated specific PCR were significantly higher for detection of all patients and those with early EOC than those for CA125 (p&lt;0.05).</b> <b>In the detection of patients with advanced EOC, the PPV of CA125 detection was obviously lower than that of multiplex nested methylated specific PCR (p&gt;0.05), but there was no significant difference in sensitivity (p&gt;0.05).Serum free DNA methylation can be used as a biological marker for EOC and multiplex nested methylated specific PCR should be considered for early diagnosis since it can accurately determine tumor methylation conditions.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25854114" target="_blank">
                    [Epigenetic mechanisms in post-stress states].
                </a>
            </h2>
            <p><b>There is a continued interest in the research of epigenetic mechanisms of genes regulation in the brain, associated with the stress-induced adaptive and pathological consequences during the formation of post-stress states, in particular post-traumatic stress disorder (PTSD).</b> Brief overview of epigenetic modifications was presented, while the process of DNA methylation was discussed comprehensively. <b>Key stress-induced epigenetic changes in the human and animal brain in response to stress were summarized.</b> <b>Epigenetic modifications in certain genes (genes of stress responses, hormones and neurotransmitters genes, NGF gene) while post-stress states and PTSD formation were discussed in detail.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25853800" target="_blank">
                    5-hydroxymethylcytosine marks promoters in colon that resist DNA hypermethylation in cancer.
                </a>
            </h2>
            <p><b>The discovery of cytosine hydroxymethylation (5hmC) as a mechanism that potentially controls DNA methylation changes typical of neoplasia prompted us to investigate its behaviour in colon cancer.</b> <b>5hmC is globally reduced in proliferating cells such as colon tumours and the gut crypt progenitors, from which tumours can arise.Here, we show that colorectal tumours and cancer cells express Ten-Eleven-Translocation (TET) transcripts at levels similar to normal tissues.</b> <b>Genome-wide analyses show that promoters marked by 5hmC in normal tissue, and those identified as TET2 targets in colorectal cancer cells, are resistant to methylation gain in cancer.</b> In vitro studies of TET2 in cancer cells confirm that these promoters are resistant to methylation gain independently of sustained TET2 expression. <b>We also find that a considerable number of the methylation gain-resistant promoters marked by 5hmC in normal colon overlap with those that are marked with poised bivalent histone modifications in embryonic stem cells.Together our results indicate that promoters that acquire 5hmC upon normal colon differentiation are innately resistant to neoplastic hypermethylation by mechanisms that do not require high levels of 5hmC in tumours.</b> Our study highlights the potential of cytosine modifications as biomarkers of cancerous cell proliferation.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25853737" target="_blank">
                    Disputing Lamarckian epigenetic inheritance in mammals.
                </a>
            </h2>
            <p><b>A recent study finds that changes to transcription and DNA methylation resulting from in utero exposure to environmental endocrine-disrupting chemicals are not inherited across generations.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25853498" target="_blank">
                    A PHF8 Homolog in C. elegans Promotes DNA Repair via Homologous Recombination.
                </a>
            </h2>
            <p>PHF8 is a JmjC domain-containing histone demethylase, defects in which are associated with X-linked mental retardation. In this study, we examined the roles of two PHF8 homologs, JMJD-1.1 and JMJD-1.2, in the model organism C. elegans in response to DNA damage. A deletion mutation in either of the genes led to hypersensitivity to interstrand DNA crosslinks (ICLs), while only mutation of jmjd-1.1 resulted in hypersensitivity to double-strand DNA breaks (DSBs). In response to ICLs, JMJD-1.1 did not affect the focus formation of FCD-2, a homolog of FANCD2, a key protein in the Fanconi anemia pathway. However, the dynamic behavior of RPA-1 and RAD-51 was affected by the mutation: the accumulations of both proteins at ICLs appeared normal, but their subsequent disappearance was retarded, suggesting that later steps of homologous recombination were defective. Similar changes in the dynamic behavior of RPA-1 and RAD-51 were seen in response to DSBs, supporting a role of JMJD-1.1 in homologous recombination. Such a role was also supported by our finding that the hypersensitivity of jmjd-1.1 worms to ICLs was rescued by knockdown of lig-4, a homolog of Ligase 4 active in nonhomologous end-joining. The hypersensitivity of jmjd-1.1 worms to ICLs was increased by rad-54 knockdown, suggesting that JMJD-1.1 acts in parallel with RAD-54 in modulating chromatin structure. Indeed, the level of histone H3 Lys9 tri-methylation, a marker of heterochromatin, was higher in jmjd-1.1 cells than in wild-type cells. We conclude that the histone demethylase JMJD-1.1 influences homologous recombination either by relaxing heterochromatin structure or by indirectly regulating the expression of multiple genes affecting DNA repair.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25853433" target="_blank">
                    Deleterious effects of endocrine disruptors are corrected in the mammalian germline by epigenome reprogramming.
                </a>
            </h2>
            <p>Exposure to environmental endocrine-disrupting chemicals during pregnancy reportedly causes transgenerationally inherited reproductive defects. <b>We hypothesized that to affect the grandchild, endocrine-disrupting chemicals must alter the epigenome of the germ cells of the in utero-exposed G1 male fetus.</b> <b>Additionally, to affect the great-grandchild, the aberration must persist in the germ cells of the unexposed G2 grandchild.Here, we treat gestating female mice with vinclozolin, bisphenol A, or di-(2-ethylhexyl)phthalate during the time when global de novo DNA methylation and imprint establishment occurs in the germ cells of the G1 male fetus.</b> <b>We map genome-wide features in purified G1 and G2 prospermatogonia, in order to detect immediate and persistent epigenetic aberrations, respectively.</b> <b>We detect changes in transcription and methylation in the G1 germline immediately after endocrine-disrupting chemicals exposure, but changes do not persist into the G2 germline.</b> <b>Additional analysis of genomic imprints shows no persistent aberrations in DNA methylation at the differentially methylated regions of imprinted genes between the G1 and G2 prospermatogonia, or in the allele-specific transcription of imprinted genes between the G2 and G3 soma.Our results suggest that endocrine-disrupting chemicals exert direct epigenetic effects in exposed fetal germ cells, which are corrected by reprogramming events in the next generation.</b> Avoiding transgenerational inheritance of environmentally-caused epigenetic aberrations may have played an evolutionary role in the development of dual waves of global epigenome reprogramming in mammals.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25853392" target="_blank">
                    A coherent approach for analysis of the Illumina HumanMethylation450 BeadChip improves data quality and performance in epigenome-wide association studies.
                </a>
            </h2>
            <p><b>DNA methylation plays a fundamental role in the regulation of the genome, but the optimal strategy for analysis of genome-wide DNA methylation data remains to be determined.</b> <b>We developed a comprehensive analysis pipeline for epigenome-wide association studies (EWAS) using the Illumina Infinium HumanMethylation450 BeadChip, based on 2,687 individuals, with 36 samples measured in duplicate.</b> We propose new approaches to quality control, data normalisation and batch correction through control-probe adjustment and establish a null hypothesis for EWAS using permutation testing. <b>Our analysis pipeline outperforms existing approaches, enabling accurate identification of methylation quantitative trait loci for hypothesis driven follow-up experiments.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25852480" target="_blank">
                    An epigenetic hypothesis for the genomic memory of pain.
                </a>
            </h2>
            <p>Chronic pain is accompanied with long-term sensory, affective and cognitive disturbances. What are the mechanisms that mediate the long-term consequences of painful experiences and embed them in the genome? <b>We hypothesize that alterations in DNA methylation, an enzymatic covalent modification of cytosine bases in DNA, serve as a "genomic" memory of pain in the adult cortex.</b> <b>DNA methylation is an epigenetic mechanism for long-term regulation of gene expression.</b> Neuronal plasticity at the neuroanatomical, functional, morphological, physiological and molecular levels has been demonstrated throughout the neuroaxis in response to persistent pain, including in the adult prefrontal cortex (PFC). <b>We have previously reported widespread changes in gene expression and DNA methylation in the PFC many months following peripheral nerve injury.</b> <b>In support of this hypothesis, we show here that up-regulation of a gene involved with synaptic function, Synaptotagmin II (syt2), in the PFC in a chronic pain model is associated with long-term changes in DNA methylation.</b> <b>The challenges of understanding the contributions of epigenetic mechanisms such as DNA methylation within the PFC to pain chronicity and their therapeutic implications are discussed.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25852443" target="_blank">
                    Connecting the CNTNAP2 Networks with Neurodevelopmental Disorders.
                </a>
            </h2>
            <p>Based on genomic rearrangements and copy number variations, the contactin-associated protein-like 2 gene (CNTNAP2) has been implicated in neurodevelopmental disorders such as Gilles de la Tourette syndrome, intellectual disability, obsessive compulsive disorder, cortical dysplasia-focal epilepsy syndrome, autism, schizophrenia, Pitt-Hopkins syndrome, and attention deficit hyperactivity disorder. <b>To explain the phenotypic pleiotropy of CNTNAP2 alterations, several hypotheses have been put forward.</b> Those include gene disruption, loss of a gene copy by a heterozygous deletion, altered regulation of gene expression due to loss of transcription factor binding and DNA methylation sites, and mutations in the amino acid sequence of the encoded protein which may provoke altered interactions of the CNTNAP2-encoded protein, Caspr2, with other proteins. <b>Also exome sequencing, which covers &lt;0.2% of the CNTNAP2 genomic DNA, has revealed numerous single nucleotide variants in healthy individuals and in patients with neurodevelopmental disorders.</b> <b>In some of these disorders, disruption of CNTNAP2 may be interpreted as a susceptibility factor rather than a directly causative mutation.</b> <b>In addition to being associated with impaired development of language, CNTNAP2 may turn out to be a central node in the molecular networks controlling neurodevelopment.</b> <b>This review discusses the impact of CNTNAP2 mutations on its functioning at multiple levels of the combinatorial genetic networks that govern brain development.</b> In addition, recommendations for genomic testing in the context of clinical genetic management of patients with neurodevelopmental disorders and their families are put forward.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25852282" target="_blank">
                    Methylation of IRAK3 is a novel prognostic marker in hepatocellular carcinoma.
                </a>
            </h2>
            <p>To examine the methylation levels of interleukin-1 receptor-associated kinase 3 (IRAK3) and GLOXD1 and their potential clinical applications in hepatocellular carcinoma (HCC).mRNA expression and promoter methylation of IRAK3 and GLOXD1 in HCC cells were analyzed by reverse transcription-polymerase chain reaction (RT-PCR) and methylation-specific PCR (MSP), respectively. Using pyrosequencing results, we further established a quantitative MSP (Q-MSP) system for the evaluation of IRAK3 and GLOXD1 methylation in 29 normal controls and 160 paired HCC tissues and their adjacent nontumor tissues. We also calculated Kaplan-Meier survival curves to determine the applications of gene methylation in the prognosis of HCC.IRAK3 and GLOXD1 expression was partially restored in several HCC cell lines after treatment with 5-aza-2'-deoxycytidine (DNA methyltransferase inhibitor; 5DAC). A partial decrease in the methylated band was also observed in the HCC cell lines after 5DAC treatment. Using GLOXD1 as an example, we found a significant correlation between the data obtained from the methylation array and from pyrosequencing. <b>The methylation frequency of IRAK3 and GLOXD1 in HCC tissues was 46.9% and 63.8%, respectively.</b> <b>Methylation of IRAK3 was statistically associated with tumor stage.</b> <b>Moreover, HCC patients with IRAK3 methylation had a trend toward poor 3-year disease-free survival (P &lt; 0.05).IRAK3 and GLOXD1 were frequently methylated in HCC tissues compared to normal controls and nontumor tissues.</b> <b>IRAK3 methylation was associated with tumor stage and poor prognosis of patients.</b> <b>These data suggest that IRAK3 methylation is a novel prognostic marker in HCC.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25852213" target="_blank">
                    Microbiome Disturbances and Autism Spectrum Disorders.
                </a>
            </h2>
            <p>Autism spectrum disorders (ASD) are considered a heterogenous set of neurobehavioral diseases with the rates of diagnosis dramatically increasing in the past few decades. As genetics alone does not explain the underlying cause in many cases, attention has turned to other environmental factors as potential etiological agents. Gastrointestinal disorders are a common comorbidity in ASD patients. It was thus hypothesized that a gut-brain link may account for some autistic cases. With the characterization of the human microbiome, this concept has been expanded to include the microbiota-gut-brain axis. <b>There are mounting reports in animal models and human epidemiological studies linking disruptive alterations in the gut microbiota or dysbiosis and ASD symptomology.</b> <b>In this review, we will explore the current evidence that gut dysbiosis in animal models and ASD patients correlates with disease risk and severity.</b> <b>The studies to date have surveyed how gut microbiome changes may affect these neurobehavioral disorders.</b> However, we harbor other microbiomes reside in the body that might impact brain function. We will consider these other microbiomes in the oral cavity, vagina, and the most recently discovered one in the placenta. <b>Based on the premise that gut microbiota alterations may be causative agents in ASD, several therapeutic options have been tested, such as diet modulations, prebiotics, probiotics, synbiotics, postbiotics, antibiotics, fecal transplantation, and activated charcoal.</b> The potential benefits of these therapies will be considered. <b>Lastly, the possible mechanisms by which changes in the gut bacterial communities may result in ASD and related neurobehavioral disorders will be examined.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25852155" target="_blank">
                    MicroRNAs and DNA methylation as epigenetic regulators of mitosis, meiosis and spermiogenesis.
                </a>
            </h2>
            <p>Spermatogenesis is composed of three distinctive phases, which include self-renewal of spermatogonia via mitosis, spermatocytes undergo meiosis I/II, and post-meiotic development of haploid spermatids via spermiogenesis. Spermatogenesis also involves condensation of chromatin in the spermatid head before transformation of spermatids to spermatozoa. <b>Epigenetic regulation refers to changes of heritably cellular and physiological traits not caused by changes in the DNA sequences of the chromatin such as mutations.</b> <b>Major advances have been made in the epigenetic regulation of spermatogenesis.</b> <b>In this review, we address the roles and mechanisms of epigenetic regulators, with a focus on the role of microRNAs and DNA methylation during mitosis, meiosis and spermiogenesis.</b> <b>We also highlight issues deserve attentions for further investigation on the epigenetic regulation of spermatogenesis.</b> <b>More importantly, a thorough understanding of the epigenetic regulation in spermatogenesis will provide insightful information into the etiology of some unexplained infertility, offering new approaches for the treatment of male infertility.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25851713" target="_blank">
                    Defective DNA Mismatch Repair Influences Expression of Endometrial Carcinoma Biomarkers.
                </a>
            </h2>
            <p>Endometrial endometrioid carcinomas are related to estrogen excess and express estrogen and progesterone receptors. However, hormone receptor expression can be variable from tumor to tumor, and this variability is not always explained by differences in tumor grade. Variable expression of other biomarkers that may be used in the diagnostic work-up of endometrial cancer has also been noted. We hypothesized that mismatch repair (MMR) defects may contribute to this variability. A total of 411 unselected endometrial carcinomas were evaluated for immunohistochemical expression of DNA MMR proteins and MLH1 methylation. <b>Loss of immunohistochemical expression of MLH1, MSH2, MSH6, or PMS2 was defined as MMR deficient; positive expression was defined as MMR intact.</b> <b>A case-control cohort of 80 Grade 2 endometrioid carcinomas was selected from this set (40 MMR deficient, 40 MMR intact).</b> Cases were matched for histotype, grade, and age. Estrogen receptor, progesterone receptor, CK7, CK20, and Pax-8 immunohistochemistry was evaluated. <b>The median percentage of CK7 tumor cells was significantly lower in the MMR deficient group compared with the MMR intact group.</b> <b>The mean percentage of tumor cells exhibiting estrogen receptor expression was similar in both the MMR-deficient and MMR intact groups.</b> However, there was greater variability in the MMR-deficient group. <b>Our study shows that MMR defects influence the expression of clinically important biomarkers for endometrioid-type endometrial carcinoma as decreased cytokeratin 7 expression is more commonly associated with MMR deficiency.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25850930" target="_blank">
                    Distinct methylation patterns in genes that affect mitochondrial function are associated with kidney disease in blood-derived DNA from individuals with Type 1 diabetes.
                </a>
            </h2>
            <p>Epigenetic modifications, such as DNA methylation, can influence the risk of developing kidney disease. <b>We studied methylation profiles in genes related to mitochondrial function to assess whether differences in these epigenetic features were associated with diabetic kidney disease in people with Type&nbsp;1 diabetes.A case-control association study was undertaken (n&nbsp;=&nbsp;196 individuals with diabetic kidney disease vs. n&nbsp;=&nbsp;246 individuals without renal disease).</b> Participants were White and diagnosed with Type&nbsp;1 diabetes before 31&nbsp;years of age. Genes that encode mitochondrial proteins (n&nbsp;=&nbsp;780) were downloaded from mitoproteome. org. DNA methylation profiles from blood-derived DNA were generated using the Illumina Infinium HumanMethylation450 (262 samples) and Illumina Infinium HumanMethylation27 (192 samples) arrays. <b>Beta values (β) were calculated and quality control was conducted, including evaluating blind duplicate DNA samples.Fifty-four Cytosine-phosphate-Guanine probes across 51 unique genes were significantly associated (P&nbsp;≤&nbsp;10(-8) ) with diabetic kidney disease across both the 450K and the 27K methylation arrays.</b> <b>A subanalysis, employing the 450K array, identified 755 Cytosine-phosphate-Guanine probes in 374 genes that were significantly associated (P&nbsp;≤&nbsp;10(-8) ) with end-stage renal disease.</b> <b>Forty-six of the top-ranked variants for diabetic kidney disease were also identified as being differentially methylated in individuals with end-stage renal disease.</b> The largest change in methylation (Δβ&nbsp;=&nbsp;0.2) was observed for cg03169527 in the TAMM41 gene, chromosome 3p25.2. <b>Three genes, PMPCB, TSFM and AUH, were observed with differential methylation at multiple Cytosine-phosphate-Guanine sites each (P&nbsp;&lt;&nbsp;10(-12) ).Differential methylation in genes that influence mitochondrial function are associated with kidney disease in individuals with Type&nbsp;1 diabetes.</b> This article is protected by copyright. All rights reserved.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25850544" target="_blank">
                    Arsenic-Induced Antioxidant Depletion, Oxidative DNA Breakage, and Tissue Damages are Prevented by the Combined Action of Folate and Vitamin B12.
                </a>
            </h2>
            <p><b>Arsenic is a grade I human carcinogen.</b> It acts by disrupting one-carbon (1C) metabolism and cellular methyl (-CH3) pool. <b>The -CH3 group helps in arsenic disposition and detoxification of the biological systems.</b> Vitamin B12 and folate, the key promoters of 1C metabolism were tested recently (daily 0.07 and 4.0&nbsp;μg, respectively/100&nbsp;g b.w. of rat for 28&nbsp;days) to evaluate their combined efficacy in the protection from mutagenic DNA-breakage and tissue damages. <b>The selected tissues like intestine (first-pass site), liver (major xenobiotic metabolizer) and lung (major arsenic accumulator) were collected from arsenic-ingested (0.6&nbsp;ppm/same schedule) female rats.</b> The hemo-toxicity and liver and kidney functions were monitored. Our earlier studies on arsenic-exposed humans can correlate carcinogenesis with DNA damage. Here, we demonstrate that the supplementation of physiological/therapeutic dose of vitamin B12 and folate protected the rodents significantly from arsenic-induced DNA damage (DNA fragmentation and comet assay) and hepatic and renal tissue degeneration (histo-architecture, HE staining). The level of arsenic-induced free-radical products (TBARS and conjugated diene) was significantly declined by the restored actions of several antioxidants viz. urate, thiol, catalase, xanthine oxidase, lactoperoxidase, and superoxide dismutase in the tissues of vitamin-supplemented group. The alkaline phosphatase, transaminases, urea and creatinine (hepatic and kidney toxicity marker), and lactate dehydrogenase (tissue degeneration marker) were significantly impaired in the arsenic-fed group. But a significant protection was evident in the vitamin-supplemented group. In conclusion, the combined action of folate and B12 results in the restitution in the 1C metabolic pathway and cellular methyl pool. <b>The cumulative outcome from the enhanced arsenic methylation and antioxidative capacity was protective against arsenic induced mutagenic DNA breakages and tissue damages.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25849984" target="_blank">
                    Maternal depression is associated with DNA methylation changes in cord blood T lymphocytes and adult hippocampi.
                </a>
            </h2>
            <p>Depression affects 10-15% of pregnant women and has been associated with preterm delivery and later developmental, behavioural and learning disabilities. We tested the hypothesis that maternal depression is associated with DNA methylation alterations in maternal T lymphocytes, neonatal cord blood T lymphocytes and adult offspring hippocampi. Genome-wide DNA methylation of CD3+ T lymphocytes isolated from 38 antepartum maternal and 44 neonatal cord blood samples were analyzed using Illumina Methylation 450 K microarrays. Previously obtained methylation data sets using methylated DNA immunoprecipitation and array-hybridization of 62 postmortem hippocampal samples of adult males were re-analyzed to test associations with history of maternal depression. <b>We found 145 (false discovery rate (FDR) q&lt;0.05) and 2520 (FDR q&lt;0.1) differentially methylated CG-sites in cord blood T lymphocytes of neonates from the maternal depression group as compared with the control group.</b> <b>However, no significant DNA methylation differences were detected in the antepartum maternal T lymphocytes of our preliminary data set.</b> <b>We also detected 294 differentially methylated probes (FDR q&lt;0.1) in hippocampal samples associated with history of maternal depression.</b> We observed a significant overlap (P=0.002) of 33 genes with changes in DNA methylation in T lymphocytes of neonates and brains of adult offspring. Many of these genes are involved in immune system functions. <b>Our results show that DNA methylation changes in offspring associated with maternal depression are detectable at birth in the immune system and persist to adulthood in the brain.</b> <b>This is consistent with the hypothesis that system-wide epigenetic changes are involved in life-long responses to maternal depression in the offspring.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25849957" target="_blank">
                    Second German-Catalan workshop on epigenetics &amp; cancer.
                </a>
            </h2>
            <p>The Second German-Catalan Workshop on Epigenetics and Cancer was held in Barcelona on November 19-21, 2014. The workshop brought together, for the second time, scientists from 2 German and 2 Catalan research institutions: the DKFZ, from Heidelberg, the CRCME, from Freiburg, and the IMPPC and PEBC/IDIBELL, both from Barcelona. The German-Catalan Workshops are intended to establish the framework for building a Research School to foster collaborations between researchers from the different institutions. Exchange programs for graduate students are among the activities of the future School. <b>The topics presented and discussed in 33 talks were diverse and included work on DNA methylation, histone modifications, chromatin biology, characterization of imprinted regions in human tissues, non-coding RNAs, and epigenetic drug discovery.</b> <b>Among novel developments from the previous Workshop are the report of the epigenetics angle of the Warburg effect and the long-range trans-acting interaction of DNA methylation and of nucleosome remodeling.</b> <b>A shift in the view on DNA methylation became apparent by the realization of the intertwined interplay between hyper- and hypo-methylation in differentiation and cancer.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25849771" target="_blank">
                    DNA Methylation and Transcriptomic Changes in Response to Different Lights and Stresses in 7B-1 Male-Sterile Tomato.
                </a>
            </h2>
            <p>We reported earlier that 7B-1 mutant in tomato (Solanum lycopersicum L., cv. Rutgers), an ABA overproducer, is defective in blue light (B) signaling leading to B-specific resistance to abiotic and biotic stresses. Using a methylation-sensitive amplified polymorphism (MSAP) assay, a number of genes were identified, which were differentially methylated between 7B-1 and its wild type (WT) seedlings in white (W), blue (B), red (R) lights and dark (D) or in response to exogenous ABA and mannitol-induced stresses. The genomic methylation level was almost similar in different lights between 7B-1 and WT seedlings, while significant differences were observed in response to stresses in D, but not B. Using a cDNA-AFLP assay, several transcripts were identified, which were differentially regulated between 7B-1 and WT by B or D or in response to stresses. <b>Blue light receptors cryptochrome 1 and 2 (CRY1 and CRY2) and phototropin 1 and 2 (PHOT1 and PHOT2) were not affected by the 7B-1 mutation at the transcriptional level, instead the mutation had likely affected downstream components of the light signaling pathway.</b> 5-azacytidine (5-azaC) induced DNA hypomethylation, inhibited stem elongation and differentially regulated the expression of a number of genes in 7B-1. In addition, it was shown that mir167 and mir390 were tightly linked to auxin signaling pathway in 5-azaC-treated 7B-1 seedlings via the regulation of auxin-response factor (ARF) transcripts. Our data showed that DNA methylation remodeling is an active epigenetic response to different lights and stresses in 7B-1 and WT, and highlighted the differences in epigenetic and transcriptional regulation of light and stress responses between 7B-1 and WT. Furthermore, it shed lights on the crosstalk between DNA hypomethylation and miRNA regulation of ARFs expression. This information could also be used as a benchmark for future studies of male-sterility in other crops.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25849659" target="_blank">
                    Chromatin Function Modifying Elements in an Industrial Antibody Production Platform - Comparison of UCOE, MAR, STAR and cHS4 Elements.
                </a>
            </h2>
            <p>The isolation of stably transfected cell lines suitable for the manufacture of biotherapeutic protein products can be an arduous process relying on the identification of a high expressing clone; this frequently involves transgene amplification and maintenance of the clones' expression over at least 60 generations. <b>Maintenance of expression, or cell line stability, is highly dependent upon the nature of the genomic environment at the site of transgene integration, where epigenetic mechanisms lead to variable expression and silencing in the vast majority of cases.</b> <b>We have assessed four chromatin function modifying elements (A2UCOE, MAR X_S29, STAR40 and cHS4) for their ability to negate chromatin insertion site position effects and their ability to express and maintain monoclonal antibody expression.</b> <b>Each element was analysed by insertion into different positions within a vector, either flanking or between heavy chain (HC) and light chain (LC) antibody expression cassettes.</b> Our results clearly show that the A2UCOE is the most beneficial element in this system, with stable cell pools and clones increasing antibody yields 6.5-fold and 6.75-fold respectively. <b>Stability analysis demonstrated that the reduction in antibody expression, seen with cells transfected with the control vector over 120 generations, was mitigated in the clones containing A2UCOE-augmented transgenes.</b> <b>Analysis also showed that the A2UCOE reduced the amount of transgene promoter DNA methylation, which contributed to the maintenance of starting levels of expression.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25849377" target="_blank">
                    Suppressor of Cytokine Signaling (SOCS) Genes Are Silenced by DNA Hypermethylation and Histone Deacetylation and Regulate Response to Radiotherapy in Cervical Cancer Cells.
                </a>
            </h2>
            <p>Suppressor of cytokine signaling (SOCS) family is an important negative regulator of cytokine signaling and deregulation of SOCS has been involved in many types of cancer. <b>All cervical cancer cell lines tested showed lower expression of SOCS1, SOCS3, and SOCS5 than normal tissue or cell lines.</b> The immunohistochemistry result for SOCS proteins in human cervical tissue also confirmed that normal tissue expressed higher level of SOCS proteins than neighboring tumor. Similar to the regulation of SOCS in other types of cancer, DNA methylation contributed to SOCS1 downregulation in CaSki, ME-180, and HeLa cells. <b>However, the expression of SOCS3 or SOCS5 was not recovered by the inhibition of DNA methylation.</b> <b>Histone deacetylation may be another regulatory mechanism involved in SOCS1 and SOCS3 expression, however, SOCS5 expression was neither affected by DNA methylation nor histone deacetylation.</b> <b>Ectopic expression of SOCS1 or SOCS3 conferred radioresistance to HeLa cells, which implied SOCS signaling regulates the response to radiation in cervical cancer.</b> <b>In this study, we have shown that SOCS expression repressed by, in part, epigenetically and altered SOCS1 and SOCS3 expression could contribute to the radiosensitive phenotype in cervical cancer.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25849103" target="_blank">
                    Parallel action of AtDRB2 and RdDM in the control of transposable element expression.
                </a>
            </h2>
            <p>In plants and animals, a large number of double-stranded RNA binding proteins (DRBs) have been shown to act as non-catalytic cofactors of DICERs and to participate in the biogenesis of small RNAs involved in RNA silencing. <b>We have previously shown that the loss of Arabidopsis thaliana's DRB2 protein results in a significant increase in the population of RNA polymerase IV (p4) dependent siRNAs, which are involved in the RNA-directed DNA methylation (RdDM) process.Surprisingly, despite this observation, we show in this work that DRB2 is part of a high molecular weight complex that does not involve RdDM actors but several chromatin regulator proteins, such as MSI4, PRMT4B and HDA19.</b> <b>We show that DRB2 can bind transposable element (TE) transcripts in vivo but that drb2 mutants do not have a significant variation in TE DNA methylation.We propose that DRB2 is part of a repressive epigenetic regulator complex involved in a negative feedback loop, adjusting epigenetic state to transcription level at TE loci, in parallel of the RdDM pathway.</b> <b>Loss of DRB2 would mainly result in an increased production of TE transcripts, readily converted in p4-siRNAs by the RdDM machinery.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25849095" target="_blank">
                    Interneuron epigenomes during the critical period of cortical plasticity: Implications for schizophrenia.
                </a>
            </h2>
            <p><b>Schizophrenia, a major psychiatric disorder defined by delusions and hallucinations, among other symptoms, often with onset in early adulthood, is potentially associated with molecular and cellular alterations in parvalbumin-expressing fast spiking interneurons and other constituents of the cortical inhibitory GABAergic circuitry.</b> The underlying mechanisms, including the role of disease-associated risk factors operating in adolescence such as drug abuse and social stressors, remain incompletely understood. Here, we summarize emerging findings from animal models, highlighting the ability of parvalbuminergic interneurons (PVI) to induce, during the juvenile period, long-term plastic changes in prefrontal and visual cortex, thereby altering perception, cognition and behavior in the adult. <b>Of note, molecular alterations in PVI from subjects with schizophrenia, including downregulated expression of a subset of GABAergic genes, have also been found in juvenile stress models of the disorder.</b> <b>Some of the transcriptional alterations observed in schizophrenia postmortem brain could be linked to changes in the epigenetic architecture of GABAergic gene promoters, including dysregulated DNA methylation, histone modification patterns and disruption of promoter-enhancer interactions at site of chromosomal loop formations.</b> <b>Therefore, we predict that, in the not-to-distant future, PVI- and other cell-type specific epigenomic mappings in the animal model and human brain will provide novel insights into the pathophysiology of schizophrenia and related psychotic diseases, including the role of cortical GABAergic circuitry in shaping long-term plasticity and cognitive function of the cerebral cortex.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25848754" target="_blank">
                    FBXL10 protects Polycomb-bound genes from hypermethylation.
                </a>
            </h2>
            <p><b>Nearly all CpG-dense promoters are occupied by the multidomain chromosomal protein FBXL10.</b> <b>We show here that complete inactivation of the Fbxl10 gene leads to dense de novo methylation only of promoters that are co-occupied by both FBXL10 and Polycomb repressive complexes; this methylation results in pervasive defects in embryonic development and the death of homozygous Fbxl10-mutant embryos at midgestation.</b> Deletion of key components of Polycomb repressive complexes 1 and 2 did not lead to ectopic genomic methylation. <b>These results indicate that FBXL10 protects Polycomb-occupied promoters against ectopic de novo methylation.</b> <b>To our knowledge, FBXL10 is the first reported factor whose loss leads to a gain in genomic DNA methylation.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25848745" target="_blank">
                    In vivo targeting of de novo DNA methylation by histone modifications in yeast and mouse.
                </a>
            </h2>
            <p>Methylation of cytosines (5(me)C) is a widespread heritable DNA modification. During mammalian development, two global demethylation events are followed by waves of de novo DNA methylation. In vivo mechanisms of DNA methylation establishment are largely uncharacterized. <b>Here we use Saccharomyces cerevisiae as a system lacking DNA methylation to define the chromatin features influencing the activity of the murine DNMT3B.</b> <b>Our data demonstrate that DNMT3B and H3K4 methylation are mutually exclusive and that DNMT3B is co-localized with H3K36 methylated regions.</b> <b>In support of this observation, DNA methylation analysis in yeast strains without Set1 and Set2 show an increase of relative 5(me)C levels at the TSS and a decrease in the gene-body, respectively.</b> <b>We extend our observation to the murine male germline, where H3K4me3 is strongly anti-correlated while H3K36me3 correlates with accelerated DNA methylation.</b> <b>These results show the importance of H3K36 methylation for gene-body DNA methylation in vivo.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25848000" target="_blank">
                    Maternal and zygotic Zfp57 modulate NOTCH signaling in cardiac development.
                </a>
            </h2>
            <p>Zfp57 is a maternal-zygotic effect gene that maintains genomic imprinting. Here we report that Zfp57 mutants exhibited a variety of cardiac defects including atrial septal defect (ASD), ventricular septal defect (VSD), thin myocardium, and reduced trabeculation. Zfp57 maternal-zygotic mutant embryos displayed more severe phenotypes with higher penetrance than the zygotic ones. Cardiac progenitor cells exhibited proliferation and differentiation defects in Zfp57 mutants. <b>ZFP57 is a master regulator of genomic imprinting, so the DNA methylation imprint was lost in embryonic heart without ZFP57.</b> <b>Interestingly, the presence of imprinted DLK1, a target of ZFP57, correlated with NOTCH1 activation in cardiac cells.</b> <b>These results suggest that ZFP57 may modulate NOTCH signaling during cardiac development.</b> <b>Indeed, loss of ZFP57 caused loss of NOTCH1 activation in embryonic heart with more severe loss observed in the maternal-zygotic mutant.</b> <b>Maternal and zygotic functions of Zfp57 appear to play redundant roles in NOTCH1 activation and cardiomyocyte differentiation.</b> This serves as an example of a maternal effect that can influence mammalian organ development. It also links genomic imprinting to NOTCH signaling and particular developmental functions.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25847947" target="_blank">
                    DUSP4 deficiency caused by promoter hypermethylation drives JNK signaling and tumor cell survival in diffuse large B cell lymphoma.
                </a>
            </h2>
            <p>The epigenetic dysregulation of tumor suppressor genes is an important driver of human carcinogenesis. We have combined genome-wide DNA methylation analyses and gene expression profiling after pharmacological DNA demethylation with functional screening to identify novel tumor suppressors in diffuse large B cell lymphoma (DLBCL). We find that a CpG island in the promoter of the dual-specificity phosphatase DUSP4 is aberrantly methylated in nodal and extranodal DLBCL, irrespective of ABC or GCB subtype, resulting in loss of DUSP4 expression in 75% of &gt;200 examined cases. <b>The DUSP4 genomic locus is further deleted in up to 13% of aggressive B cell lymphomas, and the lack of DUSP4 is a negative prognostic factor in three independent cohorts of DLBCL patients.</b> <b>Ectopic expression of wild-type DUSP4, but not of a phosphatase-deficient mutant, dephosphorylates c-JUN N-terminal kinase (JNK) and induces apoptosis in DLBCL cells.</b> <b>Pharmacological or dominant-negative JNK inhibition restricts DLBCL survival in vitro and in vivo and synergizes strongly with the Bruton's tyrosine kinase inhibitor ibrutinib.</b> <b>Our results indicate that DLBCL cells depend on JNK signaling for survival.</b> <b>This finding provides a mechanistic basis for the clinical development of JNK inhibitors in DLBCL, ideally in synthetic lethal combinations with inhibitors of chronic active B cell receptor signaling.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25847519" target="_blank">
                    Epigenetics of the failing heart.
                </a>
            </h2>
            <p>With the impressive advancement in high-throughput 'omics' technologies over the past two decades, epigenetic mechanisms have emerged as the regulatory interface between the genome and environmental factors. These mechanisms include DNA methylation, histone modifications, ATP-dependent chromatin remodeling and RNA-based mechanisms. Their highly interdependent and coordinated action modulates the chromatin structure controlling access of the transcription machinery and thereby regulating expression of target genes. Given the rather limited proliferative capability of human cardiomyocytes, epigenetic regulation appears to play a particularly important role in the myocardium. <b>The highly dynamic nature of the epigenome allows the heart to adapt to environmental challenges and to respond quickly and properly to cardiac stress.</b> <b>It is now becoming evident that histone-modifying and chromatin-remodeling enzymes as well as numerous non-coding RNAs play critical roles in cardiac development and function, while their dysregulation contributes to the onset and development of pathological cardiac remodeling culminating in HF.</b> This review focuses on up-to-date knowledge about the epigenetic mechanisms and highlights their emerging role in the healthy and failing heart. <b>Uncovering the determinants of epigenetic regulation holds great promise to accelerate the development of successful new diagnostic and therapeutic strategies in human cardiac disease.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25847269" target="_blank">
                    Expression and methylation of DNA repair genes in lens epithelium cells of age-related cataract.
                </a>
            </h2>
            <p>The development of age-related cataract (ARC) is associated with DNA damage of the lens epithelial cells (LECs). This study aimed to investigate the expression level of DNA repair genes in LECs of ARC and examine whether any altered expression observed could result from DNA methylation of the promoter region of the genes. <b>The expression levels of DNA repair genes were evaluated by microarray analysis.</b> The results were further confirmed by qRT-PCR. <b>DNA methylation of genes with altered expression was determined by bisulfite-specific (BSP) PCR.</b> <b>The mRNA levels of 10 DNA repair genes were decreased and the level of 1 DNA repair gene was increased in LECs of ARC patients compared with controls.</b> The promoter region of the MGMT gene was hypermethylated in ARC tissue compared to controls. <b>The data provide evidence that altered expression of DNA repair genes is associated with pathogenesis of ARC.</b> <b>DNA methylation of MGMT may regulate the expression of the gene and be involved in the development of ARC.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25846879" target="_blank">
                    The effect of high glucose levels on the hypermethylation of protein phosphatase 1 regulatory subunit 3C (PPP1R3C) gene in colorectal cancer.
                </a>
            </h2>
            <p>DNA methylation is an epigenetic event that occurs frequently in colorectal cancer (CRC). Increased glucose level is a strong risk factor for CRC. Protein phosphatase 1 regulatory subunit 3C (PPP1R3C) modulates glycogen metabolism, particularly glycogen synthesis. The aim of this study was to investigate the effect of high glucose levels on DNA methylation of PPP1R3C in CRC. PPP1R3C was significantly hypermethylated in CRC tissues (76/105, 72.38%, P &lt;0.05) and colon cancer cell lines (P &lt; 0.05). CRC tissues obtained from patients with high glucose levels showed that the methylation of PPP1R3C was lower than in patients who had normal levels of glucose. <b>When DLD-1 cells were cultured under conditions of high glucose, the methylation of PPP1R3C was repressed.</b> <b>The expression of PPP1R3C was inversely related to methylation status.</b> <b>In addition, a promoter luciferase assay showed that the transcriptional activity of PPP1R3C was increased in high glucose culture conditions.</b> <b>The number of cells decreased when PPP1R3C was silenced in DLD-1 cells.</b> <b>These results suggest that PPP1R3C, a novel hypermethylated gene in CRC, may play a critical role in cancer cell growth in association with glucose levels.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25846368" target="_blank">
                    Elevated tissue Cr levels, increased plasma oxidative markers, and global hypomethylation of blood DNA in male Sprague-Dawley rats exposed to potassium dichromate in drinking water.
                </a>
            </h2>
            <p>Hexavalent chromium [Cr (VI)] is prevalent in ground water in some areas, but evidence on the toxic effects of Cr (VI) via ingestion through drinking water remains insufficient. The aims of our study were to investigate the toxic effects of Cr (VI) through oral water ingestion on oxidative stress and DNA methylation. Thirty-two Sprague-Dawley rats were randomly divided into four groups, and exposed to porassium dichromate (K2 Cr2 O7 ; 0, 30, 100, and 300 mg/L) in drinking water for 4 weeks. Mean body weight gain, mean water consumption, clinical chemistry determinations, and oxidative stress levels in plasma were measured. <b>Global DNA methylation changes and DNA methylation status at the promoter of p16 gene were also detected.</b> After 4 weeks, mild anemic effects and increased plasma malondialdehyde (MDA) levels occurred in rats exposed to 100 mg/L or 300 mg/L of Cr (VI). Plasma glutathione peroxidase (GSH-Px) activity decreased in all exposed groups. <b>Global DNA methylation levels were reduced in 100 mg/L and 300 mg/L exposure groups.</b> <b>However, DNA methylation status at the promoter of P16 gene remained unchanged in all K2 Cr2 O7- treated groups.</b> <b>The correlation analysis indicated that increased MDA levels were closely correlated to global DNA hypomethylation.</b> <b>Our results indicated that oral ingestion of Cr (VI) through drinking water caused not only oxidative stress in plasma, but also global DNA hypomethylation in blood cells from male rats, and a good correlation was found between increased MDA levels and reduced global DNA methylation.</b> © 2015 Wiley Periodicals, Inc. Environ Toxicol, 2015.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25845600" target="_blank">
                    Structures of Escherichia coli DNA adenine methyltransferase (Dam) in complex with a non-GATC sequence: potential implications for methylation-independent transcriptional repression.
                </a>
            </h2>
            <p><b>DNA adenine methyltransferase (Dam) is widespread and conserved among the γ-proteobacteria.</b> Methylation of the Ade in GATC sequences regulates diverse bacterial cell functions, including gene expression, mismatch repair and chromosome replication. <b>Dam also controls virulence in many pathogenic Gram-negative bacteria.</b> <b>An unexplained and perplexing observation about Escherichia coli Dam (EcoDam) is that there is no obvious relationship between the genes that are transcriptionally responsive to Dam and the promoter-proximal presence of GATC sequences.</b> Here, we demonstrate that EcoDam interacts with a 5-base pair non-cognate sequence distinct from GATC. The crystal structure of a non-cognate complex allowed us to identify a DNA binding element, GTYTA/TARAC (where Y = C/T and R = A/G). This element immediately flanks GATC sites in some Dam-regulated promoters, including the Pap operon which specifies pyelonephritis-associated pili. <b>In addition, Dam interacts with near-cognate GATC sequences (i.e.</b> 3/4-site ATC and GAT). <b>Taken together, these results imply that Dam, in addition to being responsible for GATC methylation, could also function as a methylation-independent transcriptional repressor.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25845594" target="_blank">
                    Specificity of the ModA11, ModA12 and ModD1 epigenetic regulator N6-adenine DNA methyltransferases of Neisseria meningitidis.
                </a>
            </h2>
            <p>Phase variation (random ON/OFF switching) of gene expression is a common feature of host-adapted pathogenic bacteria. Phase variably expressed N(6)-adenine DNA methyltransferases (Mod) alter global methylation patterns resulting in changes in gene expression. These systems constitute phase variable regulons called phasevarions. Neisseria meningitidis phasevarions regulate genes including virulence factors and vaccine candidates, and alter phenotypes including antibiotic resistance. The target site recognized by these Type III N(6)-adenine DNA methyltransferases is not known. Single molecule, real-time (SMRT) methylome analysis was used to identify the recognition site for three key N. meningitidis methyltransferases: ModA11 (exemplified by M.NmeMC58I) (5'-CGY M6A: G-3'), ModA12 (exemplified by M.Nme77I, M.Nme18I and M.Nme579II) (5'-AC M6A: CC-3') and ModD1 (exemplified by M.Nme579I) (5'-CC M6A: GC-3'). Restriction inhibition assays and mutagenesis confirmed the SMRT methylome analysis. The ModA11 site is complex and atypical and is dependent on the type of pyrimidine at the central position, in combination with the bases flanking the core recognition sequence 5'-CGY M6A: G-3'. The observed efficiency of methylation in the modA11 strain (MC58) genome ranged from 4.6% at 5'-GCGC M6A: GG-3' sites, to 100% at 5'-ACGT M6A: GG-3' sites. Analysis of the distribution of modified sites in the respective genomes shows many cases of association with intergenic regions of genes with altered expression due to phasevarion switching.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25845328" target="_blank">
                    Site-specific immunochemical methylation assessment from genome DNA utilizing a conformational difference between looped-out target and stacked-in nontarget methylcytosines.
                </a>
            </h2>
            <p><b>We report the sequence-selective immunochemical discrimination of methylcytosine from genomic DNA that we achieved by utilizing selective antibody binding to a looped-out methylcytosine in a bulge region and without using bisulfite treatment, a methylation-sensitive restriction enzyme, or PCR.</b> First, we investigated the affinity of an anti-methylcytosine antibody for methylcytosine in full match, bulge, mismatch and abasic sites using surface plasmon resonance measurement and a microtiter plate to explore the differences as regards antibody binding to the target methylcytosine. <b>The highest affinity, which was comparable to that in a single strand of DNA, was observed in the bulge region.</b> In particular, no affinity was observed in a full match site. This is because there is no interaction such as hydrogen bond or π-π stacking for the bulged methylcytosine, thus enabling only the target in the bulge to be looped out. <b>Methylated and unmethylated genomic DNA were blended to form a model DNA with which to assess the methylation ratio at a specific site.</b> <b>Fragmented DNA was hybridized with a biotinylated probe DNA, which has a sequence capable of forming a single base bulge at the target.</b> The probe design is simple because it consists solely of the elimination of guanine paired with the target cytosine from a full match sequence. As a result, we successfully obtained a linear relationship (r(2)=0.9962) between the immunoassay signal and the methylation ratio of a specific site within 4h.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25845242" target="_blank">
                    [Spatial organization of house-keeping genes in interphase nuclei].
                </a>
            </h2>
            <p>Spatial organization of the eukaryotic genome is tightly connected to its functioning. In particular, the interaction of gene promoters with remote enhancer elements in active chromatin hubs, as well as the recruitment of genes to the common transcription factories plays an important role in regulation of gene transcription. <b>Most of works related to the analysis of spatial interaction of genome regulatory elements relies on models of tissue-specific genes.</b> <b>Meanwhile, it remains unclear to which extent the spatial organization of chromosomes is guided by house-keeping genes that are transcribed in most of cell types and outnumber the transcribed tissue-specific genes.</b> <b>To address this question, we used the 4C technique to characterize genome-wide the spatial contacts of the chicken house-keeping genes CARHSP1 and TRAP1 situated on chromosome 14.</b> <b>The promoters of these genes had an increased frequency of interaction with chromosome regions enriched in CpG islands and binding motifs for the ubiquitous transcription factor Sp1, both of which mark promoters of house-keeping genes, and overall with transcriptionally active regions.</b> By contrast, the analysis of interaction of a gene poor region of chromosome 14 revealed no such preferences. The evidence for the interaction of house-keeping gene promoters were also obtained in independent cytological experiments aimed at visualization of non-methylated CpG islands in individual nuclei of human cells, which showed clustering of CpG islands in the nuclear space. <b>Altogether, the results of our work suggest that the interaction of house-keeping genes constitutes an important factor that determines the spatial organization of interphase chromosomes.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25844678" target="_blank">
                    Intratubular germ cell neoplasia of the testis: a brief review.
                </a>
            </h2>
            <p>Germ cell tumors of the testis may be divided into 3 broad categories according to age at presentation. The tumors in the pediatric age group include teratoma and yolk sac tumor. These tumors are generally not associated with convincing intratubular neoplasia. The second group consists of tumors presenting in third and fourth decade of life and include seminoma, embryonal carcinoma, yolk sac tumor, choriocarcinoma, and teratoma as well as mixed germ cell tumors. The precursor cell for these tumors is an abnormal gonocyte that fails to differentiate completely into spermatogonia. <b>These abnormal cells stay dormant in the gonad during intrauterine life as well as infancy and childhood, but undergo proliferation during puberty and can be identified as intratubular germ cell neoplasia unclassified (IGCNU).</b> These tumor cells continue to manifest protein expression pattern that resembles primitive germ cells (PLAP, c-KIT, OCT3/4). <b>After a variable interval following puberty, IGCNU cells may acquire ability to penetrate the seminiferous tubules and present as an overt germ cell tumor.</b> Acquisition of isochrome 12 and other genetic abnormalities are usually associated with this transition. <b>The level of DNA methylation generally determines the phenotype of the germ cell tumor.</b> <b>The third type of germ cell tumors is spermatocytic seminoma, which is a rare tumor encountered later in life usually in fifth and sixth decade.</b> <b>The cell of origin of this tumor is probably postpubertal mature spermatogonia which acquire abnormal proliferative capability probably due to gain of chromosome 9 resulting in activation and amplification of genes such as DMRT1.</b> The tumor cells manifest many of the proteins normally expressed by mature sperms such as VASA, SSX2, and occasionally OCT2. Although spermatocytic seminoma may also have an intratubular growth phase, it completely lacks features of IGCNU.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25844668" target="_blank">
                    A canonical correlation analysis-based dynamic bayesian network prior to infer gene regulatory networks from multiple types of biological data.
                </a>
            </h2>
            <p>One of the challenging and important computational problems in systems biology is to infer gene regulatory networks (GRNs) of biological systems. Several methods that exploit gene expression data have been developed to tackle this problem. In this study, we propose the use of copy number and DNA methylation data to infer GRNs. <b>We developed an algorithm that scores regulatory interactions between genes based on canonical correlation analysis.</b> In this algorithm, copy number or DNA methylation variables are treated as potential regulator variables, and expression variables are treated as potential target variables. <b>We first validated that the canonical correlation analysis method is able to infer true interactions in high accuracy.</b> We showed that the use of DNA methylation or copy number datasets leads to improved inference over steady-state expression. <b>Our results also showed that epigenetic and structural information could be used to infer directionality of regulatory interactions.</b> Additional improvements in GRN inference can be gleaned from incorporating the result in an informative prior in a dynamic Bayesian algorithm. This is the first study that incorporates copy number and DNA methylation into an informative prior in dynamic Bayesian framework. <b>By closely examining top-scoring interactions with different sources of epigenetic or structural information, we also identified potential novel regulatory interactions.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25843718" target="_blank">
                    Differentiation of human parthenogenetic pluripotent stem cells reveals multiple tissue- and isoform-specific imprinted transcripts.
                </a>
            </h2>
            <p>Parental imprinting results in monoallelic parent-of-origin-dependent gene expression. <b>However, many imprinted genes identified by differential methylation do not exhibit complete monoallelic expression.</b> <b>Previous studies demonstrated complex tissue-dependent expression patterns for some imprinted genes.</b> Still, the complete magnitude of this phenomenon remains largely unknown. <b>By differentiating human parthenogenetic induced pluripotent stem cells into different cell types and combining DNA methylation with a 5' RNA sequencing methodology, we were able to identify tissue- and isoform-dependent imprinted genes in a genome-wide manner.</b> <b>We demonstrate that nearly half of all imprinted genes express both biallelic and monoallelic isoforms that are controlled by tissue-specific alternative promoters.</b> <b>This study provides a global analysis of tissue-specific imprinting in humans and suggests that alternative promoters are central in the regulation of imprinted genes.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25843235" target="_blank">
                    NRMT1 knockout mice exhibit phenotypes associated with impaired DNA repair and premature aging.
                </a>
            </h2>
            <p>Though defective genome maintenance and DNA repair have long been known to promote phenotypes of premature aging, the role protein methylation plays in these processes is only now emerging. <b>We have recently identified the first N-terminal methyltransferase, NRMT1, which regulates protein-DNA interactions and is necessary for both accurate mitotic division and nucleotide excision repair.</b> <b>To demonstrate if complete loss of NRMT1 subsequently resulted in developmental or aging phenotypes, we constructed the first NRMT1 knockout (Nrmt1(-/-)) mouse.</b> The majority of these mice die shortly after birth. However, the ones that survive, exhibit decreased body size, female-specific infertility, kyphosis, decreased mitochondrial function, and early-onset liver degeneration; phenotypes characteristic of other mouse models deficient in DNA repair. <b>The livers from Nrmt1(-/-) mice produce less reactive oxygen species (ROS) than wild type controls, and Nrmt1(-/-) mouse embryonic fibroblasts show a decreased capacity for handling oxidative damage.</b> <b>This indicates that decreased mitochondrial function may benefit Nrmt1(-/-) mice and protect them from excess internal ROS and subsequent DNA damage.</b> <b>These studies position the NRMT1 knockout mouse as a useful new system for studying the effects of genomic instability and defective DNA damage repair on organismal and tissue-specific aging.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25843056" target="_blank">
                    Selective inhibition of prostaglandin E2 receptors EP2 and EP4 modulates DNA methylation and histone modification machinery proteins in human endometriotic cells.
                </a>
            </h2>
            <p>Endometriosis is an inflammatory gynecological disease of reproductive-age women. The prevalence of endometriosis is 5-10% in reproductive-age women. Modern medical treatments are directed to inhibit the action of estrogen in endometriotic cells. However, hormonal therapies targeting estrogen can be prescribed only for a short time because of their undesirable side effects. <b>Recent studies from our laboratory, using endometriotic epithelial cell line 12Z and stromal cell line 22B derived from red lesion, discovered that selective inhibition of prostaglandin E2 (PGE2) receptors EP2 and EP4 inhibits adhesion, invasion, growth, and survival of 12Z and 22B cells by modulating integrins, MMPs and TIMPs, cell cycle, survival, and intrinsic apoptotic pathways, suggesting multiple epigenetic mechanisms.</b> <b>The novel findings of the present study indicate that selective pharmacological inhibition of EP2 and EP4: (i) decreases expression of DNMT3a, DNMT3b, H3K9me3, H3K27me3, SUV39H1, HP1a, H3K27, EZH2, JMJD2a, HDAC1, HDAC3, MeCP2, CoREST and Sin3A; (ii) increases expression of H3K4me3, H3H9ac, H3K27ac; and (iii) does not modulate the expression of DNMT1, hSET1, LSD1, MBD1, p300, HDAC2, and JMJD3 epigenetic machinery proteins in an epithelial and stromal cell specific manner.</b> In this study, we report for the first time that inhibition of PGE2-EP2/EP4 signaling modulates DNA methylation, H3 histone methylation and acetylation, and epigenetic memory machinery proteins in human endometriotic epithelial cells and stromal cells. Thus, targeting EP2 and EP4 receptors may emerge as long-term nonsteroidal therapy for treatment of active endometriotic lesions in women.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25843052" target="_blank">
                    DNA methylation analysis of SOCS1, SOCS3, and LINE-1 in microdissected gingival tissue.
                </a>
            </h2>
            <p><b>DNA methylation plays a critical role in the regulation of the transcription of the suppressors of cytokine signaling (SOCS) 1 and SOCS3, which are modulators in the inflammation.</b> <b>We hypothesized that the methylation status of SOCS1, SOCS3, and long interspersed nuclear element (LINE)-1 in gingival tissues previously inflamed would be similar to that found in gingival tissues without clinical inflammation in the period studied.Laser capture microdissection was performed to isolate epithelial and connective gingival tissues.</b> <b>The groups were comprised by ten patients without history of periodontitis and absence of clinical signs of inflammation in the gingiva during the study (healthy group) and ten patients with history of periodontitis, presenting inflammation in the gingival tissue at the first examination of the study (controlled chronic periodontitis group).</b> <b>The gingival biopsies from the controlled chronic periodontitis group were collected after controlling the inflammation.</b> <b>DNA methylation patterns were analyzed using methylation-specific high-resolution melting and combined bisulfite restriction analysis.DNA methylation levels for SOCS1 and SOCS3 did not differ between groups or tissues; likewise, no differences were observed in total LINE-1 methylation or at specific loci.At 3&nbsp;months following control of inflammation in gingival tissues, the methylation profile of SOCS1, SOCS3, and LINE-1 is similar between connective and epithelial tissues from patients that were previously affected or not by chronic inflammation.Clinical results of a successful treatment are observed after inflammation control and the molecular findings illustrate local and general methylation patterns in recovering tissues toward health conditions and might help to understand events that are occurring in oral cells.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25842863" target="_blank">
                    [Developmental expression of arabidopsis methyltransferase genes MET1, DRM2 and CMT3].
                </a>
            </h2>
            <p>Cytosine methylation is an epigenetic mark found in the genome of fungi, plants, and animals. <b>DNA methylation is catalyzed by DNA methyltransferases.</b> <b>The function of DNA methyltransferases was shown to be highly conversed, but biological role of these enzymes has not been clearly defined.</b> <b>We generated transgenic plants expressing METHYLTRANSFERASES::GUS reporter genes for three major DNA methyltransferases (MET1, DRM2 and CMT3) to gain insight into the potential physiological relevance of the distinct members of the DNA methyltransferase family in Arabidopsis thaliana, and to investigate the expression patterns in detail.</b> We found that METHYLTRANSFERASE::GUS genes display unique tissue, cell-type, and temporal patterns of expression throughout normal development, particularly in the flower. Our findings are supported by semi-quantitative reverse-transcription PCR, as well as by analyses of microarray databases. <b>These data suggest that DNA methyltransferase may contribute to morphogenesis at every developmental stage and in every plant organ.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25842166" target="_blank">
                    Caveolin-1 is transcribed from a hypermethylated promoter to mediate colonocyte differentiation and apoptosis.
                </a>
            </h2>
            <p>Caveolin-1(CAV1) is a tyrosine-phosphorylated scaffold protein of caveolae with multiple interacting partners. It functions both as an oncogene and a tumour suppressor depending upon the cellular contexts. In the early stage of colorectal cancers (CRC), CAV1 suppresses tumour progression, while over-expression of CAV1 reduced the tumourigenicity of colon carcinoma cells. <b>In contrast, elevated level of CAV1 was reported in stage III CRC.</b> <b>To address this ambiguity, we studied the functional role and the regulation of CAV1 expression during colonocyte differentiation and apoptosis.</b> <b>Here, we reported for the first time that CAV1 expression was increased during colonocyte differentiation and mediated butyrate-induced differentiation and apoptosis of HT29 cells.</b> <b>CAV1 expression was silenced by promoter hypermethylation in HT-29 cells and reactivated by prolonged histone hyperacetylation of the promoter upon treatment of the cells with butyrate.</b> However, the methylation status was unaltered by butyrate. <b>We for the first time showed that HDAC inhibitor-mediated transactivation of CAV1 was regulated by methylation density of the promoter.</b> Our study further explains the underlying mechanisms of the anti-cancer property of butyrate in CRC.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25841986" target="_blank">
                    Consequences of dietary methyl donor supplements: Is more always better?
                </a>
            </h2>
            <p>Epigenetic mechanisms are now recognized to play roles in disease etiology. Several diseases increasing in frequency are associated with altered DNA methylation. <b>DNA methylation is accomplished through metabolism of methyl donors such as folate, vitamin B12, methionine, betaine (trimethylglycine), and choline.</b> Increased intake of these compounds correlates with decreased neural tube defects, although this mechanism is not well understood. <b>Consumption of these methyl donor pathway components has increased in recent years due to fortification of grains and high supplemental levels of these compounds (e.g.</b> vitamins, energy drinks). <b>Additionally, people with mutations in one of the enzymes that assists in the methyl donor pathway (5-MTHFR) are directed to consume higher amounts of methyl donors to compensate.</b> <b>Recent evidence suggests that high levels of methyl donor intake may also have detrimental effects.</b> <b>Individualized medicine may be necessary to determine the appropriate amounts of methyl donors to be consumed, particularly in women of child bearing age.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25840961" target="_blank">
                    Temporal expression and DNA hypomethylation profile of CD30 in Marek's disease virus-infected chicken spleens.
                </a>
            </h2>
            <p>Marek's disease (MD) is a viral neoplastic disease of chickens caused by Marek's disease virus (MDV), which is serious threat to worldwide poultry industry. Our previous studies showed that CD30 gene was hypomethylated in MD lymphoma. In this study, we further analyzed differential expression patterns and methylation levels of CD30 gene between MDV-infected and noninfected spleens at 4, 7, 14, 21, and 28 d postinfection (dpi). <b>The results showed that the expression of CD30 in MDV-infected spleens was significantly lower than that in noninfected spleens at 4 dpi.</b> <b>The expression of CD30 did not present significant difference between MDV-infected and noninfected spleens at 7 and 14 dpi.</b> <b>However, an increased expression of CD30 was presented in MDV-infected spleens at both 21 and 28 dpi.</b> <b>Simultaneously, CD30 showed a lower DNA methylation level in MDV-infected spleens at 14, 21, and 28 dpi.</b> <b>The results indicated that CD30 gene was involved in the whole process of MD tumorigenesis and upregulated expression of CD30 in MDV-infected spleens might be attributed to the hypomethylation of promoter of CD30 gene.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25840838" target="_blank">
                    Early life lipid profile and metabolic programming in very young children.
                </a>
            </h2>
            <p>Lipid derangements during early postnatal life may induce stable epigenetic changes and alter metabolic programming. We investigated associations between serum lipid profiles in very young children and DNA methylation of tumor necrosis factor-alpha (TNFα) and leptin (LEP). Secondly, we explored if the maternal serum lipid profile modifies DNA methylation in the child.In 120 healthy children at 17 months of age, DNA methylation of TNFα and LEP was measured in DNA derived from whole blood. Linear mixed models were used to calculate exposure-specific differences and associations. Total cholesterol in children was associated with decreased methylation of TNFα (-5.8%, p&nbsp;=&nbsp;0.036), and HDL-cholesterol was associated with decreased methylation of both TNFα (-6.9%, p&nbsp;=&nbsp;0.013) and LEP (-3.4%, p&nbsp;=&nbsp;0.021). Additional adjustment for gestational age at birth, birth weight, sex, breastfeeding and educational level attenuated the effects, TNFα (-6.1%, p&nbsp;=&nbsp;0.058) and LEP (-3.1%, p&nbsp;=&nbsp;0.041). In mothers, HDL-cholesterol only was associated with decreased methylation of TNFα in the child (-8.7%, p&nbsp;=&nbsp;0.001).Our data support the developmental origin of health and disease hypothesis by showing that total cholesterol and HDL-cholesterol levels in very young children are associated with epigenetic metabolic programming, which may affect their vulnerability for developing cardiovascular diseases in later life.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25840828" target="_blank">
                    Multiple epigenetic factors predict the attention deficit/hyperactivity disorder among the Chinese Han children.
                </a>
            </h2>
            <p>Attention deficit/hyperactivity disorder (ADHD) is one of the most common psychiatric disorders of childhood. Despite its prevalence, the critical factors involved in its development remain to be identified. It was recently suggested that epigenetic mechanisms probably contribute to the etiology of ADHD. The present study was designed to examine the associations of epigenetic markers with ADHD among Chinese Han children, aiming to establish the prediction model for this syndrome from the epigenetic perspective. We conducted a pair-matching case-control study, and the ADHD children were systematically evaluated via structured diagnostic interviews, including caregiver interviews, based on the Diagnostic and Statistical Manual of Mental Disorders, 4th edition, revised criteria (DSM-IV-R). The expression levels of risk genes DAT1, DRD4, DRD5, as well as their promoter methylation, were determined respectively, followed by the expression profiles of histone-modifying genes p300, MYST4, HDAC1, MeCP2. <b>The multivariate logistic regressions were performed to establish ADHD prediction models.</b> <b>All of the seven genes tested were identified as risk factors for ADHD.</b> <b>The methylation of one critical CpG site located upstream of DRD4 was shown to affect its transcription, suggesting a role in ADHD's development.</b> <b>Aberrant DNA methylation and histone acetylation were indicated in ADHD patients.</b> In addition, a prediction model was established using the combination of p300, MYST4 and HDAC1, with the accuracy of 0.9338. <b>This is, to our knowledge, the first study to clearly demonstrate the associations between epigenetic markers and ADHD, shedding light on the preliminary diagnosis and etiological studies of this widespread disorder.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25840049" target="_blank">
                    Early onset schizophrenia: Gender analysis of genome-wide potential methylation.
                </a>
            </h2>
            <p><b>Methylation studies show that there are substantial gender differences in DNA methylation.</b> On the other hand, in schizophrenia male gender is strongly associated with early onset. <b>The primary aims of the current study are: 1) to identify CpG SNPs across the genome in schizophrenia patients; and 2) to investigate gender differences in potential methylation considering the CpG SNPs at locus, gene and chromosome levels.In this pilot analysis, we have collected detailed clinical information and DNA samples from 16 schizophrenia patients, allowing us to calculate genome-wide potential methylation at genome level in ten males and six females.</b> This cross-sectional DNA sample included subjects with a diagnosis of schizophrenia that were genotyped using the Illumina Omni 2.5 Quad. We applied a novel genetic association strategy, selecting only the CpG SNPs across the genome and analyzed under additive model, to detect gender differences. The mapping analysis was conducted using a specific bioinformatic tool that we have developed, which analyzes only the polymorphic CpG sites genome wide. <b>The bioinformatic tool can detect the SNPs that are affecting the polymorphic CpG sites across the genome.In the SNP-wise analysis, the top autosomal SNP was rs12619000 with 50% potential methylation in males and 95% in females (p=0.000008).</b> <b>In the gene-wise analysis, the KCNG3 was significantly associated with higher potential methylation in males (p=0.0004).The overall results show no robust association between CpG SNPs and gender however the information of the SNP CpG potential methylation can be used for future methylation analysis.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25840015" target="_blank">
                    Ultrasensitive homogeneous electrochemical strategy for DNA methyltransferase activity assay based on autonomous exonuclease III-assisted isothermal cycling signal amplification.
                </a>
            </h2>
            <p><b>DNA methylation catalyzed by methyltransferase (MTase) plays an important role in many biological processes, including gene transcription, genomic imprinting and cellular differentiation.</b> <b>Herein, a simple and novel homogeneous electrochemical strategy for ultrasensitive DNA MTase activity assay has been successfully developed, which is based on methylation-triggered exonuclease (Exo) III-assisted autonomous isothermal cycling signal amplification.</b> <b>A duplex DNA (P1-P2 hybrid) containing the methylation-responsive sequence is ingeniously designed.</b> <b>In the presence of DNA adenine methylation (Dam) methyltransferase (MTase), P1-P2 hybrid is methylated and subsequently recognized and cleaved by Dpn I endonuclease, which triggers the Exo III-catalyzed autonomous cycling cleavage processes.</b> Therefore, a large amount of methylene blue-labeled mononucleotides are released, generating a significantly amplified electrochemical signal toward the Dam MTase activity assay. The directly measured detection limit down to 0.004U/mL is obtained, which is one or two orders magnitude lower than that of the approaches reported in literature. Since this assay is carried out in homogeneous solution phase under isothermal condition and sophisticated probe immobilization processes are avoided, it is very simple and easy to implement. <b>Due to its advantages of ultrahigh sensitivity, excellent selectivity and simple operation, the as-proposed strategy has great potential in the applications in DNA methylation related clinical practices and biochemical researches.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25839993" target="_blank">
                    Epigenetic Regulation in Heart Failure: Part II DNA and Chromatin.
                </a>
            </h2>
            <p>Epigenetic regulatory mechanisms play key roles in cardiac development, differentiation, homeostasis, response to stress and injury and disease. <b>Human heart failure (HF) epigenetic regulatory mechanisms have not deciphered to date.</b> <b>This two part review distills the rapidly evolving research focused on human HF epigenetic regulatory mechanisms.</b> <b>Part I, which was published in the September/October issue, focused on epigenetic regulatory mechanisms involving RNA, specifically the role of short, intermediate and long non-coding RNAs and endogenous competing RNA regulatory networks.</b> <b>Part II, now in the November/December issue, focuses on the epigenetic regulatory mechanisms involving DNA, including DNA methylation, histone modifications and chromatin conformational changes.</b> <b>Part II concludes with two examples of well-studied integrated epigenetic regulatory mechanisms, the structural and functional roles of the Mediator complex in regulating transcription and the epigenetic networked "cross-talk" regulating atrial natriuretic peptide and brain natriuretic peptide promoter activation.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25839231" target="_blank">
                    TSP1-producing B cells restore Ag specific immune tolerance in an allergic environment.
                </a>
            </h2>
            <p>To restore the antigen (Ag) specific immune tolerance in an allergic environment is refractory. B cells are involved in immune regulation. Whether B cells facilitate the generation of Ag specific immune tolerance in an allergic environment is to be further investigated. This paper aims to elucidate the mechanism by which B cells restore the Ag specific immune tolerance in an allergic environment. <b>In this study, a B cell-deficient mouse model was created by injecting an anti-CD20 antibody.</b> The frequency of tolerogenic dendritic cell (TolDC) was assessed by flow cytometry. The levels of cytokines were determined by enzyme-linked immunosorbent assay. The expression of thrombospondin-1 (TSP1) was assessed by quantitative real time RT-PCR, Western blotting and the methylation specific PCR. <b>The results showed that B cells were required in the generation of the transforming growth factor (TGF)-β-producing TolDCs in mice.</b> <b>B cell-derived TSP1 converted the latent TGF-β to the active TGF-β in DCs, which generated TGF-β-producing TolDCs.</b> <b>Exposure to interleukin (IL)-13 inhibited the expression of TSP1 in B cells by enhancing the TSP1 gene DNA methylation.</b> Treating food allergy mice with specific Ag immunotherapy and IL-13 antagonists restored the generation of TolDCs and enhanced the effect of specific immunotherapy. <b>In conclusion, B cells play a critical role in the restoration of specific immune tolerance in an allergic environment.</b> Blocking IL-13 in an allergic environment facilitates the generation of TolDCs and enhanced the therapeutic effect of immunotherapy.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25838424" target="_blank">
                    Endothelial Epigenetics in Biomechanical Stress: Disturbed Flow-Mediated Epigenomic Plasticity In Vivo and In Vitro.
                </a>
            </h2>
            <p><b>Arterial endothelial phenotype is regulated by local hemodynamic forces that are linked to regional susceptibility to atherogenesis.</b> A complex hierarchy of transcriptional, translational, and post-translational mechanisms is greatly influenced by the characteristics of local arterial shear stress environments. <b>We discuss the emerging role of localized disturbed blood flow on epigenetic mechanisms of endothelial responses to biomechanical stress, including transcriptional regulation by proximal promoter DNA methylation, and post-transcriptional and translational regulation of gene and protein expression by chromatin remodeling and noncoding RNA-based mechanisms.</b> <b>Dynamic responses to flow characteristics in vivo and in vitro include site-specific differentially methylated regions of swine and mouse endothelial methylomes, histone marks regulating chromatin conformation, microRNAs, and long noncoding RNAs.</b> <b>Flow-mediated epigenomic responses intersect with cis and trans factor regulation to maintain endothelial function in a shear-stressed environment and may contribute to localized endothelial dysfunctions that promote atherosusceptibility.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25838386" target="_blank">
                    Epigenetics. Restricted epigenetic inheritance of H3K9 methylation.
                </a>
            </h2>
            <p>Posttranslational histone modifications are believed to allow the epigenetic transmission of distinct chromatin states, independently of associated DNA sequences. <b>Histone H3 lysine 9 (H3K9) methylation is essential for heterochromatin formation; however, a demonstration of its epigenetic heritability is lacking.</b> <b>Fission yeast has a single H3K9 methyltransferase, Clr4, that directs all H3K9 methylation and heterochromatin.</b> <b>Using releasable tethered Clr4 reveals that an active process rapidly erases H3K9 methylation from tethering sites in wild-type cells.</b> <b>However, inactivation of the putative histone demethylase Epe1 allows H3K9 methylation and silent chromatin maintenance at the tethering site through many mitotic divisions, and transgenerationally through meiosis, after release of tethered Clr4.</b> <b>Thus, H3K9 methylation is a heritable epigenetic mark whose transmission is usually countered by its active removal, which prevents the unauthorized inheritance of heterochromatin.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25838322" target="_blank">
                    BDNF promoter methylation associated with suicidal ideation in patients with breast cancer.
                </a>
            </h2>
            <p>Brain-derived neurotrophic factor (BDNF) has been considered a risk factor for suicidality. BDNF secretion is influenced by epigenetic (DNA methylation) and genetic (val66met polymorphism) profiles. <b>We aimed to investigate the independent effects of BDNF promoter methylation status on suicidal ideation as well as the effects of its interaction with the val66met polymorphism in patients with breast cancer.A total of279 patients with breast cancer were evaluated 1 week after breast surgery, and 244 (87%) were followed up 1 year later.</b> Suicidal ideation was identified using the item addressing suicidal thoughts on the Beck Depression Inventory. <b>The independent effects of BDNF methylation status on suicidal ideation at two points was investigated using multivariable logistic regression models.</b> <b>The two-way interactive effects of BDNF methylation status and the val66met polymorphism on suicidal ideation were also estimated using the same models.Increased BDNF methylation was significantly associated with suicidal ideation and depression 1 year after breast surgery, and this association was independent of potential covariates, including previous depression, current depressive symptoms, and BDNF genotype.</b> <b>No significant methylation-genotype interactions were found.The BDNF hypothesis and the epigenetic origin of suicidality in patients with breast cancer were supported.</b> <b>BDNF gene methylation status may be a biological marker for suicidality in patients with breast cancer.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25838002" target="_blank">
                    Toward understanding the role of the neuron-specific BAF chromatin remodeling complex in memory formation.
                </a>
            </h2>
            <p>The long-term storage of memory requires the finely tuned coordination of intracellular signaling with the transcriptional, translational and epigenetic regulations of gene expression. Among the epigenetic mechanisms, however, we know relatively little about the involvement of chromatin remodeling-dependent control of gene expression in cognitive brain functions, compared with our knowledge of other such mechanisms (for example, histone modifications and DNA methylation). <b>A few recent studies have implicated the Brm/Brg-associated factor (BAF) chromatin-remodeling complex, a mammalian homolog of the yeast Swi/Snf complex, in neuronal structural/functional plasticity and memory formation.</b> <b>The BAF complex was previously known to have a critical role in neurodevelopment, but these recent findings indicate that it also contributes to both cognitive functions in the adult brain and human mental disorders characterized by intellectual disability.</b> <b>In this review, we provide a brief overview of the BAF complexes, introduce recent research findings that link their functions to memory formation, and speculate on the yet-unknown molecular mechanisms that may be relevant to these processes.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25837868" target="_blank">
                    The p53 network as therapeutic target in gastroenteropancreatic neuroendocrine neoplasms.
                </a>
            </h2>
            <p>Gastroenteropancreatic neuroendocrine neoplasms (GEP-NENs) are heterogeneous and especially the midgut tumors currently lack effective therapy options. Actionable driver mutations as therapeutic targets are rare. Subtype specific data concerning regulatory mechanisms or epigenetic aberrations are necessary for novel clinical trials. <b>Although the p53 protein itself is rarely mutated in GEP-NENs, epigenetic and regulatory aberrations interfere with the p53 network activity and might function as s target for novel therapeutic approaches.</b> <b>In this review we analyze the current knowledge about the p53 network in GEP-NENs and discuss three possible strategies that include recovering p53 function, enforcing apoptosis by genotoxic stress induction and restoring silenced gene function, based on in vitro, in vivo and clinical data.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25837417" target="_blank">
                    Profiling DNA methylome landscapes of mammalian cells with single-cell reduced-representation bisulfite sequencing.
                </a>
            </h2>
            <p>The heterogeneity of DNA methylation within a population of cells necessitates DNA methylome profiling at single-cell resolution. Recently, we developed a single-cell reduced-representation bisulfite sequencing (scRRBS) technique in which we modified the original RRBS method by integrating all the experimental steps before PCR amplification into a single-tube reaction. These modifications enable scRRBS to provide digitized methylation information on ∼1 million CpG sites within an individual diploid mouse or human cell at single-base resolution. Compared with the single-cell bisulfite sequencing (scBS) technique, scRRBS covers fewer CpG sites, but it provides better coverage for CpG islands (CGIs), which are likely to be the most informative elements for DNA methylation. The entire procedure takes ∼3 weeks, and it requires strong molecular biology skills.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25837375" target="_blank">
                    The alternative role of DNA methylation in splicing regulation.
                </a>
            </h2>
            <p><b>Although DNA methylation was originally thought to only affect transcription, emerging evidence shows that it also regulates alternative splicing.</b> <b>Exons, and especially splice sites, have higher levels of DNA methylation than flanking introns, and the splicing of about 22% of alternative exons is regulated by DNA methylation.</b> <b>Two different mechanisms convey DNA methylation information into the regulation of alternative splicing.</b> <b>The first involves modulation of the elongation rate of RNA polymerase II (Pol II) by CCCTC-binding factor (CTCF) and methyl-CpG binding protein 2 (MeCP2); the second involves the formation of a protein bridge by heterochromatin protein 1 (HP1) that recruits splicing factors onto transcribed alternative exons.</b> These two mechanisms, however, regulate only a fraction of such events, implying that more underlying mechanisms remain to be found.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25836763" target="_blank">
                    Chromatin modifiers and histone modifications in bone formation, regeneration, and therapeutic intervention for bone-related disease.
                </a>
            </h2>
            <p><b>Post-translational modifications of chromatin such as DNA methylation and different types of histone acetylation, methylation and phosphorylation are well-appreciated epigenetic mechanisms that confer information to progeny cells during lineage commitment.</b> <b>These distinct epigenetic modifications have defined roles in bone, development, tissue regeneration, cell commitment and differentiation, as well as disease etiologies.</b> <b>In this review, we discuss the role of these chromatin modifications and the enzymes regulating these marks (methyltransferases, demethylases, acetyltransferases, and deacetylases) in progenitor cells, osteoblasts and bone-related cells.</b> <b>In addition, the clinical relevance of deregulated histone modifications and enzymes as well as current and potential therapeutic interventions targeting chromatin modifiers are addressed.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25836705" target="_blank">
                    Common Genetic and Epigenetic Syndromes.
                </a>
            </h2>
            <p><b>Cytogenetic anomalies should be considered in individuals with multiple congenital anomalies.</b> DNA methylation analysis is the most sensitive initial test in evaluating for Prader-Willi and Angelman syndromes. <b>The timely identification of cytogenetic anomalies allows for prompt initiation of early intervention services to maximize the potential of every individual as they grow older.</b> Although many of these conditions are rare, keeping them in mind can have a profound impact on the clinical course of affected individuals. This article reviews some of the more common genetic syndromes.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25836049" target="_blank">
                    Reversed phase and cation exchange liquid chromatography with spectrophotometric and elemental/molecular mass spectrometric detection for S-adenosyl methionine/S-adenosyl homocysteine ratios as methylation index in cell cultures of ovarian cancer.
                </a>
            </h2>
            <p>S-adenosylmethionine (SAM) and S-adenosylhomocysteine (SAH) are essential compounds in the carbon metabolic cycle that have clinical implications in a broad range of disease conditions. <b>The measurement of the ratio SAM/SAH also called methylation index, has become a way of monitoring the DNA methylation of a cell which is an epigenetic event with important clinical implications in diagnosis; therefore the development of suitable methods to accurately quantify these compounds is mandatory.</b> <b>This work illustrates the comparison of three independent methods for the determination of the methylation index, all of them based on the chromatographic separation of the two species (SAM and SAH) using either ion-pairing reversed phase or cation exchange chromatography.</b> The species detection was conducted using either molecular absorption spectrophotometry (HPLC-UV) or mass spectrometry with electrospray (ESI-MS/MS) as ionization source or inductively coupled plasma (DF-ICP-MS) by monitoring the S-atom contained in both analytes. The analytical performance characteristics of the three methods were critically compared obtaining best features for the combination of reversed phase HPLC with ESI-MS in the MRM mode. <b>In this case, detection limits of about 0.5ngmL(-1) for both targeted analytes permitted the application of the designed strategy to evaluate the effect of cisplatin on the changes of the methylation index among epithelial ovarian cancer cell lines sensitive (A2780) and resistant (A2780CIS) to this drug after exposition to cisplatin.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25833781" target="_blank">
                    Dietary modifications, weight loss, and changes in metabolic markers affect global DNA methylation in Hispanic, african american, and afro-Caribbean breast cancer survivors.
                </a>
            </h2>
            <p>Lower levels of global DNA methylation in tissue and blood have been associated with increased cancer risk. Conversely, cross-sectional analyses of healthier lifestyle patterns have been associated with higher levels of global DNA methylation.In this trial, we explored the associations between changes in lifestyle modifications (diet, weight loss), metabolic markers, and global epigenetic biomarkers in white blood cells.Study participants were Hispanic, African American, and Afro-Caribbean overweight and sedentary female breast cancer survivors (n = 24) who participated in a larger randomized, crossover, pilot study of a 6-mo weight loss intervention and who had available blood specimens. Anthropometric measures, a food-frequency questionnaire, and peripheral blood were collected at baseline, 6 mo, and 12 mo. Plasma samples were analyzed for metabolic markers (insulin, glucose). We measured DNA methylation of long interspersed nucleotide element 1 (LINE-1) and satellite 2 by pyrosequencing and MethyLight, respectively, and global DNA methylation by the luminometric methylation assay (LUMA).DNA methylation of LINE-1 was statistically significantly elevated at 6 mo [75.5% vs. 78.5% (P &lt; 0.0001)] and 12 mo [75.5% vs. 77.7% (P &lt; 0.0001)], compared to baseline. Over a 12-mo period, changes in percentage body fat and plasma glucose concentrations were positively associated with LINE-1 DNA methylation (β = 0.19, P = 0.001) and LUMA DNA methylation levels (β = 0.24, P = 0.02), respectively. Similarly, 12-mo changes in dietary measures such as vegetable (β = 0.009, P = 0.048), protein (β = 0.04, P = 0.001), and total caloric (β = 0.05, P = 0.01) intake were positively associated with changes in LUMA DNA methylation, as was intake of fruit positively associated with changes in LINE-1 DNA methylation (β = 0.004, P = 0.02).Our hypothesis-generating results suggest that lifestyle modifications may be associated with changes in global DNA methylation detectable at 6 and 12 mo. These biomarkers may be useful intermediate biomarkers to use in future intervention trials. This trial was registered at clinicaltrials.gov as NCT00811824.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25833887" target="_blank">
                    Epigenetics of Inflammation, Maternal Infection, and Nutrition.
                </a>
            </h2>
            <p><b>Studies have demonstrated that epigenetic changes such as DNA methylation, histone modification, and chromatin remodeling are linked to an increased inflammatory response as well as increased risk of chronic disease development.</b> A few studies have begun to investigate whether dietary nutrients play a beneficial role by modifying or reversing epigenetically induced inflammation. <b>Results of these studies show that nutrients modify epigenetic pathways.</b> <b>However, little is known about how nutrients modulate inflammation by regulating immune cell function and/or immune cell differentiation via epigenetic pathways.</b> <b>This overview will provide information about the current understanding of the role of nutrients in the epigenetic control mechanisms of immune function.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25833840" target="_blank">
                    Chromosomal meiotic segregation, embryonic developmental kinetics and DNA (hydroxy)methylation analysis consolidate the safety of human oocyte vitrification.
                </a>
            </h2>
            <p>Oocyte vitrification has been introduced into clinical settings without extensive pre-clinical safety testing. In this study, we analysed major safety aspects of human oocyte vitrification in a high security closed system: (i) chromosomal meiotic segregation, (ii) embryonic developmental kinetics and (iii) DNA (hydroxy)methylation status. Fresh and vitrified sibling oocytes from young donors after intracytoplasmic sperm injection (ICSI) were compared in three different assays. <b>Firstly, the chromosomal constitution of the fertilized zygotes was deduced from array comparative genomic hybridization results obtained from both polar bodies biopsied at Day 1.</b> <b>Secondly, embryo development up to Day 3 was analysed by time-lapse imaging.</b> <b>Ten specific time points, six morphokinetic time intervals and the average cell number on Day 3 were recorded.</b> <b>Thirdly, global DNA methylation and hydroxymethylation patterns were analysed by immunostaining on Day 3 embryos.</b> The nuclear fluorescence intensity was measured by Volocity imaging software. Comprehensive chromosomal screening of the polar bodies demonstrated that at least half of the zygotes obtained after ICSI of fresh and vitrified oocytes were euploid. Time-lapse analysis showed that there was no significant difference in cleavage timings, the predictive morphokinetic time intervals nor the average cell number between embryos developed from fresh and vitrified oocytes. <b>Finally, global DNA (hydroxy)methylation patterns were not significantly different between Day 3 embryos obtained from fresh and from vitrified oocytes.</b> Our data further consolidate the safety of the oocyte vitrification technique. Nevertheless, additional testing in young and older sub-fertile/infertile patients and sound follow-up studies of children born after oocyte cryopreservation remain mandatory.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25833708" target="_blank">
                    Glycogen Synthase Kinase-3 (Gsk-3) Plays A Fundamental Role In Maintaining DNA Methylation At Imprinted Loci In Mouse Embryonic Stem Cells.
                </a>
            </h2>
            <p>Glycogen synthase kinase-3 (Gsk-3) is a key regulator of multiple signal transduction pathways. Recently, we described a novel role for Gsk-3 in the regulation of DNA methylation at imprinted loci in mouse embryonic stem cells (ESCs), suggesting that epigenetic changes regulated by Gsk-3 are likely an unrecognized facet of Gsk-3 signaling. Here, we extend our initial observation to the entire mouse genome by enriching for methylated DNA with the MethylMiner kit and performing next-generation sequencing (MBD-Seq) in wild-type and Gsk-3α(-/-);Gsk-3β(-/-) ESCs. Consistent with our previous data, we found that 77% of known imprinted loci have reduced DNA methylation in Gsk-3-deficient ESCs. More specifically, we unambiguously identified changes in DNA methylation within regions that have been confirmed to function as imprinting control regions (ICRs). In many cases, the reduced DNA methylation at imprinted loci in Gsk-3α(-/-);Gsk-3β(-/-) ESCs was accompanied by changes in gene expression as well. Furthermore, many of the Gsk-3-dependent differentially methylated regions (DMRs) are identical to the DMRs recently identified in uniparental ESCs. Our data demonstrate the importance of Gsk-3 activity in the maintenance of DNA methylation at a majority of the imprinted loci in ESCs, and emphasizes the importance for Gsk-3-mediated signal transduction on the epigenome.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25833413" target="_blank">
                    Global DNA methylation and hydroxymethylation differ in hepatocellular carcinoma and cholangiocarcinoma and relate to survival rate.
                </a>
            </h2>
            <p>In addition to DNA methylation, hydroxymethylation of DNA is recognized as a novel epigenetic mark. Primary liver cancers, i.e. hepatocellular carcinoma (HCC) and cholangiocarcinoma (CC) are highly prevalent but epigenetically poorly characterized, so far. In the present study we measured global methylcytosine (mCyt) and hydroxymethylcytosine (hmCyt) in HCC and CC tissues, and in peripheral blood mononuclear cells (PBMCs) DNA to define mCyt and hmCyt status and, accordingly, the survival rate. The mCyt and hmCyt was measured by an LC/MS/MS method in neoplastic and homologous non-neoplastic tissues, i.e. liver and gall bladder, and in PBMCs of 31 HCC and 16 CC patients. Methylcytosine content was notably lower in HCC than in CC tissues (3.97% vs. 5.26% respectively, p&lt;0.0001). Significantly reduced mCyt was also detected in HCC as compared to non-neoplastic tissue (3.97% vs. 4.82% mCyt, p&lt;0.0001) but no such difference was found for CC vs. homologous non-neoplastic tissue. Hydroxymethylation was significantly decreased in HCC vs. non-neoplastic liver tissue (0.044 vs. 0.128, p&lt;0.0001), and in CC vs. both liver and gall bladder non-neoplastic tissue (0.030 vs. 0.124, p=0.026; 0.030 vs. 0.123, p=0.006, respectively). When the survival rate was evaluated according to mCyt PBMCs content by Kaplan-Meier analysis, patients with mCyt≥5.59% had a significantly higher life expectancy than those with mCyt&lt;5.59% (p=0.034) at a follow-up period up to 48 months.A significant DNA hypomethylation distinguishes HCC from CC while DNA hypo-hydroxymethylation characterizes both HCC and CC. PBMCs DNA mCyt content≥5.59% relates to a favorable outcome in primary liver cancers. This article is protected by copyright. All rights reserved.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25832671" target="_blank">
                    DNA Methylation in Memory Formation: Emerging Insights.
                </a>
            </h2>
            <p>The establishment of synaptic plasticity and long-term memory requires lasting cellular and molecular modifications that, as a whole, must endure despite the rapid turnover of their constituent parts. Such a molecular feat must be mediated by a stable, self-perpetuating, cellular information storage mechanism. <b>DNA methylation, being the archetypal cellular information storage mechanism, has been heavily implicated as being necessary for stable activity-dependent transcriptional alterations within the CNS.</b> <b>This review details the foundational discoveries from both gene-targeted and whole-genome sequencing studies that have brought DNA methylation to our attention as a chief regulator of activity- and experience-dependent transcriptional alterations within the CNS.</b> <b>We present a hypothetical framework to resolve disparate experimental findings regarding distinct manipulations of DNA methylation and their effect on memory, taking into account the unique impact activity-dependent alterations in DNA methylation potentially have on both memory-promoting and memory-suppressing gene expression.</b> <b>And last, we discuss potential avenues for future inquiry into the role of DNA methylation during remote memory formation.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25831549" target="_blank">
                    Epigenetics. Epigenetic inheritance uncoupled from sequence-specific recruitment.
                </a>
            </h2>
            <p><b>Changes in histone posttranslational modifications are associated with epigenetic states that define distinct patterns of gene expression.</b> <b>It remains unclear whether epigenetic information can be transmitted through histone modifications independently of specific DNA sequence, DNA methylation, or RNA interference.</b> <b>Here we show that, in the fission yeast Schizosaccharomyces pombe, ectopically induced domains of histone H3 lysine 9 methylation (H3K9me), a conserved marker of heterochromatin, are inherited through several mitotic and meiotic cell divisions after removal of the sequence-specific initiator.</b> The putative JmjC domain H3K9 demethylase, Epe1, and the chromodomain of the H3K9 methyltransferase, Clr4/Suv39h, play opposing roles in maintaining silent H3K9me domains. <b>These results demonstrate how a direct "read-write" mechanism involving Clr4 propagates histone modifications and allows histones to act as carriers of epigenetic information.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25831497" target="_blank">
                    The frequency natural antisense transcript first promotes, then represses, frequency gene expression via facultative heterochromatin.
                </a>
            </h2>
            <p>The circadian clock is controlled by a network of interconnected feedback loops that require histone modifications and chromatin remodeling. <b>Long noncoding natural antisense transcripts (NATs) originate from Period in mammals and frequency (frq) in Neurospora.</b> <b>To understand the role of NATs in the clock, we put the frq antisense transcript qrf (frq spelled backwards) under the control of an inducible promoter.</b> <b>Replacing the endogenous qrf promoter altered heterochromatin formation and DNA methylation at frq.</b> In addition, constitutive, low-level induction of qrf caused a dramatic effect on the endogenous rhythm and elevated circadian output. <b>Surprisingly, even though qrf is needed for heterochromatic silencing, induction of qrf initially promoted frq gene expression by creating a more permissible local chromatin environment.</b> The observation that antisense expression can initially promote sense gene expression before silencing via heterochromatin formation at convergent loci is also found when a NAT to hygromycin resistance gene is driven off the endogenous vivid (vvd) promoter in the Δvvd strain. <b>Facultative heterochromatin silencing at frq functions in a parallel pathway to previously characterized VVD-dependent silencing and is needed to establish the appropriate circadian phase.</b> Thus, repression via dicer-independent siRNA-mediated facultative heterochromatin is largely independent of, and occurs alongside, other feedback processes.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25831061" target="_blank">
                    Genome-wide DNA methylation modified by soy phytoestrogens: role for epigenetic therapeutics in prostate cancer?
                </a>
            </h2>
            <p>In prostate cancer, DNA methylation is significantly associated with tumor initiation, progression, and metastasis. Previous studies have suggested that soy phytoestrogens might regulate DNA methylation at individual candidate gene loci and that they play a crucial role as potential therapeutic agents for prostate cancer. The purpose of our study was to examine the modulation effects of phytoestrogens on a genome-wide scale in regards to DNA methylation in prostate cancer. Prostate cancer cell lines DU-145 and LNCaP were treated with 40 μM of genistein and 110 μM of daidzein. DNMT inhibitor 5-azacytidine (2 μM) and the methylating agent budesonide (2 μM) were used to compare their demethylation/methylation effects with phytoestrogens. <b>The regulatory effects of phytoestrogens on DNA methylation were analyzed by using a methyl-DNA immunoprecipitation method coupled with Human DNA Methylation Microarrays (MeDIP-chip).</b> <b>We observed that the methylation profiles of 58 genes were altered by genistein and daidzein treatments in DU-145 and LNCaP prostate cancer cells.</b> <b>In addition, the methylation frequencies of the MAD1L1, TRAF7, KDM4B, and hTERT genes were remarkably modified by genistein treatment.</b> <b>Our results suggest that the modulation effects of phytoestrogens on DNA methylation essentially lead to inhibition of cell growth and induction of apoptosis.</b> <b>Genome-wide methylation profiling reported here suggests that epigenetic regulation mechanisms and, by extension, epigenetics-driven novel therapeutic candidates warrant further consideration in future "omics" studies of prostate cancer.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25830902" target="_blank">
                    A novel Werner Syndrome mutation: pharmacological treatment by read-through of nonsense mutations and epigenetic therapies.
                </a>
            </h2>
            <p>Werner Syndrome (WS) is a rare inherited disease characterized by premature aging and increased propensity for cancer. <b>Mutations in the WRN gene can be of several types, including nonsense mutations, leading to a truncated protein form.</b> <b>WRN is a RecQ family member with both helicase and exonuclease activities, and it participates in several cell metabolic pathways, including DNA replication, DNA repair, and telomere maintenance.</b> Here, we reported a novel homozygous WS mutation (c.3767 C &gt; G) in 2 Argentinian brothers, which resulted in a stop codon and a truncated protein (p.S1256X). <b>We also observed increased WRN promoter methylation in the cells of patients and decreased messenger WRN RNA (WRN mRNA) expression.</b> <b>Finally, we showed that the read-through of nonsense mutation pharmacologic treatment with both aminoglycosides (AGs) and ataluren (PTC-124) in these cells restores full-length protein expression and WRN functionality.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25830725" target="_blank">
                    Prolonged re-expression of the hypermethylated gene EPB41L3 using artificial transcription factors and epigenetic drugs.
                </a>
            </h2>
            <p>Epigenetic silencing of tumor suppressor genes (TSGs) is considered a significant event in the progression of cancer. For example, EPB41L3, a potential biomarker in cervical cancer, is often silenced by cancer-specific promoter methylation. Artificial transcription factors (ATFs) are unique tools to re-express such silenced TSGs to functional levels; however, the induced effects are considered transient. Here, we aimed to improve the efficiency and sustainability of gene re-expression using engineered zinc fingers fused to VP64 (ZF-ATFs) or DNA methylation modifiers (ZF-Tet2 or ZF-TDG) and/or by co-treatment with epigenetic drugs [5-aza-dC2'-deoxycytidine or Trichostatin A (TSA)]. EPB41L3-ZF effectively bound its methylated endogenous locus, as also confirmed by ChIP-seq. ZF-ATFs reactivated the epigenetically silenced target gene EPB41L3 (˜10-fold) in breast, ovarian, and cervical cancer cell lines. Prolonged high levels of EPB41L3 (˜150-fold) induction could be achieved by short-term co-treatment with epigenetic drugs. Interestingly, for otherwise ineffective ZF-Tet2 or ZF-TDG treatments, TSA facilitated re-expression of EPB41L3 by up to twofold. ATF-mediated re-expression demonstrated a tumor suppressive role for EPB41L3 in cervical cancer cell lines. In conclusion, epigenetic reprogramming provides a novel way to improve sustainability of re-expression of epigenetically silenced promoters.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25830654" target="_blank">
                    Correlation Between Reversal of DNA Methylation and Clinical Symptoms in Psoriatic Epidermis Following Narrow-Band UVB Phototherapy.
                </a>
            </h2>
            <p>Epigenetic modifications by DNA methylation are associated with a wide range of diseases. Previous studies in psoriasis have concentrated on epigenetic changes in immune cells or in total skin biopsies that include stromal-associated changes. <b>In order to improve our understanding of the role of DNA methylation in psoriasis, we sought to obtain a comprehensive DNA methylation signature specific for the epidermal component of psoriasis and to analyze methylation changes during therapy.</b> <b>Genome-wide DNA methylation profiling of epidermal cells from 12 patients undergoing narrow-band UVB phototherapy and 12 corresponding healthy controls revealed a distinct DNA methylation pattern in psoriasis compared to controls.</b> <b>A total of 3665 methylation variable positions (MVPs) were identified with an overall hypomethylation in psoriasis patient samples.</b> <b>DNA methylation pattern was reversed at the end of phototherapy in patients showing excellent clinical improvement.</b> Only 7% of phototherapy affected MVPs (150 out of 2108) correlates with nearby gene expression. Enrichment of MVPs in enhancers indicates tissue-specific modulation of the transcriptional regulatory machinery in psoriasis. <b>Our study identified key epigenetic events associated with psoriasis pathogenesis and helps to understand the dynamic DNA methylation landscape in the human genome.Journal of Investigative Dermatology accepted article preview online, 01 April 2015. doi:10.1038/jid.2015.128.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25830127" target="_blank">
                    Promoter methylation and expression levels of selected hematopoietic genes in pediatric B-cell acute lymphoblastic leukemia.
                </a>
            </h2>
            <p>Precursor B-cell acute lymphoblastic leukemia (B-cell ALL) is the most common neoplasm in children and is characterized by genetic and epigenetic aberrations in hematopoietic transcription factor (TF) genes. This study evaluated promoter DNA methylation and aberrant expression levels of early- and late-acting hematopoietic TF genes homeobox A4 and A5 (HOXA4 and HOXA5), Meis homeobox 1 (MEIS1), T-cell acute lymphocytic leukemia 1 (TAL1), and interferon regulatory factors 4 and 8 (IRF4 and IRF8) in pediatric B-cell ALL.Blood samples of 38 ALL patients and 20 controls were obtained. DNA was treated with sodium bisulfite and DNA methylation level of HOXA4, HOXA5, MEIS1, TAL1, IRF4, and IRF8 was assessed using quantitative methylation-specific polymerase chain reaction (PCR). <b>Relative gene expression was measured using quantitative reverse transcription-PCR.Aberrant methylation of TAL1, IRF8, MEIS1, and IRF4 was observed in 26.3%, 7.9%, 5.3%, and 2.6% patients, respectively, but not in controls.</b> <b>HOXA4 and HOXA5 were methylated in some controls and hypermethylated in 16% and 5% patients, respectively.</b> <b>IRF8, MEIS1, and TAL1 expression was lower in patients than in controls.</b> MEIS1 expression was inversely correlated with white blood cell (WBC) count. <b>HOXA4 expression was down-regulated in patients with high risk according to the National Cancer Institute (NCI) classification.</b> <b>TAL1 methylation was slightly elevated in patients aged &gt;9 years and in patients showing relapse, suggesting its potential prognostic value.Aberrant methylation and expression of the selected hematopoietic genes were correlated with demographic/clinical prognostic factors of pediatric ALL, such as age, WBC count, and NCI risk classification.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25829970" target="_blank">
                    Premature aging of leukocyte DNA methylation is associated with type 2 diabetes prevalence.
                </a>
            </h2>
            <p>Type 2 diabetes mellitus (T2D) is highly prevalent in Middle-Eastern and North African Arab populations, but the molecular basis for this susceptibility is unknown. <b>Altered DNA methylation levels were reported in insulin-secreting and responding tissues, but whether methylation in accessible tissues such as peripheral blood is associated with T2D risk remains an open question.</b> <b>Age-related alteration of DNA methylation level was reported in certain methylation sites, but no association with T2D has been shown.</b> Here we report on a population-based study of 929 men and women representing the East Jerusalem Palestinian (EJP) Arab population and compare with the findings among Israeli Ashkenazi Jews. <b>This is the first reported epigenetic study of an Arab population with a characteristic high prevalence of T2D.We found that DNA methylation of a prespecified regulatory site in peripheral blood leukocytes (PBLs) is associated with impaired glucose metabolism and T2D independent of sex, body mass index, and white blood cell composition.</b> This CpG site (Chr16: 53,809,231-2; hg19) is located in a region within an intron of the FTO gene, suspected to serve as a tissue-specific enhancer. <b>The association between PBL hypomethylation and T2D varied by age, revealing differential patterns of methylation aging in healthy and diabetic individuals and between ethnic groups: T2D patients displayed prematurely low methylation levels, and this hypomethylation was greater and occurred earlier in life among Palestinian Arabs than Ashkenazi Jews.Our study suggests that premature DNA methylation aging is associated with increased risk of T2D.</b> <b>These findings should stimulate the search for more such sites and may pave the way to improved T2D risk prediction within and between human populations.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25829251" target="_blank">
                    MiR-101 reverses the hypomethylation of the LMO3 promoter in glioma cells.
                </a>
            </h2>
            <p>LIM-only protein 3 (LMO3), a member of the LIM-only protein group, is a new DNA methylation gene that was identified in gliomas via the MeDIP-Chip in our previous study. <b>In this study, we found that LIM-only protein 3 (LMO3) is hypomethylated and overexpressed in glioma cells and tissues.</b> <b>The overexpression of LMO3 was correlated with a poor prognosis in glioma patients, and LMO3 was indirectly inhibited by the tumor suppressor miR-101, which is a potential prognosis marker of gliomas.</b> <b>MiR-101 decreased the expression of LMO3 by reversing the methylation status of the LMO3 promoter and by inhibiting the presence of the methylation-related histones H3K4me2 and H3K27me3 and increasing the presence of H3K9me3 and H4K20me3 on the promoter.</b> <b>It was determined that miR-101 decreases the occupancy of H3K27me3 by inhibiting EZH2, DNMT3A and EED and decreases the H3K9me3 occupancy on the LMO3 promoter via SUV39H1, SUV39H2, G9a and PHF8.</b> <b>Furthermore, miR-101 suppresses the expression of LMO3 by decreasing USF and MZF1.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25829105" target="_blank">
                    MWCNT uptake in Allium cepa root cells induces cytotoxic and genotoxic responses and results in DNA hyper-methylation.
                </a>
            </h2>
            <p>Advances in nanotechnology have led to the large-scale production of nanoparticles, which, in turn, increases the chances of environmental exposure. While humans (consumers/workers) are primarily at risk of being exposed to the adverse effect of nanoparticles, the effect on plants and other components of the environment cannot be ignored. <b>The present work investigates the cytotoxic, genotoxic, and epigenetic (DNA methylation) effect of MWCNT on the plant system- Allium cepa.</b> MWCNT uptake in root cells significantly altered cellular morphology. Membrane integrity and mitochondrial function were also compromised. <b>The nanotubes induced significant DNA damage, micronucleus formation and chromosome aberration.</b> <b>DNA laddering assay revealed the formation of internucleosomal fragments, which is indicative of apoptotic cell death.</b> This finding was confirmed by an accumulation of cells in the sub-G0 phase of the cell cycle. An increase in CpG methylation was observed using the isoschizomers MspI/HpaII. <b>HPLC analysis of DNA samples revealed a significant increase in the levels of 5-methyl-deoxy-cytidine (5mdC).</b> These results confirm the cyto-genotoxic effect of MWCNT in the plant system and simultaneously highlight the importance of this epigenetic study in nanoparticle toxicity.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25829012" target="_blank">
                    Small RNAs: Artificial piRNAs for Transcriptional Silencing.
                </a>
            </h2>
            <p><b>Technologies have been developed in animal germ cells that produce artificial piRNAs from transgenes in piRNA clusters to silence target genes by cleaving their transcripts.</b> <b>A new study provides a simple way to generate artificial piRNAs to direct de novo DNA methylation in mice.</b></p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25828518" target="_blank">
                    CHFR methylation strongly correlates with methylation of DNA damage repair and apoptotic pathway genes in non-small cell lung cancer.
                </a>
            </h2>
            <p>DNA methylation occurs commonly in non-small cell lung cancer (NSCLC). We sought to determine the frequency and relationship of methylation of key genes involved in the pathways of mitotic checkpoint control, DNA damage repair, apoptosis, and growth factor signaling in these patients. We analyzed the DNA methylation status of eight genes (CHFR, FANCF, MGMT, p16, DAPK, ASC or TMS-1, RAR-B, and CRBP1) using nested methylation-specific PCR (MSP) on over 314 paraffin-embedded, human non-small cell lung cancer samples. We determined the methylation frequency of each gene in addition to the association of the methylation of each gene with other members of the panel. <b>Methylation was a common event in these samples.</b> <b>Our methylation analysis showed frequencies of methylation of 10% for CHFR, 14% for FANCF, 30% for MGMT, 29% for p16, 17% for DAPK, 33% for ASC, 38% for RAR-B, and 7% for CRBP1.</b> <b>There was a strong correlation between methylation of the mitotic G2-M checkpoint gene, CHFR, and methylation of other genes in our panel involved in DNA damage repair (FANCF and MGMT) and apoptosis (DAPK and ASC) but not with other genes in our panel, including p16 (the G1-S checkpoint gene), CRBP1, or RAR-B.</b> <b>In addition, MGMT methylation strongly correlated with the pro-apoptotic gene, ASC.</b> There are distinct associations of methylated genes in non-small cell lung cancer involving DNA damage repair, apoptosis, and the G2-M mitotic checkpoint control. <b>Further studies are warranted to determine whether these methylation patterns have implications for prognosis in addition to prediction of response to chemotherapeutic agents commonly used in the treatment of non-small cell lung cancer, such as radiotherapy and platinum- or taxane-based chemotherapy.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25827883" target="_blank">
                    Immunostaining analysis of tissue cultured cells and tissue sections using phospho-histone h3 (serine 10) antibody.
                </a>
            </h2>
            <p>Post-translational modifications of histones play an important role in regulation of gene expression through condensation and decondensation of chromatin structure. These modifications include acetylation, methylation, phosphorylation and ubiquitination. Phosphorylation on histones is associated with cellular responses such as DNA damage, transcription, chromatin compaction and mitosis or meiosis. One of the most extensively studied modifications of histones is the Serine 10 phosphorylation on histone H3 N-terminal tail. <b>This specific phosphorylation on Histone H3 has been associated with condensation and transcriptional inactivation of mitotic chromosomes, but recent studies have suggested a role for this specific phosphorylation in chromatin relaxation and activation of transcription in interphase cells.</b> <b>Co-immunostaining analysis of cells using antibodies specific to serine 10P-Histone H3 together with those to cell cycle specific markers will allow us to determine the nature of phosphorylation in a cell cycle-specific manner.</b> <b>In a complex system, such as tissue specimens, analysis using P-Histone H3 and a cell type specific antibody will allow identification of specific cells that are affected by this histone modification.</b> This is of particular interest in the field of cancer biology or neurobiology where identification or quantification of the transcriptionally active or mitotic cells will enable one to evaluate the progression of the disease development. The protocol described here provides details on how co-immunostaining and analysis can be performed in tissue cultured cells or tissue sections.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25827882" target="_blank">
                    In vitro and in vivo assays for studying histone ubiquitination and deubiquitination.
                </a>
            </h2>
            <p>Posttranslational histone modifications play important roles in regulating chromatin structure and function (Rando, Curr Opin Genet Dev 22:148-155, 2012; Zentner and Henikoff, Nat Struct Mol Biol 20:259-266, 2013). One example of such modifications is histone ubiquitination, which occurs predominately on H2A and H2B. <b>Recent studies have highlighted important regulatory roles of H2A ubiquitination in Polycomb group protein-mediated gene silencing and DNA damage repair (de Napoles et al., Dev Cell 7:663-676, 2004; Wang et al., Nature 431:873-878, 2004; Doil et al., Cell 136:435-446, 2009; Gatti et al., Cell Cycle 11:2538-2544, 2012; Mattiroli et al., Cell 150:1182-1195, 2012; Stewart et al., Cell 136:420-434, 2009; Bergink et al., Genes Dev 20:1343-1352, 2006; Facchino et al., J Neurosci 30:10096-10111, 2010; Ginjala et al., Mol Cell Biol 31:1972-1982, 2011; Ismail et al., J Cell Biol 191:45-60, 2010), H2B ubiquitination in transcription initiation and elongation (Xiao et al., Mol Cell Biol 25:637-651, 2005; Kao et al., Genes Dev 18:184-195, 2004; Pavri et al., Cell 125:703-717, 2006; Kim et al., Cell 137:459-471, 2009), pre-mRNA splicing (Jung et al.</b> Genome Res 22:1026-1035, 2012; Shieh et al., BMC Genomics 12:627, 2011; Zhang et al., Genes Dev 27:1581-1595, 2013), nucleosome stabilities (Fleming et al., Mol Cell 31:57-66, 2008; Chandrasekharan et al., Proc Natl Acad Sci U S A 106:16686-16691, 2009), H3 methylation (Sun and Allis, Nature 418:104-108, 2002; Briggs et al., Nature 418:498, 2002; Dover et al., J Biol Chem 277:28368-28371, 2002; Ng et al., J Biol Chem 277:34655-34657, 2002), and DNA methylation (Sridhar et al., Nature 447:735-738, 2007). Here we describe methods for in vitro histone ubiquitination and deubiquitination assays. We also describe approaches to investigate the in vivo function of putative histone ubiquitin ligase(s) and deubiquitinase(s). These experimental procedures are largely based on our studies in mammalian cells. These methods should provide useful tools for studying this bulky histone modification.</p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25827881" target="_blank">
                    Investigation of genomic methylation status using methylation-specific and bisulfite sequencing polymerase chain reaction.
                </a>
            </h2>
            <p>Epigenetic modification plays a central role in the regulation of gene expression and therefore in the development of disease states. <b>In particular, genomic methylation of cytosines within CpG dinucleotides is crucial to development, gene silencing, and chromosome inactivation.</b> <b>Importantly, aberrant methylation profiles of various genes are associated with cancer as well as autoimmune disease, psychiatric and neurodegenerative disorders, diabetes, and heart disease.</b> <b>Various methods are available for the detection and quantification of methylation in a given sample.</b> Most of these methods rely upon bisulfite conversion of DNA, which converts unmethylated cytosines to uracil, while methylated cytosines remain as cytosines. <b>Methylation-specific amplification of DNA can be used to detect methylation at one or more (typically up to about 4) CpG sites by using primers specific to either methylated or unmethylated DNA.</b> <b>Alternatively, amplification of both methylated and unmethylated DNA followed by sequencing can be used to detect methylation status at multiple CpG sites.</b> The following chapter provides protocols for bisulfite conversion of DNA, methylation-specific PCR and bisulfite sequencing PCR.</p>
            </div>
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25827880" target="_blank">
                    Determination of DNA Methylation Levels Using Illumina HumanMethylation450 BeadChips.
                </a>
            </h2>
            <p>DNA methylation is a modifiable epigenetic phenomenon that has a strong influence over transcriptional regulation and as such has been consistently implicated in development and disease. <b>Several platforms are targeted toward the identification of DNA methylation changes that might be pertinent to the disease process and include regional analysis (e.g., pyrosequencing) as well as genome-wide analysis (e.g., next-generation sequencing and microarray).</b> <b>The Illumina HumanMethylation450 BeadChip is one of the most comprehensive microarray platforms available, and due to the high costs associated with next-generation sequencing, it is becoming a widely used tool for the analysis of genome-wide DNA methylation levels.</b> <b>Providing quantitative DNA methylation levels at 482,421 CpG sites within CpG islands, shores, and shelves, as well as intergenic regions, the HumanMethylation450 BeadChip can allow accurate assessment of differential methylation across large studies.</b> <b>This chapter outlines the laboratory methodologies associated with performing the Illumina Infinium Methylation Assay, including bisulfite conversion, whole-genome amplification, BeadChip hybridization, XStain procedures, and imaging systems.</b> Furthermore, this chapter provides an outline of data analysis tools, including the GenomeStudio pipeline, quality control measures, and additional statistical considerations. <b>This comprehensive overview can aid not only in performing the Illumina Infinium Methylation Assay but also in the interpretation of data derived from this platform.</b></p>
            </div>
    </div>
    <div class="row">
            <div class="col-md-6">

            <h2>
                <a class="brand" href="http://www.ncbi.nlm.nih.gov/pubmed/25827879" target="_blank">
                    Integrated DNA methylation and chromatin structural analysis at single-molecule resolution.
                </a>
            </h2>
            <p>Chromatin limits the accessibility of DNA to trans-acting factors in transcription, replication, and repair. <b>Although transcriptional variation between cells in a population may contribute to survival and disease, most assays of chromatin structure recover only population averages.</b> <b>We have developed DNA methyltransferases (MTases) as probing agents of DNA accessibility in chromatin, either expressed in vivo in budding yeast or as recombinant enzymatic probes of nuclei isolated from mammalian cells.</b> <b>In this chapter, we focus on the use of recombinant MTase (M) M.CviPI to probe chromatin accessibility in nuclei isolated from mammalian cell lines and animal tissue.</b> This technique, named methylation accessibility protocol for individual templates (MAPit), reports protein-DNA interactions at single-molecule resolution. <b>The single-molecule readout allows identification of chromatin subpopulations and rare epigenetic variants within a cell population.</b> <b>Furthermore, the use of M.CviPI in mammalian systems gives a comprehensive view of both chromatin structure and endogenous DNA methylation in a single assay.</b></p>
            </div>
    </div>

</div>



    <div class="footer">
        <div class="container">
            <p class="text-muted pull-left">
                ©left: <b>rlim</b> 
                <a href="https://github.com/rickylim19/pubmedylicious" target="_blank"><i class="fa fa-github"></i></a>
            </p>
        </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="./DNAMeth_files/jquery.min.js"></script>
    <script src="./DNAMeth_files/bootstrap.min.js"></script>
    <script src="./DNAMeth_files/jquery.min(1).js"></script>

  

  
<script src="./DNAMeth_files/tagcanvas.min.js" type="text/javascript"></script>
<script src="./DNAMeth_files/Chart.js"></script>
<script src="./DNAMeth_files/d3.min.js"></script>
<script src="./DNAMeth_files/d3.layout.cloud.js"></script>
<script type="text/javascript">
    // Author Tag
    var authortag = function(){
        try {
        TagCanvas.Start('authortag','tags',{
            textColour: '#3a87ad',
            outlineColour: 'transparent',
            textFont: 'Helvetica',
            reverse: true,
            weight: true,
            depth: 0.8,
            maxSpeed: 0.05
        });


        } catch(e) {
        // something went wrong, hide the canvas container
        document.getElementById('authorContainer').style.display = 'none';

        }
    }

    // pubtrend
    var lineChartData = {
        //labels : ["January","February","March"],
        labels : ['2014', '2015'],
        //labels : ,
        datasets : [
            {
                label: "# of Yearly Publications",
                fillColor : "rgba(220,220,220,0.2)",
                strokeColor : "rgba(220,220,220,1)",
                pointColor : "rgba(220,220,220,1)",
                pointStrokeColor : "#fff",
                pointHighlightFill : "#fff",
                pointHighlightStroke : "rgba(220,220,220,1)",
                data : [2, 188]
                //data : 
                //data : [randomScalingFactor(),randomScalingFactor(),randomScalingFactor()]
            }
        ]

    }

    var pubtrend = function(){
        var ctx = document.getElementById("pubtrend").getContext("2d");
        window.myLine = new Chart(ctx).Line(lineChartData, {
            responsive: true,
            bezierCurve: false,
            scaleBeginAtZero: true,
        });
    }

    // countrytrend
    var countrydata = [
        
        // e.g. countrydict = {'chine': (3, '#F7464A', '#FF5A5E')} 
        // value[0] is the freq, value[1] is the color, value[2] is the highlight color
            // value: 50,
            // color: "#46BFBD",
            // highlight: "#5AD3D1",
            // label: "Green"
        
        {
            value: 4,
            color: "#F7464A",
            highlight: "#FF5A5E", 
            label: "switzerland"
        },
        
        {
            value: 40,
            color: "#46BFBD",
            highlight: "#5AD3D1", 
            label: "united states"
        },
        
        {
            value: 5,
            color: "#FDB45C",
            highlight: "FFC870", 
            label: "netherlands"
        },
        
        {
            value: 32,
            color: "#949FB1",
            highlight: "#A8B3C5", 
            label: "england"
        },
        
        {
            value: 7,
            color: "#4D5360",
            highlight: "#616774", 
            label: "germany"
        },
        

    ];
    var countrytrend = function(){
        //debugger
        var ctx = document.getElementById("countrytrend").getContext("2d");
        window.myLine = new Chart(ctx).PolarArea(countrydata, {
            responsive: true,
            scaleShowLabels: true,
        });
    }

    window.onload = function() {
        authortag();
        pubtrend();
        countrytrend();
    };

    // journal cloud
    var frequency_list = [
    {"text":"Cancer treatment reviews","size":20},
    {"text":"The journal of experimental medicine","size":20},
    {"text":"International journal of epidemiology","size":120},
    {"text":"The journal of biological chemistry","size":40},
    {"text":"Nucleic acids research","size":40},
    {"text":"Bmc neuroscience","size":20},
    {"text":"Oncotarget","size":40},
    {"text":"Comparative biochemistry and physiology. part b, biochemistry & molecular biology","size":20},
    {"text":"Molecular biology of the cell","size":20},
    {"text":"Bone","size":40},
    {"text":"Journal of developmental origins of health and disease","size":40},
    {"text":"Pharmacology & therapeutics","size":20},
    {"text":"Molekuliarnaia biologiia","size":40},
    {"text":"Neuropsychopharmacology : official publication of the american college of neuropsychopharmacology","size":20},
    {"text":"Biochemical and biophysical research communications","size":20},
    {"text":"Laboratory investigation; a journal of technical methods and pathology","size":20},
    {"text":"Epigenomics","size":40},
    {"text":"Medicine","size":20},
    {"text":"Bmc bioinformatics","size":40},
    {"text":"Clinica chimica acta; international journal of clinical chemistry","size":20},
    {"text":"Methods in molecular biology (clifton, n.j.)","size":100},
    {"text":"Frontiers in endocrinology","size":20},
    {"text":"Journal of affective disorders","size":20},
    {"text":"Journal of translational medicine","size":40},
    {"text":"Epigenetics : official journal of the dna methylation society","size":120},
    {"text":"Hepatology (baltimore, md.)","size":20},
    {"text":"Experimental cell research","size":40},
    {"text":"Biosensors & bioelectronics","size":40},
    {"text":"Human molecular genetics","size":40},
    {"text":"Gene","size":20},
    {"text":"Embo reports","size":20},
    {"text":"Diabetic medicine : a journal of the british diabetic association","size":20},
    {"text":"Molecular syndromology","size":20},
    {"text":"Bmc genomics","size":140},
    {"text":"Molecular cancer","size":40},
    {"text":"Science (new york, n.y.)","size":40},
    {"text":"Endocrine-related cancer","size":20},
    {"text":"Clinical epigenetics","size":140},
    {"text":"Alcohol (fayetteville, n.y.)","size":20},
    {"text":"Angewandte chemie (international ed. in english)","size":20},
    {"text":"Journal of clinical pathology","size":20},
    {"text":"Mutation research","size":60},
    {"text":"Current problems in pediatric and adolescent health care","size":20},
    {"text":"Genome biology","size":140},
    {"text":"Wounds : a compendium of clinical research and practice","size":20},
    {"text":"Journal of psychiatric research","size":40},
    {"text":"Bmc research notes","size":40},
    {"text":"The journal of nutrition","size":40},
    {"text":"Proceedings of the national academy of sciences of the united states of america","size":60},
    {"text":"Plos one","size":300},

    ]
var color = d3.scale.linear()
        .domain([0,1,2,3,4,5,6,10,15,20,100])
        .range(["#ddd", "#ccc", "#bbb", "#aaa", "#999", "#888", "#777", "#666", "#555", "#444", "#333", "#222"]);

d3.layout.cloud().size([500,500 ])
        .words(frequency_list)
        .padding(5)
        .rotate(0)
        .font("Impact")
        .fontSize(function(d) { return d.size; })
        .on("end", draw)
        .start();

function draw(words) {
    d3.select("#journalcontainer").append("svg")
            .attr("width", 500)
            .attr("height", 500)
            .attr("class", "wordcloud")
            .append("g")
            // without the transform, words words would get cutoff to the left and top, they would
            // appear outside of the SVG area
            .attr("transform", "translate(250,250)")
            .selectAll("text")
            .data(words)
            .enter().append("text")
            .style("font-size", function(d) { return d.size + "px"; })
            .style("fill", function(d, i) { return color(i); })
            .attr("text-anchor", "middle")
            .attr("transform", function(d) {
                return "translate(" + [d.x, d.y] + ")rotate(" + d.rotate + ")";
            })
            .text(function(d) { return d.text; });
}

</script>


</body></html>
